<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Compute read counts — compute_read_counts • recount3</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet"><meta property="og:title" content="Compute read counts — compute_read_counts"><meta property="og:description" content="As described in the recount workflow, the counts provided by the recount2
project are base-pair counts. You can scale them using transform_counts()
or compute the read counts using the area under coverage information (AUC)."><meta property="og:image" content="/logo.png"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.9.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/LieberInstitute/recount3/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Compute read counts</h1>
    <small class="dont-index">Source: <a href="https://github.com/LieberInstitute/recount3/blob/HEAD/R/transform_counts.R" class="external-link"><code>R/transform_counts.R</code></a></small>
    <div class="hidden name"><code>compute_read_counts.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>As described in the recount workflow, the counts provided by the recount2
project are base-pair counts. You can scale them using <code><a href="transform_counts.html">transform_counts()</a></code>
or compute the read counts using the area under coverage information (AUC).</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">compute_read_counts</span><span class="op">(</span></span>
<span>  <span class="va">rse</span>,</span>
<span>  round <span class="op">=</span> <span class="cn">TRUE</span>,</span>
<span>  avg_mapped_read_length <span class="op">=</span> <span class="st">"recount_qc.star.average_mapped_length"</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>
    <dl><dt>rse</dt>
<dd><p>A
<a href="https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html" class="external-link">RangedSummarizedExperiment-class</a>
created by <code><a href="create_rse.html">create_rse()</a></code>.</p></dd>


<dt>round</dt>
<dd><p>A <code>logical(1)</code> specifying whether to round the transformed
counts or not.</p></dd>


<dt>avg_mapped_read_length</dt>
<dd><p>A <code>character(1)</code> specifying the metdata column
name that contains the average fragment length after aligning. This is
typically twice the average read length for paired-end reads.</p></dd>

</dl></div>
    <div id="value">
    <h2>Value</h2>
    

<p>A <code><a href="https://rdrr.io/r/base/matrix.html" class="external-link">matrix()</a></code> with the read counts. By default this function uses
the average read length to the QC annotation.</p>
    </div>
    <div id="details">
    <h2>Details</h2>
    <p>This function is similar to
<code>recount::read_counts(use_paired_end = TRUE, round = TRUE)</code> but more general
and with a different name to avoid NAMESPACE conflicts. Note that the default
value of <code>round</code> is different than in <code><a href="https://rdrr.io/pkg/recount/man/read_counts.html" class="external-link">recount::read_counts()</a></code>. This
was done to match the default value of <code>round</code> in <code><a href="transform_counts.html">transform_counts()</a></code>.</p>
    </div>
    <div id="references">
    <h2>References</h2>
    <p>Collado-Torres L, Nellore A and Jaffe AE. recount workflow: Accessing over
70,000 human RNA-seq samples with Bioconductor version 1; referees: 1
approved, 2 approved with reservations. F1000Research 2017, 6:1558
doi: 10.12688/f1000research.12223.1.</p>
    </div>
    <div id="see-also">
    <h2>See also</h2>
    <div class="dont-index"><p>Other count transformation functions: 
<code><a href="compute_scale_factors.html">compute_scale_factors</a>()</code>,
<code><a href="is_paired_end.html">is_paired_end</a>()</code>,
<code><a href="transform_counts.html">transform_counts</a>()</code></p></div>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Create a RSE object at the gene level</span></span></span>
<span class="r-in"><span><span class="va">rse_gene_SRP009615</span> <span class="op">&lt;-</span> <span class="fu"><a href="create_rse_manual.html">create_rse_manual</a></span><span class="op">(</span><span class="st">"SRP009615"</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:45:57 downloading and reading the metadata.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:45:57 caching file sra.sra.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:45:58 caching file sra.recount_project.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:45:59 caching file sra.recount_qc.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:45:59 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:00 caching file sra.recount_pred.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:00 downloading and reading the feature information.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:01 caching file human.gene_sums.G026.gtf.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:02 downloading and reading the counts: 12 samples across 63856 features.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:02 caching file sra.gene_sums.SRP009615.G026.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:02 constructing the RangedSummarizedExperiment (rse) object.</span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="fu">compute_read_counts</span><span class="op">(</span><span class="va">rse_gene_SRP009615</span><span class="op">)</span><span class="op">)</span> <span class="op">/</span> <span class="fl">1e6</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> SRR387777 SRR387778 SRR387779 SRR387780 SRR389079 SRR389080 SRR389081 SRR389082 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  21.29962  23.66484  32.62525  25.46085  41.48717  28.05485  17.78092  15.46669 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> SRR389083 SRR389084 SRR389077 SRR389078 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  18.84982  18.18674  13.88166  15.14785 </span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Create a RSE object at the gene level</span></span></span>
<span class="r-in"><span><span class="va">rse_gene_DRP000499</span> <span class="op">&lt;-</span> <span class="fu"><a href="create_rse_manual.html">create_rse_manual</a></span><span class="op">(</span><span class="st">"DRP000499"</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:03 downloading and reading the metadata.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:03 caching file sra.sra.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:04 caching file sra.recount_project.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:05 caching file sra.recount_qc.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:06 caching file sra.recount_seq_qc.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:06 caching file sra.recount_pred.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:06 downloading and reading the feature information.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:07 caching file human.gene_sums.G026.gtf.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:08 downloading and reading the counts: 21 samples across 63856 features.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:08 caching file sra.gene_sums.DRP000499.G026.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:09 constructing the RangedSummarizedExperiment (rse) object.</span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="fu">compute_read_counts</span><span class="op">(</span><span class="va">rse_gene_DRP000499</span><span class="op">)</span><span class="op">)</span> <span class="op">/</span> <span class="fl">1e6</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> DRR001622 DRR001623 DRR001624 DRR001625 DRR001626 DRR001627 DRR001628 DRR001629 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 10.952140  7.995718 11.874082       NaN  7.518984 11.272031 53.412769 27.539984 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> DRR001630 DRR001631 DRR001632 DRR001633 DRR001634 DRR001635 DRR001636 DRR001637 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 27.721930 35.301338 48.673389 20.585057 35.074474 31.622335 34.844632 46.270835 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> DRR001638 DRR001639 DRR001640 DRR001641 DRR001642 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 31.505395 26.991040 18.916860 17.788569 25.389886 </span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## You can compare the read counts against those from recount::read_counts()</span></span></span>
<span class="r-in"><span><span class="co">## from the recount2 project which used a different RNA-seq aligner</span></span></span>
<span class="r-in"><span><span class="co">## If needed, install recount, the R/Bioconductor package for recount2:</span></span></span>
<span class="r-in"><span><span class="co"># BiocManager::install("recount")</span></span></span>
<span class="r-in"><span><span class="va">recount2_readsums</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="fu">assay</span><span class="op">(</span><span class="fu">recount</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/recount/man/read_counts.html" class="external-link">read_counts</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="fu">recount</span><span class="fu">::</span><span class="va"><a href="https://rdrr.io/pkg/recount/man/rse_gene_SRP009615.html" class="external-link">rse_gene_SRP009615</a></span></span></span>
<span class="r-in"><span><span class="op">)</span>, <span class="st">"counts"</span><span class="op">)</span><span class="op">)</span> <span class="op">/</span> <span class="fl">1e6</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Setting options('download.file.method.GEOquery'='auto')</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Setting options('GEOquery.inmemory.gpl'=FALSE)</span>
<span class="r-in"><span><span class="va">recount3_readsums</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="fu">compute_read_counts</span><span class="op">(</span><span class="va">rse_gene_SRP009615</span><span class="op">)</span><span class="op">)</span> <span class="op">/</span> <span class="fl">1e6</span></span></span>
<span class="r-in"><span><span class="va">recount_readsums</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    recount2 <span class="op">=</span> <span class="va">recount2_readsums</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/order.html" class="external-link">order</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">recount2_readsums</span><span class="op">)</span><span class="op">)</span><span class="op">]</span>,</span></span>
<span class="r-in"><span>    recount3 <span class="op">=</span> <span class="va">recount3_readsums</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/order.html" class="external-link">order</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">recount3_readsums</span><span class="op">)</span><span class="op">)</span><span class="op">]</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">recount2</span> <span class="op">~</span> <span class="va">recount3</span>, data <span class="op">=</span> <span class="va">recount_readsums</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/graphics/abline.html" class="external-link">abline</a></span><span class="op">(</span>a <span class="op">=</span> <span class="fl">0</span>, b <span class="op">=</span> <span class="fl">1</span>, col <span class="op">=</span> <span class="st">"purple"</span>, lwd <span class="op">=</span> <span class="fl">2</span>, lty <span class="op">=</span> <span class="fl">2</span><span class="op">)</span></span></span>
<span class="r-plt img"><img src="compute_read_counts-1.png" alt="" width="700" height="433"></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Repeat for DRP000499, a paired-end study</span></span></span>
<span class="r-in"><span><span class="fu">recount</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/recount/man/download_study.html" class="external-link">download_study</a></span><span class="op">(</span><span class="st">"DRP000499"</span>, outdir <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/tempfile.html" class="external-link">tempdir</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-01-11 17:46:14 downloading file rse_gene.Rdata to /tmp/RtmpUobOFE</span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/load.html" class="external-link">load</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/file.path.html" class="external-link">file.path</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/tempfile.html" class="external-link">tempdir</a></span><span class="op">(</span><span class="op">)</span>, <span class="st">"rse_gene.Rdata"</span><span class="op">)</span>, verbose <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Loading objects:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   rse_gene</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="va">recount2_readsums</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="fu">assay</span><span class="op">(</span><span class="fu">recount</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/recount/man/read_counts.html" class="external-link">read_counts</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="va">rse_gene</span></span></span>
<span class="r-in"><span><span class="op">)</span>, <span class="st">"counts"</span><span class="op">)</span><span class="op">)</span> <span class="op">/</span> <span class="fl">1e6</span></span></span>
<span class="r-in"><span><span class="va">recount3_readsums</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="fu">compute_read_counts</span><span class="op">(</span><span class="va">rse_gene_DRP000499</span><span class="op">)</span><span class="op">)</span> <span class="op">/</span> <span class="fl">1e6</span></span></span>
<span class="r-in"><span><span class="va">recount_readsums</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    recount2 <span class="op">=</span> <span class="va">recount2_readsums</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/order.html" class="external-link">order</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">recount2_readsums</span><span class="op">)</span><span class="op">)</span><span class="op">]</span>,</span></span>
<span class="r-in"><span>    recount3 <span class="op">=</span> <span class="va">recount3_readsums</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/order.html" class="external-link">order</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">recount3_readsums</span><span class="op">)</span><span class="op">)</span><span class="op">]</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">recount2</span> <span class="op">~</span> <span class="va">recount3</span>, data <span class="op">=</span> <span class="va">recount_readsums</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/graphics/abline.html" class="external-link">abline</a></span><span class="op">(</span>a <span class="op">=</span> <span class="fl">0</span>, b <span class="op">=</span> <span class="fl">1</span>, col <span class="op">=</span> <span class="st">"purple"</span>, lwd <span class="op">=</span> <span class="fl">2</span>, lty <span class="op">=</span> <span class="fl">2</span><span class="op">)</span></span></span>
<span class="r-plt img"><img src="compute_read_counts-2.png" alt="" width="700" height="433"></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer></div>

  


  

  </body></html>

