<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Compute read counts — compute_read_counts • recount3</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Compute read counts — compute_read_counts" />
<meta property="og:description" content="As described in the recount workflow, the counts provided by the recount2
project are base-pair counts. You can scale them using transform_counts()
or compute the read counts using the area under coverage information (AUC)." />
<meta property="og:image" content="/logo.png" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.9</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LieberInstitute/recount3/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Compute read counts</h1>
    <small class="dont-index">Source: <a href='https://github.com/LieberInstitute/recount3/blob/master/R/transform_counts.R'><code>R/transform_counts.R</code></a></small>
    <div class="hidden name"><code>compute_read_counts.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>As described in the recount workflow, the counts provided by the recount2
project are base-pair counts. You can scale them using <code><a href='transform_counts.html'>transform_counts()</a></code>
or compute the read counts using the area under coverage information (AUC).</p>
    </div>

    <pre class="usage"><span class='fu'>compute_read_counts</span><span class='op'>(</span>
  <span class='va'>rse</span>,
  round <span class='op'>=</span> <span class='cn'>TRUE</span>,
  avg_mapped_read_length <span class='op'>=</span> <span class='st'>"recount_qc.star.average_mapped_length"</span>
<span class='op'>)</span></pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>rse</th>
      <td><p>A
<a href='https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html'>RangedSummarizedExperiment-class</a>
created by <code><a href='create_rse.html'>create_rse()</a></code>.</p></td>
    </tr>
    <tr>
      <th>round</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/logical.html'>logical(1)</a></code> specifying whether to round the transformed
counts or not.</p></td>
    </tr>
    <tr>
      <th>avg_mapped_read_length</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying the metdata column
name that contains the average fragment length after aligning. This is
typically twice the average read length for paired-end reads.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A <code><a href='https://rdrr.io/r/base/matrix.html'>matrix()</a></code> with the read counts. By default this function uses
the average read length to the QC annotation.</p>
    <h2 class="hasAnchor" id="details"><a class="anchor" href="#details"></a>Details</h2>

    <p>This function is similar to
<code><a href='https://rdrr.io/pkg/recount/man/read_counts.html'>recount::read_counts(use_paired_end = TRUE, round = TRUE)</a></code> but more general
and with a different name to avoid NAMESPACE conflicts. Note that the default
value of <code>round</code> is different than in <code><a href='https://rdrr.io/pkg/recount/man/read_counts.html'>recount::read_counts()</a></code>. This
was done to match the default value of <code>round</code> in <code><a href='transform_counts.html'>transform_counts()</a></code>.</p>
    <h2 class="hasAnchor" id="references"><a class="anchor" href="#references"></a>References</h2>

    <p>Collado-Torres L, Nellore A and Jaffe AE. recount workflow: Accessing over
70,000 human RNA-seq samples with Bioconductor version 1; referees: 1
approved, 2 approved with reservations. F1000Research 2017, 6:1558
doi: 10.12688/f1000research.12223.1.</p>
    <h2 class="hasAnchor" id="see-also"><a class="anchor" href="#see-also"></a>See also</h2>

    <div class='dont-index'><p>Other count transformation functions: 
<code><a href='compute_scale_factors.html'>compute_scale_factors</a>()</code>,
<code><a href='is_paired_end.html'>is_paired_end</a>()</code>,
<code><a href='transform_counts.html'>transform_counts</a>()</code></p></div>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='co'>## Create a RSE object at the gene level</span>
<span class='va'>rse_gene_SRP009615</span> <span class='op'>&lt;-</span> <span class='fu'><a href='create_rse_manual.html'>create_rse_manual</a></span><span class='op'>(</span><span class='st'>"SRP009615"</span><span class='op'>)</span>
</div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:04 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:05 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.sra.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:05 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_project.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:05 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_qc.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:06 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_seq_qc.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:06 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_pred.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:07 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:07 caching file human.gene_sums.G026.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/gene_sums/human.gene_sums.G026.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:08 downloading and reading the counts: 12 samples across 63856 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:08 caching file sra.gene_sums.SRP009615.G026.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/15/SRP009615/sra.gene_sums.SRP009615.G026.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:09 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'><a href='https://rdrr.io/r/base/colSums.html'>colSums</a></span><span class='op'>(</span><span class='fu'>compute_read_counts</span><span class='op'>(</span><span class='va'>rse_gene_SRP009615</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>/</span> <span class='fl'>1e6</span>
</div><div class='output co'>#&gt; SRR387777 SRR387778 SRR387779 SRR387780 SRR389079 SRR389080 SRR389081 SRR389082 
#&gt;  21.33424  23.70639  32.67551  25.50398  41.55222  28.10329  17.81035  15.49213 
#&gt; SRR389083 SRR389084 SRR389077 SRR389078 
#&gt;  18.88160  18.21900  13.90269  15.17254 </div><div class='input'>
<span class='co'>## Create a RSE object at the gene level</span>
<span class='va'>rse_gene_DRP000499</span> <span class='op'>&lt;-</span> <span class='fu'><a href='create_rse_manual.html'>create_rse_manual</a></span><span class='op'>(</span><span class='st'>"DRP000499"</span><span class='op'>)</span>
</div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:09 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:09 caching file sra.sra.DRP000499.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/99/DRP000499/sra.sra.DRP000499.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:10 caching file sra.recount_project.DRP000499.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/99/DRP000499/sra.recount_project.DRP000499.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:10 caching file sra.recount_qc.DRP000499.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/99/DRP000499/sra.recount_qc.DRP000499.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:10 caching file sra.recount_seq_qc.DRP000499.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/99/DRP000499/sra.recount_seq_qc.DRP000499.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:11 caching file sra.recount_pred.DRP000499.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/99/DRP000499/sra.recount_pred.DRP000499.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:11 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:11 caching file human.gene_sums.G026.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:12 downloading and reading the counts: 21 samples across 63856 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:17 caching file sra.gene_sums.DRP000499.G026.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/99/DRP000499/sra.gene_sums.DRP000499.G026.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:18 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'><a href='https://rdrr.io/r/base/colSums.html'>colSums</a></span><span class='op'>(</span><span class='fu'>compute_read_counts</span><span class='op'>(</span><span class='va'>rse_gene_DRP000499</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>/</span> <span class='fl'>1e6</span>
</div><div class='output co'>#&gt; DRR001622 DRR001623 DRR001624 DRR001625 DRR001626 DRR001627 DRR001628 DRR001629 
#&gt; 10.972020  8.007966 11.893009       NaN  7.530776 11.289832 53.523929 27.598855 
#&gt; DRR001630 DRR001631 DRR001632 DRR001633 DRR001634 DRR001635 DRR001636 DRR001637 
#&gt; 27.775202 35.363611 48.757827 20.621330 35.136076 31.677099 34.907348 46.346187 
#&gt; DRR001638 DRR001639 DRR001640 DRR001641 DRR001642 
#&gt; 31.559936 27.044245 18.953532 17.820245 25.432056 </div><div class='input'>
<span class='co'>## You can compare the read counts against those from recount::read_counts()</span>
<span class='co'>## from the recount2 project which used a different RNA-seq aligner</span>
<span class='co'>## If needed, install recount, the R/Bioconductor package for recount2:</span>
<span class='co'># BiocManager::install("recount")</span>
<span class='va'>recount2_readsums</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/colSums.html'>colSums</a></span><span class='op'>(</span><span class='fu'>assay</span><span class='op'>(</span><span class='fu'>recount</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/recount/man/read_counts.html'>read_counts</a></span><span class='op'>(</span>
    <span class='fu'>recount</span><span class='fu'>::</span><span class='va'><a href='https://rdrr.io/pkg/recount/man/rse_gene_SRP009615.html'>rse_gene_SRP009615</a></span>
<span class='op'>)</span>, <span class='st'>"counts"</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>/</span> <span class='fl'>1e6</span>
</div><div class='output co'>#&gt; <span class='message'>Setting options('download.file.method.GEOquery'='auto')</span></div><div class='output co'>#&gt; <span class='message'>Setting options('GEOquery.inmemory.gpl'=FALSE)</span></div><div class='output co'>#&gt; <span class='message'>No methods found in package ‘IRanges’ for request: ‘values&lt;-’ when loading ‘derfinder’</span></div><div class='output co'>#&gt; <span class='message'>No methods found in package ‘IRanges’ for request: ‘values’ when loading ‘derfinder’</span></div><div class='input'><span class='va'>recount3_readsums</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/colSums.html'>colSums</a></span><span class='op'>(</span><span class='fu'>compute_read_counts</span><span class='op'>(</span><span class='va'>rse_gene_SRP009615</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>/</span> <span class='fl'>1e6</span>
<span class='va'>recount_readsums</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span><span class='op'>(</span>
    recount2 <span class='op'>=</span> <span class='va'>recount2_readsums</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/order.html'>order</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span><span class='op'>(</span><span class='va'>recount2_readsums</span><span class='op'>)</span><span class='op'>)</span><span class='op'>]</span>,
    recount3 <span class='op'>=</span> <span class='va'>recount3_readsums</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/order.html'>order</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span><span class='op'>(</span><span class='va'>recount3_readsums</span><span class='op'>)</span><span class='op'>)</span><span class='op'>]</span>
<span class='op'>)</span>
<span class='fu'><a href='https://rdrr.io/r/graphics/plot.default.html'>plot</a></span><span class='op'>(</span><span class='va'>recount2</span> <span class='op'>~</span> <span class='va'>recount3</span>, data <span class='op'>=</span> <span class='va'>recount_readsums</span><span class='op'>)</span>
</div><div class='input'><span class='fu'><a href='https://rdrr.io/r/graphics/abline.html'>abline</a></span><span class='op'>(</span>a <span class='op'>=</span> <span class='fl'>0</span>, b <span class='op'>=</span> <span class='fl'>1</span>, col <span class='op'>=</span> <span class='st'>"purple"</span>, lwd <span class='op'>=</span> <span class='fl'>2</span>, lty <span class='op'>=</span> <span class='fl'>2</span><span class='op'>)</span>
</div><div class='img'><img src='compute_read_counts-1.png' alt='' width='700' height='433' /></div><div class='input'>
<span class='co'>## Repeat for DRP000499, a paired-end study</span>
<span class='fu'>recount</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/recount/man/download_study.html'>download_study</a></span><span class='op'>(</span><span class='st'>"DRP000499"</span>, outdir <span class='op'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/tempfile.html'>tempdir</a></span><span class='op'>(</span><span class='op'>)</span><span class='op'>)</span>
</div><div class='output co'>#&gt; <span class='message'>2020-10-16 23:35:24 downloading file rse_gene.Rdata to /tmp/RtmpqhS9Ga</span></div><div class='input'><span class='fu'><a href='https://rdrr.io/r/base/load.html'>load</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/file.path.html'>file.path</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/tempfile.html'>tempdir</a></span><span class='op'>(</span><span class='op'>)</span>, <span class='st'>"rse_gene.Rdata"</span><span class='op'>)</span>, verbose <span class='op'>=</span> <span class='cn'>TRUE</span><span class='op'>)</span>
</div><div class='output co'>#&gt; Loading objects:
#&gt;   rse_gene</div><div class='input'>
<span class='va'>recount2_readsums</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/colSums.html'>colSums</a></span><span class='op'>(</span><span class='fu'>assay</span><span class='op'>(</span><span class='fu'>recount</span><span class='fu'>::</span><span class='fu'><a href='https://rdrr.io/pkg/recount/man/read_counts.html'>read_counts</a></span><span class='op'>(</span>
    <span class='va'>rse_gene</span>
<span class='op'>)</span>, <span class='st'>"counts"</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>/</span> <span class='fl'>1e6</span>
<span class='va'>recount3_readsums</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/colSums.html'>colSums</a></span><span class='op'>(</span><span class='fu'>compute_read_counts</span><span class='op'>(</span><span class='va'>rse_gene_DRP000499</span><span class='op'>)</span><span class='op'>)</span> <span class='op'>/</span> <span class='fl'>1e6</span>
<span class='va'>recount_readsums</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span><span class='op'>(</span>
    recount2 <span class='op'>=</span> <span class='va'>recount2_readsums</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/order.html'>order</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span><span class='op'>(</span><span class='va'>recount2_readsums</span><span class='op'>)</span><span class='op'>)</span><span class='op'>]</span>,
    recount3 <span class='op'>=</span> <span class='va'>recount3_readsums</span><span class='op'>[</span><span class='fu'><a href='https://rdrr.io/r/base/order.html'>order</a></span><span class='op'>(</span><span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span><span class='op'>(</span><span class='va'>recount3_readsums</span><span class='op'>)</span><span class='op'>)</span><span class='op'>]</span>
<span class='op'>)</span>
<span class='fu'><a href='https://rdrr.io/r/graphics/plot.default.html'>plot</a></span><span class='op'>(</span><span class='va'>recount2</span> <span class='op'>~</span> <span class='va'>recount3</span>, data <span class='op'>=</span> <span class='va'>recount_readsums</span><span class='op'>)</span>
</div><div class='input'><span class='fu'><a href='https://rdrr.io/r/graphics/abline.html'>abline</a></span><span class='op'>(</span>a <span class='op'>=</span> <span class='fl'>0</span>, b <span class='op'>=</span> <span class='fl'>1</span>, col <span class='op'>=</span> <span class='st'>"purple"</span>, lwd <span class='op'>=</span> <span class='fl'>2</span>, lty <span class='op'>=</span> <span class='fl'>2</span><span class='op'>)</span>
</div><div class='img'><img src='compute_read_counts-2.png' alt='' width='700' height='433' /></div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.9000.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


