<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Compute count scaling factors — compute_scale_factors • recount3</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet"><meta property="og:title" content="Compute count scaling factors — compute_scale_factors"><meta property="og:description" content="This function computes the count scaling factors used by
transform_counts(). This function is similar to
recount::scale_counts(factor_only = TRUE), but it is more general."><meta property="og:image" content="/logo.png"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.9.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/LieberInstitute/recount3/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Compute count scaling factors</h1>
    <small class="dont-index">Source: <a href="https://github.com/LieberInstitute/recount3/blob/HEAD/R/transform_counts.R" class="external-link"><code>R/transform_counts.R</code></a></small>
    <div class="hidden name"><code>compute_scale_factors.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function computes the count scaling factors used by
<code><a href="transform_counts.html">transform_counts()</a></code>. This function is similar to
<code>recount::scale_counts(factor_only = TRUE)</code>, but it is more general.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">compute_scale_factors</span><span class="op">(</span></span>
<span>  <span class="va">x</span>,</span>
<span>  by <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"auc"</span>, <span class="st">"mapped_reads"</span><span class="op">)</span>,</span>
<span>  targetSize <span class="op">=</span> <span class="fl">4e+07</span>,</span>
<span>  L <span class="op">=</span> <span class="fl">100</span>,</span>
<span>  auc <span class="op">=</span> <span class="st">"recount_qc.bc_auc.all_reads_all_bases"</span>,</span>
<span>  avg_mapped_read_length <span class="op">=</span> <span class="st">"recount_qc.star.average_mapped_length"</span>,</span>
<span>  mapped_reads <span class="op">=</span> <span class="st">"recount_qc.star.all_mapped_reads"</span>,</span>
<span>  paired_end <span class="op">=</span> <span class="fu"><a href="is_paired_end.html">is_paired_end</a></span><span class="op">(</span><span class="va">x</span>, <span class="va">avg_mapped_read_length</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>
    <dl><dt>x</dt>
<dd><p>Either a
<a href="https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html" class="external-link">RangedSummarizedExperiment-class</a>
created by <code><a href="create_rse.html">create_rse()</a></code> or the sample metadata created by
<code><a href="read_metadata.html">read_metadata()</a></code>.</p></dd>


<dt>by</dt>
<dd><p>Either <code>auc</code> or <code>mapped_reads</code>. If set to <code>auc</code> it
will compute the scaling factor by the total coverage of the sample. That is,
the area under the curve (AUC) of the coverage. If set to <code>mapped_reads</code> it
will scale the counts by the number of mapped reads (in the QC annotation),
whether the library was paired-end or not, and the desired read length (<code>L</code>).</p></dd>


<dt>targetSize</dt>
<dd><p>A <code>numeric(1)</code> specifying the target library size in number
of single end reads.</p></dd>


<dt>L</dt>
<dd><p>A <code>integer(1)</code> specifying the target read length. It is only used
when <code>by = 'mapped_reads'</code> since it cancels out in the calculation when
using <code>by = 'auc'</code>.</p></dd>


<dt>auc</dt>
<dd><p>A <code>character(1)</code> specifying the metadata column
name that contains the area under the coverage (AUC). Note that there are
several possible AUC columns provided in the sample metadata generated
by <code><a href="create_rse.html">create_rse()</a></code>.</p></dd>


<dt>avg_mapped_read_length</dt>
<dd><p>A <code>character(1)</code> specifying the metdata column
name that contains the average fragment length after aligning. This is
typically twice the average read length for paired-end reads.</p></dd>


<dt>mapped_reads</dt>
<dd><p>A <code>character(1)</code> specifying the metadata column
name that contains the number of mapped reads.</p></dd>


<dt>paired_end</dt>
<dd><p>A <code><a href="https://rdrr.io/r/base/logical.html" class="external-link">logical()</a></code> vector specifying whether each
sample is paired-end or not.</p></dd>

</dl></div>
    <div id="value">
    <h2>Value</h2>
    

<p>A <code><a href="https://rdrr.io/r/base/numeric.html" class="external-link">numeric()</a></code> with the sample scale factors that are used by
<code><a href="transform_counts.html">transform_counts()</a></code>.</p>
    </div>
    <div id="see-also">
    <h2>See also</h2>
    <div class="dont-index"><p>Other count transformation functions: 
<code><a href="compute_read_counts.html">compute_read_counts</a>()</code>,
<code><a href="is_paired_end.html">is_paired_end</a>()</code>,
<code><a href="transform_counts.html">transform_counts</a>()</code></p></div>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Download the metadata for SRP009615, a single-end study</span></span></span>
<span class="r-in"><span><span class="va">SRP009615_meta</span> <span class="op">&lt;-</span> <span class="fu"><a href="read_metadata.html">read_metadata</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    metadata_files <span class="op">=</span> <span class="fu"><a href="file_retrieve.html">file_retrieve</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>        <span class="fu"><a href="locate_url.html">locate_url</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>            <span class="st">"SRP009615"</span>,</span></span>
<span class="r-in"><span>            <span class="st">"data_sources/sra"</span>,</span></span>
<span class="r-in"><span>        <span class="op">)</span></span></span>
<span class="r-in"><span>    <span class="op">)</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:16 caching file sra.sra.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:16 caching file sra.recount_project.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:17 caching file sra.recount_qc.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:17 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:17 caching file sra.recount_pred.SRP009615.MD.gz.</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Compute the scaling factors</span></span></span>
<span class="r-in"><span><span class="fu">compute_scale_factors</span><span class="op">(</span><span class="va">SRP009615_meta</span>, by <span class="op">=</span> <span class="st">"auc"</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  SRR387777  SRR387778  SRR387779  SRR387780  SRR389079  SRR389080  SRR389081 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.03996103 0.03457138 0.03071544 0.03833955 0.02367289 0.03517519 0.05286476 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  SRR389082  SRR389083  SRR389084  SRR389077  SRR389078 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.06658138 0.05071328 0.05204845 0.04585471 0.04066468 </span>
<span class="r-in"><span><span class="fu">compute_scale_factors</span><span class="op">(</span><span class="va">SRP009615_meta</span>, by <span class="op">=</span> <span class="st">"mapped_reads"</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  SRR387777  SRR387778  SRR387779  SRR387780  SRR389079  SRR389080  SRR389081 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.11235750 0.09725162 0.08624279 0.10768442 0.06657889 0.09877155 0.14878642 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  SRR389082  SRR389083  SRR389084  SRR389077  SRR389078 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.18697660 0.14237100 0.14600002 0.12904391 0.11425062 </span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Download the metadata for DRP000499, a paired-end study</span></span></span>
<span class="r-in"><span><span class="va">DRP000499_meta</span> <span class="op">&lt;-</span> <span class="fu"><a href="read_metadata.html">read_metadata</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    metadata_files <span class="op">=</span> <span class="fu"><a href="file_retrieve.html">file_retrieve</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>        <span class="fu"><a href="locate_url.html">locate_url</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>            <span class="st">"DRP000499"</span>,</span></span>
<span class="r-in"><span>            <span class="st">"data_sources/sra"</span>,</span></span>
<span class="r-in"><span>        <span class="op">)</span></span></span>
<span class="r-in"><span>    <span class="op">)</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:17 caching file sra.sra.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:17 caching file sra.recount_project.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:17 caching file sra.recount_qc.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:18 caching file sra.recount_seq_qc.DRP000499.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2022-11-01 21:17:18 caching file sra.recount_pred.DRP000499.MD.gz.</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Compute the scaling factors</span></span></span>
<span class="r-in"><span><span class="fu">compute_scale_factors</span><span class="op">(</span><span class="va">DRP000499_meta</span>, by <span class="op">=</span> <span class="st">"auc"</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001622   DRR001623   DRR001624   DRR001625   DRR001626   DRR001627 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.022680966 0.032187945 0.021844235         Inf 0.034138563 0.022909709 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001628   DRR001629   DRR001630   DRR001631   DRR001632   DRR001633 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.005089376 0.011296977 0.009467943 0.007160352 0.005208585 0.012453028 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001634   DRR001635   DRR001636   DRR001637   DRR001638   DRR001639 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.007230697 0.008044157 0.007327219 0.005505240 0.008029237 0.009597381 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001640   DRR001641   DRR001642 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.013427737 0.014480916 0.010110987 </span>
<span class="r-in"><span><span class="fu">compute_scale_factors</span><span class="op">(</span><span class="va">DRP000499_meta</span>, by <span class="op">=</span> <span class="st">"mapped_reads"</span><span class="op">)</span></span></span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>is_paired_end(): Looks like some samples failed to align and will return NA.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001622   DRR001623   DRR001624   DRR001625   DRR001626   DRR001627 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.029459016 0.042985428 0.028993649          NA 0.045477396 0.030802276 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001628   DRR001629   DRR001630   DRR001631   DRR001632   DRR001633 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.007368791 0.018508532 0.011344073 0.009351792 0.006493370 0.014206925 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001634   DRR001635   DRR001636   DRR001637   DRR001638   DRR001639 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.009455928 0.010047395 0.008369215 0.007094889 0.009827127 0.011310495 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   DRR001640   DRR001641   DRR001642 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 0.015537455 0.016555494 0.012514436 </span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## You can compare the factors against those from recount::scale_counts()</span></span></span>
<span class="r-in"><span><span class="co">## from the recount2 project which used a different RNA-seq aligner</span></span></span>
<span class="r-in"><span><span class="co">## If needed, install recount, the R/Bioconductor package for recount2:</span></span></span>
<span class="r-in"><span><span class="co"># BiocManager::install("recount")</span></span></span>
<span class="r-in"><span><span class="va">recount2_factors</span> <span class="op">&lt;-</span> <span class="fu">recount</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/recount/man/scale_counts.html" class="external-link">scale_counts</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="fu">recount</span><span class="fu">::</span><span class="va"><a href="https://rdrr.io/pkg/recount/man/rse_gene_SRP009615.html" class="external-link">rse_gene_SRP009615</a></span>,</span></span>
<span class="r-in"><span>    by <span class="op">=</span> <span class="st">"auc"</span>, factor_only <span class="op">=</span> <span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">recount3_factors</span> <span class="op">&lt;-</span> <span class="fu">compute_scale_factors</span><span class="op">(</span><span class="va">SRP009615_meta</span>, by <span class="op">=</span> <span class="st">"auc"</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">recount_factors</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/data.frame.html" class="external-link">data.frame</a></span><span class="op">(</span></span></span>
<span class="r-in"><span>    recount2 <span class="op">=</span> <span class="va">recount2_factors</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/order.html" class="external-link">order</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">recount2_factors</span><span class="op">)</span><span class="op">)</span><span class="op">]</span>,</span></span>
<span class="r-in"><span>    recount3 <span class="op">=</span> <span class="va">recount3_factors</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/order.html" class="external-link">order</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="va">recount3_factors</span><span class="op">)</span><span class="op">)</span><span class="op">]</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">recount2</span> <span class="op">~</span> <span class="va">recount3</span>, data <span class="op">=</span> <span class="va">recount_factors</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/graphics/abline.html" class="external-link">abline</a></span><span class="op">(</span>a <span class="op">=</span> <span class="fl">0</span>, b <span class="op">=</span> <span class="fl">1</span>, col <span class="op">=</span> <span class="st">"purple"</span>, lwd <span class="op">=</span> <span class="fl">2</span>, lty <span class="op">=</span> <span class="fl">2</span><span class="op">)</span></span></span>
<span class="r-plt img"><img src="compute_scale_factors-1.png" alt="" width="700" height="433"></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

      </footer></div>

  


  

  </body></html>

