<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Internal function for creating a recount3 RangedSummarizedExperiment object — create_rse_manual • recount3</title><!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png"><link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png"><link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png"><link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png"><link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png"><link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png"><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet"><meta property="og:title" content="Internal function for creating a recount3 RangedSummarizedExperiment object — create_rse_manual"><meta property="og:description" content="This function is used internally by create_rse() to construct a recount3
RangedSummarizedExperiment-class
object that contains the base-pair coverage counts at the gene or exon
feature level for a given annotation."><meta property="og:image" content="/logo.png"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.11.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/LieberInstitute/recount3/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Internal function for creating a recount3 RangedSummarizedExperiment object</h1>
    <small class="dont-index">Source: <a href="https://github.com/LieberInstitute/recount3/blob/HEAD/R/create_rse_manual.R" class="external-link"><code>R/create_rse_manual.R</code></a></small>
    <div class="hidden name"><code>create_rse_manual.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function is used internally by <code><a href="create_rse.html">create_rse()</a></code> to construct a <code>recount3</code>
<a href="https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html" class="external-link">RangedSummarizedExperiment-class</a>
object that contains the base-pair coverage counts at the <code>gene</code> or <code>exon</code>
feature level for a given annotation.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">create_rse_manual</span><span class="op">(</span></span>
<span>  <span class="va">project</span>,</span>
<span>  project_home <span class="op">=</span> <span class="fu"><a href="project_homes.html">project_homes</a></span><span class="op">(</span>organism <span class="op">=</span> <span class="va">organism</span>, recount3_url <span class="op">=</span> <span class="va">recount3_url</span><span class="op">)</span>,</span>
<span>  type <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"gene"</span>, <span class="st">"exon"</span>, <span class="st">"jxn"</span><span class="op">)</span>,</span>
<span>  organism <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"human"</span>, <span class="st">"mouse"</span><span class="op">)</span>,</span>
<span>  annotation <span class="op">=</span> <span class="fu"><a href="annotation_options.html">annotation_options</a></span><span class="op">(</span><span class="va">organism</span><span class="op">)</span>,</span>
<span>  bfc <span class="op">=</span> <span class="fu"><a href="recount3_cache.html">recount3_cache</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span>  jxn_format <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"ALL"</span>, <span class="st">"UNIQUE"</span><span class="op">)</span>,</span>
<span>  recount3_url <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/options.html" class="external-link">getOption</a></span><span class="op">(</span><span class="st">"recount3_url"</span>, <span class="st">"http://duffel.rail.bio/recount3"</span><span class="op">)</span>,</span>
<span>  verbose <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/options.html" class="external-link">getOption</a></span><span class="op">(</span><span class="st">"recount3_verbose"</span>, <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>
    <dl><dt>project</dt>
<dd><p>A <code>character(1)</code> with the ID for a given study.</p></dd>


<dt>project_home</dt>
<dd><p>A <code>character(1)</code> with the home directory for the
<code>project</code>. You can find these using <code><a href="project_homes.html">project_homes()</a></code>.</p></dd>


<dt>type</dt>
<dd><p>A <code>character(1)</code> specifying whether you want to access gene,
exon, or exon-exon junction counts.</p></dd>


<dt>organism</dt>
<dd><p>A <code>character(1)</code> specifying which organism you want to
download data from. Supported options are <code>"human"</code> or <code>"mouse"</code>.</p></dd>


<dt>annotation</dt>
<dd><p>A <code>character(1)</code> specifying which annotation you want to
download. Only used when <code>type</code> is either <code>gene</code> or <code>exon</code>.</p></dd>


<dt>bfc</dt>
<dd><p>A <a href="https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html" class="external-link">BiocFileCache-class</a>
object where the files will be cached to, typically created by
<code><a href="recount3_cache.html">recount3_cache()</a></code>.</p></dd>


<dt>jxn_format</dt>
<dd><p>A <code>character(1)</code> specifying whether the exon-exon junction
files are derived from all the reads (<code>ALL</code>) or only the uniquely mapping
read counts (<code>UNIQUE</code>). Note that <code>UNIQUE</code> is only available for some
projects: GTEx and TCGA for human.</p></dd>


<dt>recount3_url</dt>
<dd><p>A <code>character(1)</code> specifying the home URL for <code>recount3</code>
or a local directory where you have mirrored <code>recount3</code>. Defaults to the
load balancer <a href="http://duffel.rail.bio/recount3" class="external-link">http://duffel.rail.bio/recount3</a>, but can also be
<a href="https://recount-opendata.s3.amazonaws.com/recount3/release" class="external-link">https://recount-opendata.s3.amazonaws.com/recount3/release</a> from
<a href="https://registry.opendata.aws/recount/" class="external-link">https://registry.opendata.aws/recount/</a> or SciServer datascope from
IDIES at JHU <a href="https://sciserver.org/public-data/recount3/data" class="external-link">https://sciserver.org/public-data/recount3/data</a>. You can
set the R option <code>recount3_url</code> (for example in your <code>.Rprofile</code>) if
you have a favorite mirror.</p></dd>


<dt>verbose</dt>
<dd><p>A <code>logical(1)</code> indicating whether to show messages with
updates.</p></dd>

</dl></div>
    <div id="value">
    <h2>Value</h2>
    

<p>A
<a href="https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html" class="external-link">RangedSummarizedExperiment-class</a></p>


<p>object.</p>
    </div>
    <div id="references">
    <h2>References</h2>
    <p><a href="https://doi.org/10.12688/f1000research.12223.1" class="external-link">https://doi.org/10.12688/f1000research.12223.1</a> for details on the
base-pair coverage counts used in recount2 and recount3.</p>
    </div>
    <div id="see-also">
    <h2>See also</h2>
    <div class="dont-index"><p>Other internal functions for accessing the recount3 data: 
<code><a href="annotation_ext.html">annotation_ext</a>()</code>,
<code><a href="file_retrieve.html">file_retrieve</a>()</code>,
<code><a href="locate_url_ann.html">locate_url_ann</a>()</code>,
<code><a href="locate_url.html">locate_url</a>()</code>,
<code><a href="project_homes.html">project_homes</a>()</code>,
<code><a href="read_counts.html">read_counts</a>()</code>,
<code><a href="read_metadata.html">read_metadata</a>()</code></p></div>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Unlike create_rse(), here we create an RSE object by</span></span></span>
<span class="r-in"><span><span class="co">## fully specifying all the arguments for locating this study</span></span></span>
<span class="r-in"><span><span class="va">rse_gene_SRP009615_manual</span> <span class="op">&lt;-</span> <span class="fu">create_rse_manual</span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"SRP009615"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"data_sources/sra"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:56.081234 downloading and reading the metadata.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:56.2317 caching file sra.sra.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:56.438292 caching file sra.recount_project.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:56.637406 caching file sra.recount_qc.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:56.838457 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:57.034173 caching file sra.recount_pred.SRP009615.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:57.441103 downloading and reading the feature information.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:57.584768 caching file human.gene_sums.G026.gtf.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:58.074906 downloading and reading the counts: 12 samples across 63856 features.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:58.222507 caching file sra.gene_sums.SRP009615.G026.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:58.401081 constructing the RangedSummarizedExperiment (rse) object.</span>
<span class="r-in"><span><span class="va">rse_gene_SRP009615_manual</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> class: RangedSummarizedExperiment </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> dim: 63856 12 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> metadata(8): time_created recount3_version ... annotation recount3_url</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> assays(1): raw_counts</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rowData names(10): source type ... havana_gene tag</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colData names(175): rail_id external_id ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   recount_pred.curated.cell_line BigWigURL</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Check how much memory this RSE object uses</span></span></span>
<span class="r-in"><span><span class="fu">pryr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/pryr/man/object_size.html" class="external-link">object_size</a></span><span class="op">(</span><span class="va">rse_gene_SRP009615_manual</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 24.81 MB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Test with a collection that has a single sample</span></span></span>
<span class="r-in"><span><span class="co">## NOTE: this requires loading the full data for this study when</span></span></span>
<span class="r-in"><span><span class="co">## creating the RSE object</span></span></span>
<span class="r-in"><span><span class="va">rse_gene_ERP110066_collection_manual</span> <span class="op">&lt;-</span> <span class="fu">create_rse_manual</span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"ERP110066"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"collections/geuvadis_smartseq"</span>,</span></span>
<span class="r-in"><span>    recount3_url <span class="op">=</span> <span class="st">"http://snaptron.cs.jhu.edu/data/temp/recount3"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:58.54771 downloading and reading the metadata.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:58.740328 caching file geuvadis_smartseq.recount_project.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:58.844435 caching file sra.sra.ERP110066.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:58.936969 caching file sra.recount_project.ERP110066.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:59.034626 caching file sra.recount_qc.ERP110066.MD.gz.</span>
<span class="r-wrn co"><span class="r-pr">#&gt;</span> <span class="warning">Warning: </span>The 'url' &lt;http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/metadata/66/ERP110066/sra.recount_seq_qc.ERP110066.MD.gz&gt; does not exist or is not available.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:59.159735 caching file sra.recount_pred.ERP110066.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:59.261156 caching file geuvadis_smartseq.custom.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:59.564211 downloading and reading the feature information.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:43:59.591634 caching file human.gene_sums.G026.gtf.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:00.118188 downloading and reading the counts: 1 sample across 63856 features.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:00.145744 caching file sra.gene_sums.ERP110066.G026.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:02.620963 constructing the RangedSummarizedExperiment (rse) object.</span>
<span class="r-in"><span><span class="va">rse_gene_ERP110066_collection_manual</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> class: RangedSummarizedExperiment </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> dim: 63856 1 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> metadata(8): time_created recount3_version ... annotation recount3_url</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> assays(1): raw_counts</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rowData names(10): source type ... havana_gene tag</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colnames(1): ERR2713106</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colData names(162): rail_id external_id ... custom.sequencing_type</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   BigWigURL</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Check how much memory this RSE object uses</span></span></span>
<span class="r-in"><span><span class="fu">pryr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/pryr/man/object_size.html" class="external-link">object_size</a></span><span class="op">(</span><span class="va">rse_gene_ERP110066_collection_manual</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 19.16 MB</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Mouse example</span></span></span>
<span class="r-in"><span><span class="va">rse_gene_DRP002367_manual</span> <span class="op">&lt;-</span> <span class="fu">create_rse_manual</span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"DRP002367"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"data_sources/sra"</span>,</span></span>
<span class="r-in"><span>    organism <span class="op">=</span> <span class="st">"mouse"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:02.767445 downloading and reading the metadata.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:03.41801 caching file sra.sra.DRP002367.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:03.630164 caching file sra.recount_project.DRP002367.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:03.848559 caching file sra.recount_qc.DRP002367.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:04.078264 caching file sra.recount_seq_qc.DRP002367.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:04.309397 caching file sra.recount_pred.DRP002367.MD.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:04.400879 downloading and reading the feature information.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:04.554811 caching file mouse.gene_sums.M023.gtf.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:06.641292 downloading and reading the counts: 4 samples across 55421 features.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:06.775444 caching file sra.gene_sums.DRP002367.M023.gz.</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> 2023-05-06 23:44:06.914767 constructing the RangedSummarizedExperiment (rse) object.</span>
<span class="r-in"><span><span class="va">rse_gene_DRP002367_manual</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> class: RangedSummarizedExperiment </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> dim: 55421 4 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> metadata(8): time_created recount3_version ... annotation recount3_url</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> assays(1): raw_counts</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rownames(55421): ENSMUSG00000079800.2 ENSMUSG00000095092.1 ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   ENSMUSG00000096850.1 ENSMUSG00000099871.1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rowData names(11): source type ... havana_gene tag</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colnames(4): DRR023307 DRR023308 DRR023309 DRR023310</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colData names(177): rail_id external_id ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   recount_pred.curated.cell_line BigWigURL</span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Information about how this RSE was made</span></span></span>
<span class="r-in"><span><span class="fu">metadata</span><span class="op">(</span><span class="va">rse_gene_DRP002367_manual</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $time_created</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "2023-05-06 23:44:06 UTC"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $recount3_version</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           package ondiskversion loadedversion                        path</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> recount3 recount3        1.11.1        1.11.1 /__w/_temp/Library/recount3</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                           loadedpath attached is_base       date       source</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> recount3 /__w/_temp/Library/recount3     TRUE   FALSE 2023-05-06 Bioconductor</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>          md5ok            library</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> recount3    NA /__w/_temp/Library</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $project</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "DRP002367"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $project_home</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "data_sources/sra"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $type</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "gene"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $organism</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "mouse"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $annotation</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "gencode_v23"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $recount3_url</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "http://duffel.rail.bio/recount3"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Test with a collection that has one sample, at the exon level</span></span></span>
<span class="r-in"><span><span class="co">## NOTE: this requires loading the full data for this study (nearly 6GB!)</span></span></span>
<span class="r-in"><span><span class="kw">if</span> <span class="op">(</span><span class="cn">FALSE</span><span class="op">)</span> <span class="op">{</span></span></span>
<span class="r-in"><span><span class="va">rse_exon_ERP110066_collection_manual</span> <span class="op">&lt;-</span> <span class="fu">create_rse_manual</span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"ERP110066"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"collections/geuvadis_smartseq"</span>,</span></span>
<span class="r-in"><span>    type <span class="op">=</span> <span class="st">"exon"</span>,</span></span>
<span class="r-in"><span>    recount3_url <span class="op">=</span> <span class="st">"http://snaptron.cs.jhu.edu/data/temp/recount3"</span></span></span>
<span class="r-in"><span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">rse_exon_ERP110066_collection_manual</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Check how much memory this RSE object uses</span></span></span>
<span class="r-in"><span><span class="fu">pryr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/pryr/man/object_size.html" class="external-link">object_size</a></span><span class="op">(</span><span class="va">rse_exon_ERP110066_collection_manual</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="co"># 409 MB</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Test with a collection that has one sample, at the junction level</span></span></span>
<span class="r-in"><span><span class="co">## NOTE: this requires loading the full data for this study</span></span></span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/system.time.html" class="external-link">system.time</a></span><span class="op">(</span><span class="va">rse_jxn_ERP110066_collection_manual</span> <span class="op">&lt;-</span> <span class="fu">create_rse_manual</span><span class="op">(</span></span></span>
<span class="r-in"><span>    <span class="st">"ERP110066"</span>,</span></span>
<span class="r-in"><span>    <span class="st">"collections/geuvadis_smartseq"</span>,</span></span>
<span class="r-in"><span>    type <span class="op">=</span> <span class="st">"jxn"</span>,</span></span>
<span class="r-in"><span>    recount3_url <span class="op">=</span> <span class="st">"http://snaptron.cs.jhu.edu/data/temp/recount3"</span></span></span>
<span class="r-in"><span><span class="op">)</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">rse_jxn_ERP110066_collection_manual</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="co">## Check how much memory this RSE object uses</span></span></span>
<span class="r-in"><span><span class="co">## NOTE: this doesn't run since 2 files are missing on the test site!</span></span></span>
<span class="r-in"><span><span class="fu">pryr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/pryr/man/object_size.html" class="external-link">object_size</a></span><span class="op">(</span><span class="va">rse_jxn_ERP110066_collection_manual</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="op">}</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="kw">if</span> <span class="op">(</span><span class="cn">FALSE</span><span class="op">)</span> <span class="op">{</span></span></span>
<span class="r-in"><span><span class="co">## For testing and debugging</span></span></span>
<span class="r-in"><span><span class="va">project</span> <span class="op">&lt;-</span> <span class="st">"ERP110066"</span></span></span>
<span class="r-in"><span><span class="va">project_home</span> <span class="op">&lt;-</span> <span class="st">"collections/geuvadis_smartseq"</span></span></span>
<span class="r-in"><span></span></span>
<span class="r-in"><span><span class="va">project</span> <span class="op">&lt;-</span> <span class="st">"SRP009615"</span></span></span>
<span class="r-in"><span><span class="va">project_home</span> <span class="op">&lt;-</span> <span class="st">"data_sources/sra"</span></span></span>
<span class="r-in"><span><span class="va">type</span> <span class="op">&lt;-</span> <span class="st">"gene"</span></span></span>
<span class="r-in"><span><span class="va">organism</span> <span class="op">&lt;-</span> <span class="st">"human"</span></span></span>
<span class="r-in"><span><span class="va">annotation</span> <span class="op">&lt;-</span> <span class="st">"gencode_v26"</span></span></span>
<span class="r-in"><span><span class="va">jxn_format</span> <span class="op">&lt;-</span> <span class="st">"ALL"</span></span></span>
<span class="r-in"><span><span class="va">bfc</span> <span class="op">&lt;-</span> <span class="fu"><a href="recount3_cache.html">recount3_cache</a></span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="va">recount3_url</span> <span class="op">&lt;-</span> <span class="st">"http://idies.jhu.edu/recount3/data"</span></span></span>
<span class="r-in"><span><span class="va">verbose</span> <span class="op">&lt;-</span> <span class="cn">TRUE</span></span></span>
<span class="r-in"><span><span class="op">}</span></span></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

      </footer></div>

  


  

  </body></html>

