<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Internal function for creating a recount3 RangedSummarizedExperiment object — create_rse_manual • recount3</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Internal function for creating a recount3 RangedSummarizedExperiment object — create_rse_manual" />
<meta property="og:description" content="This function is used internally by create_rse() to construct a recount3
RangedSummarizedExperiment-class
object that contains the base-pair coverage counts at the gene or exon
feature level for a given annotation." />
<meta property="og:image" content="/logo.png" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LieberInstitute/recount3/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Internal function for creating a recount3 RangedSummarizedExperiment object</h1>
    <small class="dont-index">Source: <a href='https://github.com/LieberInstitute/recount3/blob/master/R/create_rse_manual.R'><code>R/create_rse_manual.R</code></a></small>
    <div class="hidden name"><code>create_rse_manual.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function is used internally by <code>create_rse()</code> to construct a <code>recount3</code>
<a href='https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html'>RangedSummarizedExperiment-class</a>
object that contains the base-pair coverage counts at the <code>gene</code> or <code>exon</code>
feature level for a given annotation.</p>
    </div>

    <pre class="usage"><span class='fu'>create_rse_manual</span>(
  <span class='kw'>project</span>,
  project_home = <span class='fu'>project_home_available</span>(organism = <span class='kw'>organism</span>, recount3_url =
    <span class='kw'>recount3_url</span>, bfc = <span class='kw'>bfc</span>),
  type = <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span>(<span class='st'>"gene"</span>, <span class='st'>"exon"</span>, <span class='st'>"jxn"</span>),
  organism = <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span>(<span class='st'>"human"</span>, <span class='st'>"mouse"</span>),
  annotation = <span class='fu'>annotation_options</span>(<span class='kw'>organism</span>),
  bfc = <span class='kw'>BiocFileCache</span>::<span class='fu'><a href='https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html'>BiocFileCache</a></span>(),
  recount3_url = <span class='st'>"http://snaptron.cs.jhu.edu/data/temp/recount3"</span>
)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>project</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> with the ID for a given study.</p></td>
    </tr>
    <tr>
      <th>project_home</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> with the home directory for the
<code>project</code>. You can find these using <code>project_home_available()</code>.</p></td>
    </tr>
    <tr>
      <th>type</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying whether you want to access gene,
exon, or exon-exon junction counts.</p></td>
    </tr>
    <tr>
      <th>organism</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying which organism you want to
download data from. Supported options are <code>"human"</code> or <code>"mouse"</code>.</p></td>
    </tr>
    <tr>
      <th>annotation</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying which annotation you want to
download. Only used when <code>type</code> is either <code>gene</code> or <code>exon</code>.</p></td>
    </tr>
    <tr>
      <th>bfc</th>
      <td><p>A <a href='https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html'>BiocFileCache-class</a>
object where the files will be cached to.</p></td>
    </tr>
    <tr>
      <th>recount3_url</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying the home URL for <code>recount3</code>
or a local directory where you have mirrored <code>recount3</code>.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A
<a href='https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html'>RangedSummarizedExperiment-class</a>
object.</p>
    <h2 class="hasAnchor" id="references"><a class="anchor" href="#references"></a>References</h2>

    <p><a href='https://doi.org/10.12688/f1000research.12223.1'>https://doi.org/10.12688/f1000research.12223.1</a> for details on the
base-pair coverage counts used in recount2 and recount3.</p>
    <h2 class="hasAnchor" id="see-also"><a class="anchor" href="#see-also"></a>See also</h2>

    <div class='dont-index'><p>Other internal functions for accessing the recount3 data: 
<code>annotation_ext()</code>,
<code>file_locate_url_annotation()</code>,
<code>file_locate_url()</code>,
<code>file_retrieve()</code>,
<code>project_home_available()</code>,
<code>read_counts()</code>,
<code>read_metadata()</code></p></div>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='kw'>rse_gene_ERP110066_manual</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse_manual</span>(
    <span class='st'>"ERP110066"</span>,
    <span class='st'>"data_sources/sra"</span>
)</div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:05 downloading and reading the metadata</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.sra.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_project.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_qc.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:07 downloading and reading the feature information</span></div><div class='output co'>#&gt; <span class='message'>Caching file human.gene_sums.G026.gtf.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:08 downloading and reading the counts: 1024 samples across 63856 features.</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.gene_sums.ERP110066.G026.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:11 construcing the RangedSummarizedExperiment (rse) object</span></div><div class='input'><span class='kw'>rse_gene_ERP110066_manual</span></div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 63856 1024 
#&gt; metadata(0):
#&gt; assays(1): counts
#&gt; rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ...
#&gt;   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
#&gt; rowData names(10): source type ... havana_gene tag
#&gt; colnames(1024): ERR2713604 ERR2713605 ... ERR2713998 ERR2713999
#&gt; colData names(139): rail_id run_acc ...
#&gt;   recount_qc.junction_avg_coverage BigWigURL</div><div class='input'>
<span class='kw'>rse_gene_ERP110066_collection_manual</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse_manual</span>(
    <span class='st'>"ERP110066"</span>,
    <span class='st'>"collections/geuvadis_smartseq"</span>
)</div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:12 downloading and reading the metadata</span></div><div class='output co'>#&gt; <span class='message'>Caching file geuvadis_smartseq.recount_project.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/collections/geuvadis_smartseq/metadata/geuvadis_smartseq.recount_project.gz'</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.sra.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_project.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_qc.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file geuvadis_smartseq.custom.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/collections/geuvadis_smartseq/metadata/geuvadis_smartseq.custom.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:15 downloading and reading the feature information</span></div><div class='output co'>#&gt; <span class='message'>Caching file human.gene_sums.G026.gtf.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:15 downloading and reading the counts: 1 sample across 63856 features.</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.gene_sums.ERP110066.G026.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:18 construcing the RangedSummarizedExperiment (rse) object</span></div><div class='input'><span class='kw'>rse_gene_ERP110066_collection_manual</span></div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 63856 1 
#&gt; metadata(0):
#&gt; assays(1): counts
#&gt; rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ...
#&gt;   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
#&gt; rowData names(10): source type ... havana_gene tag
#&gt; colnames(1): ERR2713106
#&gt; colData names(140): rail_id run_acc ... custom.sequencing_type
#&gt;   BigWigURL</div><div class='input'>

<span class='kw'>rse_exon_ERP110066_collection_manual</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse_manual</span>(
    <span class='st'>"ERP110066"</span>,
    <span class='st'>"collections/geuvadis_smartseq"</span>,
    type = <span class='st'>"exon"</span>
)</div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:18 downloading and reading the metadata</span></div><div class='output co'>#&gt; <span class='message'>Caching file geuvadis_smartseq.recount_project.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.sra.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_project.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_qc.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file geuvadis_smartseq.custom.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:21 downloading and reading the feature information</span></div><div class='output co'>#&gt; <span class='message'>Caching file human.exon_sums.G026.gtf.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/annotations/exon_sums/human.exon_sums.G026.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:26:43 downloading and reading the counts: 1 sample across 1299686 features.</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.exon_sums.ERP110066.G026.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/exon_sums/66/ERP110066/sra.exon_sums.ERP110066.G026.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:27:44 construcing the RangedSummarizedExperiment (rse) object</span></div><div class='input'><span class='kw'>rse_exon_ERP110066_collection_manual</span></div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 1299686 1 
#&gt; metadata(0):
#&gt; assays(1): counts
#&gt; rownames(1299686): GL000009.2|56140|58376|- GL000194.1|53594|54832|-
#&gt;   ... chrY|57213880|57213964|- chrY|57214350|57214397|-
#&gt; rowData names(21): source type ... ont ccdsid
#&gt; colnames(1): ERR2713106
#&gt; colData names(140): rail_id run_acc ... custom.sequencing_type
#&gt;   BigWigURL</div><div class='input'>
<span class='fu'><a href='https://rdrr.io/r/base/system.time.html'>system.time</a></span>(<span class='kw'>rse_jxn_ERP110066_collection_manual</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse_manual</span>(
    <span class='st'>"ERP110066"</span>,
    <span class='st'>"collections/geuvadis_smartseq"</span>,
    type = <span class='st'>"jxn"</span>
))</div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:27:46 downloading and reading the metadata</span></div><div class='output co'>#&gt; <span class='message'>Caching file geuvadis_smartseq.recount_project.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.sra.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_project.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_qc.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file geuvadis_smartseq.custom.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:27:49 downloading and reading the feature information</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.junctions.ERP110066.RR.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/junctions/66/ERP110066/sra.junctions.ERP110066.RR.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:27:55 downloading and reading the counts: 1 sample across 585473 features.</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.junctions.ERP110066.MM.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/junctions/66/ERP110066/sra.junctions.ERP110066.MM.gz'</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.sra.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_project.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_qc.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-06-23 14:28:20 construcing the RangedSummarizedExperiment (rse) object</span></div><div class='output co'>#&gt;    user  system elapsed 
#&gt;  31.708   1.978  34.533 </div><div class='input'><span class='kw'>rse_jxn_ERP110066_collection_manual</span></div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 585473 1 
#&gt; metadata(0):
#&gt; assays(1): counts
#&gt; rownames: NULL
#&gt; rowData names(7): snaptron_id length ... left_annotated right_annotated
#&gt; colnames(1): ERR2713106
#&gt; colData names(140): rail_id run_acc ... custom.sequencing_type
#&gt;   BigWigURL</div><div class='input'><span class='co'>if</span> (<span class='fl'>FALSE</span>) {
<span class='kw'>project</span> <span class='op'>&lt;-</span> <span class='st'>"ERP110066"</span>
<span class='kw'>project_home</span> <span class='op'>&lt;-</span> <span class='st'>"collections/geuvadis_smartseq"</span>
<span class='kw'>type</span> <span class='op'>&lt;-</span> <span class='st'>"jxn"</span>
<span class='kw'>organism</span> <span class='op'>&lt;-</span> <span class='st'>"human"</span>
<span class='kw'>annotation</span> <span class='op'>&lt;-</span> <span class='st'>"gencode_v26"</span>
<span class='kw'>bfc</span> <span class='op'>&lt;-</span> <span class='kw'>BiocFileCache</span>::<span class='fu'><a href='https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html'>BiocFileCache</a></span>()
<span class='kw'>recount3_url</span> <span class='op'>&lt;-</span> <span class='st'>"http://snaptron.cs.jhu.edu/data/temp/recount3"</span>
}</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.9000.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


