<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Create a recount3 RangedSummarizedExperiment gene or exon object — create_rse • recount3</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Create a recount3 RangedSummarizedExperiment gene or exon object — create_rse" />
<meta property="og:description" content="Once you have identified a project you want to work with, you can use this
function to construct a recount3
RangedSummarizedExperiment-class
(RSE) object at the gene or exon expression feature level. This function will
retrieve the data, cache it, then assemble the RSE object." />
<meta property="og:image" content="/logo.png" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LieberInstitute/recount3/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Create a recount3 RangedSummarizedExperiment gene or exon object</h1>
    <small class="dont-index">Source: <a href='https://github.com/LieberInstitute/recount3/blob/master/R/create_rse.R'><code>R/create_rse.R</code></a></small>
    <div class="hidden name"><code>create_rse.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Once you have identified a project you want to work with, you can use this
function to construct a <code>recount3</code>
<a href='https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html'>RangedSummarizedExperiment-class</a>
(RSE) object at the gene or exon expression feature level. This function will
retrieve the data, cache it, then assemble the RSE object.</p>
    </div>

    <pre class="usage"><span class='fu'>create_rse</span>(
  <span class='kw'>project_info</span>,
  type = <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span>(<span class='st'>"gene"</span>, <span class='st'>"exon"</span>, <span class='st'>"jxn"</span>),
  annotation = <span class='fu'><a href='annotation_options.html'>annotation_options</a></span>(<span class='kw'>project_info</span><span class='op'>$</span><span class='kw'>organism</span>),
  bfc = <span class='kw'>BiocFileCache</span>::<span class='fu'><a href='https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html'>BiocFileCache</a></span>(),
  jxn_format = <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span>(<span class='st'>"ALL"</span>, <span class='st'>"UNIQUE"</span>),
  recount3_url = <span class='st'>"http://idies.jhu.edu/recount3/data"</span>
)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>project_info</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/data.frame.html'>data.frame()</a></code> with one row that contains the
information for the project you are interested in. You can find which
project to work on using <code><a href='available_projects.html'>available_projects()</a></code> or <code><a href='recount3_hub.html'>recount3_hub()</a></code>..</p></td>
    </tr>
    <tr>
      <th>type</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying whether you want to access gene,
exon, or exon-exon junction counts.</p></td>
    </tr>
    <tr>
      <th>annotation</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying which annotation you want to
download. Only used when <code>type</code> is either <code>gene</code> or <code>exon</code>.</p></td>
    </tr>
    <tr>
      <th>bfc</th>
      <td><p>A <a href='https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html'>BiocFileCache-class</a>
object where the files will be cached to.</p></td>
    </tr>
    <tr>
      <th>jxn_format</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying whether the exon-exon junction
files are derived from all the reads (<code>ALL</code>) or only the uniquely mapping
read counts (<code>UNIQUE</code>). Note that <code>UNIQUE</code> is only available for some
projects: GTEx and TCGA for human.</p></td>
    </tr>
    <tr>
      <th>recount3_url</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying the home URL for <code>recount3</code>
or a local directory where you have mirrored <code>recount3</code>.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A
<a href='https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html'>RangedSummarizedExperiment-class</a>
object.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='co'>## Find all available human projects</span>
<span class='kw'>human_projects</span> <span class='op'>&lt;-</span> <span class='fu'><a href='available_projects.html'>available_projects</a></span>()
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:06 caching file sra.recount_project.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:06 caching file gtex.recount_project.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:06 caching file tcga.recount_project.MD.gz.</span></div><div class='input'>
<span class='co'>## Find the project you are interested in</span>
<span class='kw'>proj_info</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/subset.html'>subset</a></span>(
    <span class='kw'>human_projects</span>,
    <span class='kw'>project</span> <span class='op'>==</span> <span class='st'>"SRP009615"</span> <span class='op'>&amp;</span> <span class='kw'>project_type</span> <span class='op'>==</span> <span class='st'>"data_sources"</span>
)

<span class='co'>## Create a RSE object at the gene level</span>
<span class='kw'>rse_gene_SRP009615</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(<span class='kw'>proj_info</span>)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:10 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:10 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.sra.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:11 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_project.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:11 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_qc.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:11 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_seq_qc.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:12 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/metadata/15/SRP009615/sra.recount_pred.SRP009615.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:12 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:12 caching file human.gene_sums.G026.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/gene_sums/human.gene_sums.G026.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:13 downloading and reading the counts: 12 samples across 63856 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:13 caching file sra.gene_sums.SRP009615.G026.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/15/SRP009615/sra.gene_sums.SRP009615.G026.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:14 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'>
<span class='co'>## Explore the resulting RSE gene object</span>
<span class='kw'>rse_gene_SRP009615</span>
</div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 63856 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ...
#&gt;   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
#&gt; rowData names(10): source type ... havana_gene tag
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL</div><div class='input'>
<span class='co'>## Information about how this RSE object was made</span>
<span class='fu'>metadata</span>(<span class='kw'>rse_gene_SRP009615</span>)
</div><div class='output co'>#&gt; $time_created
#&gt; [1] "2020-09-30 01:41:14 UTC"
#&gt; 
#&gt; $recount3_version
#&gt;           package ondiskversion loadedversion
#&gt; recount3 recount3        0.99.0        0.99.0
#&gt;                                                 path
#&gt; recount3 /usr/local/lib/R/host-site-library/recount3
#&gt;                                           loadedpath attached is_base
#&gt; recount3 /usr/local/lib/R/host-site-library/recount3     TRUE   FALSE
#&gt;                date       source md5ok                            library
#&gt; recount3 2020-09-30 Bioconductor    NA /usr/local/lib/R/host-site-library
#&gt; 
#&gt; $project
#&gt; [1] "SRP009615"
#&gt; 
#&gt; $project_home
#&gt; [1] "data_sources/sra"
#&gt; 
#&gt; $type
#&gt; [1] "gene"
#&gt; 
#&gt; $organism
#&gt; [1] "human"
#&gt; 
#&gt; $annotation
#&gt; [1] "gencode_v26"
#&gt; 
#&gt; $recount3_url
#&gt; [1] "http://idies.jhu.edu/recount3/data"
#&gt; </div><div class='input'>
<span class='co'>## Number of genes by number of samples</span>
<span class='fu'><a href='https://rdrr.io/r/base/dim.html'>dim</a></span>(<span class='kw'>rse_gene_SRP009615</span>)
</div><div class='output co'>#&gt; [1] 63856    12</div><div class='input'>
<span class='co'>## Information about the genes</span>
<span class='fu'>rowRanges</span>(<span class='kw'>rse_gene_SRP009615</span>)
</div><div class='output co'>#&gt; GRanges object with 63856 ranges and 10 metadata columns:
#&gt;                              seqnames            ranges strand |   source
#&gt;                                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;factor&gt;
#&gt;          ENSG00000278704.1 GL000009.2       56140-58376      - |  ENSEMBL
#&gt;          ENSG00000277400.1 GL000194.1      53590-115018      - |  ENSEMBL
#&gt;          ENSG00000274847.1 GL000194.1      53594-115055      - |  ENSEMBL
#&gt;          ENSG00000277428.1 GL000195.1       37434-37534      - |  ENSEMBL
#&gt;          ENSG00000276256.1 GL000195.1       42939-49164      - |  ENSEMBL
#&gt;                        ...        ...               ...    ... .      ...
#&gt;   ENSG00000124334.17_PAR_Y       chrY 57184101-57197337      + |   HAVANA
#&gt;   ENSG00000185203.12_PAR_Y       chrY 57201143-57203357      - |   HAVANA
#&gt;    ENSG00000270726.6_PAR_Y       chrY 57190738-57208756      + |   HAVANA
#&gt;   ENSG00000182484.15_PAR_Y       chrY 57207346-57212230      + |   HAVANA
#&gt;    ENSG00000227159.8_PAR_Y       chrY 57212184-57214397      - |   HAVANA
#&gt;                                type     score     phase                gene_id
#&gt;                            &lt;factor&gt; &lt;numeric&gt; &lt;integer&gt;            &lt;character&gt;
#&gt;          ENSG00000278704.1     gene      2237      &lt;NA&gt;      ENSG00000278704.1
#&gt;          ENSG00000277400.1     gene      2179      &lt;NA&gt;      ENSG00000277400.1
#&gt;          ENSG00000274847.1     gene      1599      &lt;NA&gt;      ENSG00000274847.1
#&gt;          ENSG00000277428.1     gene       101      &lt;NA&gt;      ENSG00000277428.1
#&gt;          ENSG00000276256.1     gene      2195      &lt;NA&gt;      ENSG00000276256.1
#&gt;                        ...      ...       ...       ...                    ...
#&gt;   ENSG00000124334.17_PAR_Y     gene      2504      &lt;NA&gt; ENSG00000124334.17_P..
#&gt;   ENSG00000185203.12_PAR_Y     gene      1054      &lt;NA&gt; ENSG00000185203.12_P..
#&gt;    ENSG00000270726.6_PAR_Y     gene       773      &lt;NA&gt; ENSG00000270726.6_PA..
#&gt;   ENSG00000182484.15_PAR_Y     gene      4618      &lt;NA&gt; ENSG00000182484.15_P..
#&gt;    ENSG00000227159.8_PAR_Y     gene      1306      &lt;NA&gt; ENSG00000227159.8_PA..
#&gt;                                         gene_type   gene_name       level
#&gt;                                       &lt;character&gt; &lt;character&gt; &lt;character&gt;
#&gt;          ENSG00000278704.1         protein_coding  BX004987.1           3
#&gt;          ENSG00000277400.1         protein_coding  AC145212.2           3
#&gt;          ENSG00000274847.1         protein_coding  AC145212.1           3
#&gt;          ENSG00000277428.1               misc_RNA       Y_RNA           3
#&gt;          ENSG00000276256.1         protein_coding  AC011043.1           3
#&gt;                        ...                    ...         ...         ...
#&gt;   ENSG00000124334.17_PAR_Y         protein_coding        IL9R           2
#&gt;   ENSG00000185203.12_PAR_Y              antisense      WASIR1           2
#&gt;    ENSG00000270726.6_PAR_Y   processed_transcript AJ271736.10           2
#&gt;   ENSG00000182484.15_PAR_Y transcribed_unproces..      WASH6P           2
#&gt;    ENSG00000227159.8_PAR_Y unprocessed_pseudogene    DDX11L16           2
#&gt;                                     havana_gene         tag
#&gt;                                     &lt;character&gt; &lt;character&gt;
#&gt;          ENSG00000278704.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000277400.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000274847.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000277428.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000276256.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;                        ...                  ...         ...
#&gt;   ENSG00000124334.17_PAR_Y OTTHUMG00000022720.1         PAR
#&gt;   ENSG00000185203.12_PAR_Y OTTHUMG00000022676.3         PAR
#&gt;    ENSG00000270726.6_PAR_Y OTTHUMG00000184987.2         PAR
#&gt;   ENSG00000182484.15_PAR_Y OTTHUMG00000022677.5         PAR
#&gt;    ENSG00000227159.8_PAR_Y OTTHUMG00000022678.1         PAR
#&gt;   -------
#&gt;   seqinfo: 374 sequences from an unspecified genome; no seqlengths</div><div class='input'>
<span class='co'>## Sample metadata</span>
<span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>colnames</a></span>(<span class='fu'>colData</span>(<span class='kw'>rse_gene_SRP009615</span>))
</div><div class='output co'>#&gt;   [1] "rail_id"                                                           
#&gt;   [2] "external_id"                                                       
#&gt;   [3] "study"                                                             
#&gt;   [4] "sra.sample_acc.x"                                                  
#&gt;   [5] "sra.experiment_acc"                                                
#&gt;   [6] "sra.submission_acc"                                                
#&gt;   [7] "sra.submission_center"                                             
#&gt;   [8] "sra.submission_lab"                                                
#&gt;   [9] "sra.study_title"                                                   
#&gt;  [10] "sra.study_abstract"                                                
#&gt;  [11] "sra.study_description"                                             
#&gt;  [12] "sra.experiment_title"                                              
#&gt;  [13] "sra.design_description"                                            
#&gt;  [14] "sra.sample_description"                                            
#&gt;  [15] "sra.library_name"                                                  
#&gt;  [16] "sra.library_strategy"                                              
#&gt;  [17] "sra.library_source"                                                
#&gt;  [18] "sra.library_selection"                                             
#&gt;  [19] "sra.library_layout"                                                
#&gt;  [20] "sra.paired_nominal_length"                                         
#&gt;  [21] "sra.paired_nominal_stdev"                                          
#&gt;  [22] "sra.library_construction_protocol"                                 
#&gt;  [23] "sra.platform_model"                                                
#&gt;  [24] "sra.sample_attributes"                                             
#&gt;  [25] "sra.experiment_attributes"                                         
#&gt;  [26] "sra.spot_length"                                                   
#&gt;  [27] "sra.sample_name"                                                   
#&gt;  [28] "sra.sample_title"                                                  
#&gt;  [29] "sra.sample_bases"                                                  
#&gt;  [30] "sra.sample_spots"                                                  
#&gt;  [31] "sra.run_published"                                                 
#&gt;  [32] "sra.size"                                                          
#&gt;  [33] "sra.run_total_bases"                                               
#&gt;  [34] "sra.run_total_spots"                                               
#&gt;  [35] "sra.num_reads"                                                     
#&gt;  [36] "sra.num_spots"                                                     
#&gt;  [37] "sra.read_info"                                                     
#&gt;  [38] "sra.run_alias"                                                     
#&gt;  [39] "sra.run_center_name"                                               
#&gt;  [40] "sra.run_broker_name"                                               
#&gt;  [41] "sra.run_center"                                                    
#&gt;  [42] "recount_project.project"                                           
#&gt;  [43] "recount_project.organism"                                          
#&gt;  [44] "recount_project.file_source"                                       
#&gt;  [45] "recount_project.metadata_source"                                   
#&gt;  [46] "recount_project.date_processed"                                    
#&gt;  [47] "recount_qc.aligned_reads%.chrm"                                    
#&gt;  [48] "recount_qc.aligned_reads%.chrx"                                    
#&gt;  [49] "recount_qc.aligned_reads%.chry"                                    
#&gt;  [50] "recount_qc.bc_auc.all_reads_all_bases"                             
#&gt;  [51] "recount_qc.bc_auc.all_reads_annotated_bases"                       
#&gt;  [52] "recount_qc.bc_auc.unique_reads_all_bases"                          
#&gt;  [53] "recount_qc.bc_auc.unique_reads_annotated_bases"                    
#&gt;  [54] "recount_qc.bc_auc.all_%"                                           
#&gt;  [55] "recount_qc.bc_auc.unique_%"                                        
#&gt;  [56] "recount_qc.bc_frag.count"                                          
#&gt;  [57] "recount_qc.bc_frag.kallisto_count"                                 
#&gt;  [58] "recount_qc.bc_frag.kallisto_mean_length"                           
#&gt;  [59] "recount_qc.bc_frag.mean_length"                                    
#&gt;  [60] "recount_qc.bc_frag.mode_length"                                    
#&gt;  [61] "recount_qc.bc_frag.mode_length_count"                              
#&gt;  [62] "recount_qc.exon_fc.all_%"                                          
#&gt;  [63] "recount_qc.exon_fc.unique_%"                                       
#&gt;  [64] "recount_qc.exon_fc_count_all.total"                                
#&gt;  [65] "recount_qc.exon_fc_count_all.assigned"                             
#&gt;  [66] "recount_qc.exon_fc_count_unique.total"                             
#&gt;  [67] "recount_qc.exon_fc_count_unique.assigned"                          
#&gt;  [68] "recount_qc.gene_fc.all_%"                                          
#&gt;  [69] "recount_qc.gene_fc.unique_%"                                       
#&gt;  [70] "recount_qc.gene_fc_count_all.total"                                
#&gt;  [71] "recount_qc.gene_fc_count_all.assigned"                             
#&gt;  [72] "recount_qc.gene_fc_count_unique.total"                             
#&gt;  [73] "recount_qc.gene_fc_count_unique.assigned"                          
#&gt;  [74] "recount_qc.intron_sum"                                             
#&gt;  [75] "recount_qc.intron_sum_%"                                           
#&gt;  [76] "recount_qc.star.%_of_chimeric_reads"                               
#&gt;  [77] "recount_qc.star.%_of_chimeric_reads2"                              
#&gt;  [78] "recount_qc.star.%_of_reads_mapped_to_multiple_loci"                
#&gt;  [79] "recount_qc.star.%_of_reads_mapped_to_multiple_loci2"               
#&gt;  [80] "recount_qc.star.%_of_reads_mapped_to_too_many_loci"                
#&gt;  [81] "recount_qc.star.%_of_reads_mapped_to_too_many_loci2"               
#&gt;  [82] "recount_qc.star.%_of_reads_unmapped:_other"                        
#&gt;  [83] "recount_qc.star.%_of_reads_unmapped:_other2"                       
#&gt;  [84] "recount_qc.star.%_of_reads_unmapped:_too_many_mismatches"          
#&gt;  [85] "recount_qc.star.%_of_reads_unmapped:_too_many_mismatches2"         
#&gt;  [86] "recount_qc.star.%_of_reads_unmapped:_too_short"                    
#&gt;  [87] "recount_qc.star.%_of_reads_unmapped:_too_short2"                   
#&gt;  [88] "recount_qc.star.all_mapped_reads"                                  
#&gt;  [89] "recount_qc.star.all_mapped_reads2"                                 
#&gt;  [90] "recount_qc.star.average_input_read_length"                         
#&gt;  [91] "recount_qc.star.average_input_read_length2"                        
#&gt;  [92] "recount_qc.star.average_mapped_length"                             
#&gt;  [93] "recount_qc.star.average_mapped_length2"                            
#&gt;  [94] "recount_qc.star.deletion_average_length"                           
#&gt;  [95] "recount_qc.star.deletion_average_length2"                          
#&gt;  [96] "recount_qc.star.deletion_rate_per_base"                            
#&gt;  [97] "recount_qc.star.deletion_rate_per_base2"                           
#&gt;  [98] "recount_qc.star.insertion_average_length"                          
#&gt;  [99] "recount_qc.star.insertion_average_length2"                         
#&gt; [100] "recount_qc.star.insertion_rate_per_base"                           
#&gt; [101] "recount_qc.star.insertion_rate_per_base2"                          
#&gt; [102] "recount_qc.star.mapping_speed,_million_of_reads_per_hour"          
#&gt; [103] "recount_qc.star.mapping_speed,_million_of_reads_per_hour2"         
#&gt; [104] "recount_qc.star.mismatch_rate_per_base,_%"                         
#&gt; [105] "recount_qc.star.mismatch_rate_per_base,_%2"                        
#&gt; [106] "recount_qc.star.number_of_chimeric_reads"                          
#&gt; [107] "recount_qc.star.number_of_chimeric_reads2"                         
#&gt; [108] "recount_qc.star.number_of_input_reads"                             
#&gt; [109] "recount_qc.star.number_of_input_reads2"                            
#&gt; [110] "recount_qc.star.number_of_reads_mapped_to_multiple_loci"           
#&gt; [111] "recount_qc.star.number_of_reads_mapped_to_multiple_loci2"          
#&gt; [112] "recount_qc.star.number_of_reads_mapped_to_too_many_loci"           
#&gt; [113] "recount_qc.star.number_of_reads_mapped_to_too_many_loci2"          
#&gt; [114] "recount_qc.star.number_of_reads_unmapped:_other"                   
#&gt; [115] "recount_qc.star.number_of_reads_unmapped:_other2"                  
#&gt; [116] "recount_qc.star.number_of_reads_unmapped:_too_many_mismatches"     
#&gt; [117] "recount_qc.star.number_of_reads_unmapped:_too_many_mismatches2"    
#&gt; [118] "recount_qc.star.number_of_reads_unmapped:_too_short"               
#&gt; [119] "recount_qc.star.number_of_reads_unmapped:_too_short2"              
#&gt; [120] "recount_qc.star.number_of_splices:_at/ac"                          
#&gt; [121] "recount_qc.star.number_of_splices:_at/ac2"                         
#&gt; [122] "recount_qc.star.number_of_splices:_annotated_(sjdb)"               
#&gt; [123] "recount_qc.star.number_of_splices:_annotated_(sjdb)2"              
#&gt; [124] "recount_qc.star.number_of_splices:_gc/ag"                          
#&gt; [125] "recount_qc.star.number_of_splices:_gc/ag2"                         
#&gt; [126] "recount_qc.star.number_of_splices:_gt/ag"                          
#&gt; [127] "recount_qc.star.number_of_splices:_gt/ag2"                         
#&gt; [128] "recount_qc.star.number_of_splices:_non-canonical"                  
#&gt; [129] "recount_qc.star.number_of_splices:_non-canonical2"                 
#&gt; [130] "recount_qc.star.number_of_splices:_total"                          
#&gt; [131] "recount_qc.star.number_of_splices:_total2"                         
#&gt; [132] "recount_qc.star.uniquely_mapped_reads_%"                           
#&gt; [133] "recount_qc.star.uniquely_mapped_reads_%2"                          
#&gt; [134] "recount_qc.star.uniquely_mapped_reads_number"                      
#&gt; [135] "recount_qc.star.uniquely_mapped_reads_number2"                     
#&gt; [136] "recount_qc.junction_count"                                         
#&gt; [137] "recount_qc.junction_coverage"                                      
#&gt; [138] "recount_qc.junction_avg_coverage"                                  
#&gt; [139] "recount_qc.star.number_of_input_reads_both"                        
#&gt; [140] "recount_qc.star.all_mapped_reads_both"                             
#&gt; [141] "recount_qc.star.number_of_chimeric_reads_both"                     
#&gt; [142] "recount_qc.star.number_of_reads_mapped_to_multiple_loci_both"      
#&gt; [143] "recount_qc.star.number_of_reads_mapped_to_too_many_loci_both"      
#&gt; [144] "recount_qc.star.number_of_reads_unmapped:_other_both"              
#&gt; [145] "recount_qc.star.number_of_reads_unmapped:_too_many_mismatches_both"
#&gt; [146] "recount_qc.star.number_of_reads_unmapped:_too_short_both"          
#&gt; [147] "recount_qc.star.uniquely_mapped_reads_number_both"                 
#&gt; [148] "recount_qc.star.%_mapped_reads_both"                               
#&gt; [149] "recount_qc.star.%_chimeric_reads_both"                             
#&gt; [150] "recount_qc.star.%_reads_mapped_to_multiple_loci_both"              
#&gt; [151] "recount_qc.star.%_reads_mapped_to_too_many_loci_both"              
#&gt; [152] "recount_qc.star.%_reads_unmapped:_other_both"                      
#&gt; [153] "recount_qc.star.%_reads_unmapped:_too_many_mismatches_both"        
#&gt; [154] "recount_qc.star.%_reads_unmapped:_too_short_both"                  
#&gt; [155] "recount_qc.star.uniquely_mapped_reads_%_both"                      
#&gt; [156] "recount_seq_qc.min_len"                                            
#&gt; [157] "recount_seq_qc.max_len"                                            
#&gt; [158] "recount_seq_qc.avg_len"                                            
#&gt; [159] "recount_seq_qc.#distinct_quality_values"                           
#&gt; [160] "recount_seq_qc.#bases"                                             
#&gt; [161] "recount_seq_qc.%a"                                                 
#&gt; [162] "recount_seq_qc.%c"                                                 
#&gt; [163] "recount_seq_qc.%g"                                                 
#&gt; [164] "recount_seq_qc.%t"                                                 
#&gt; [165] "recount_seq_qc.%n"                                                 
#&gt; [166] "recount_seq_qc.avgq"                                               
#&gt; [167] "recount_seq_qc.errq"                                               
#&gt; [168] "recount_pred.sample_acc.y"                                         
#&gt; [169] "recount_pred.curated.type"                                         
#&gt; [170] "recount_pred.curated.tissue"                                       
#&gt; [171] "recount_pred.pattern.predict.type"                                 
#&gt; [172] "recount_pred.pred.type"                                            
#&gt; [173] "BigWigURL"                                                         </div><div class='input'>
<span class='co'>## Check how much memory this RSE object uses</span>
<span class='kw'>pryr</span>::<span class='fu'><a href='https://rdrr.io/pkg/pryr/man/object_size.html'>object_size</a></span>(<span class='kw'>rse_gene_SRP009615</span>)
</div><div class='output co'>#&gt; <span class='message'>Registered S3 method overwritten by 'pryr':</span>
#&gt; <span class='message'>  method      from</span>
#&gt; <span class='message'>  print.bytes Rcpp</span></div><div class='output co'>#&gt; 21.7 MB</div><div class='input'>
<span class='co'>## Create an RSE object using gencode_v29 instead of gencode_v26</span>
<span class='kw'>rse_gene_SRP009615_gencode_v29</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='kw'>proj_info</span>,
    annotation = <span class='st'>"gencode_v29"</span>
)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:14 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:14 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:14 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:15 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:15 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:15 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:15 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:15 caching file human.gene_sums.G029.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/gene_sums/human.gene_sums.G029.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:17 downloading and reading the counts: 12 samples across 64837 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:17 caching file sra.gene_sums.SRP009615.G029.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/15/SRP009615/sra.gene_sums.SRP009615.G029.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:18 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'>rowRanges</span>(<span class='kw'>rse_gene_SRP009615_gencode_v29</span>)
</div><div class='output co'>#&gt; GRanges object with 64837 ranges and 10 metadata columns:
#&gt;                              seqnames            ranges strand |   source
#&gt;                                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;factor&gt;
#&gt;          ENSG00000278704.1 GL000009.2       56140-58376      - |  ENSEMBL
#&gt;          ENSG00000277400.1 GL000194.1      53590-115018      - |  ENSEMBL
#&gt;          ENSG00000274847.1 GL000194.1      53594-115055      - |  ENSEMBL
#&gt;          ENSG00000277428.1 GL000195.1       37434-37534      - |  ENSEMBL
#&gt;          ENSG00000276256.1 GL000195.1       42939-49164      - |  ENSEMBL
#&gt;                        ...        ...               ...    ... .      ...
#&gt;   ENSG00000124334.17_PAR_Y       chrY 57184101-57197337      + |   HAVANA
#&gt;   ENSG00000185203.12_PAR_Y       chrY 57201143-57203357      - |   HAVANA
#&gt;    ENSG00000270726.6_PAR_Y       chrY 57190738-57208756      + |   HAVANA
#&gt;   ENSG00000182484.15_PAR_Y       chrY 57207346-57212230      + |   HAVANA
#&gt;    ENSG00000227159.8_PAR_Y       chrY 57212184-57214397      - |   HAVANA
#&gt;                                type     score     phase                gene_id
#&gt;                            &lt;factor&gt; &lt;numeric&gt; &lt;integer&gt;            &lt;character&gt;
#&gt;          ENSG00000278704.1     gene      2237      &lt;NA&gt;      ENSG00000278704.1
#&gt;          ENSG00000277400.1     gene      2179      &lt;NA&gt;      ENSG00000277400.1
#&gt;          ENSG00000274847.1     gene      1599      &lt;NA&gt;      ENSG00000274847.1
#&gt;          ENSG00000277428.1     gene       101      &lt;NA&gt;      ENSG00000277428.1
#&gt;          ENSG00000276256.1     gene      2195      &lt;NA&gt;      ENSG00000276256.1
#&gt;                        ...      ...       ...       ...                    ...
#&gt;   ENSG00000124334.17_PAR_Y     gene      2504      &lt;NA&gt; ENSG00000124334.17_P..
#&gt;   ENSG00000185203.12_PAR_Y     gene      1054      &lt;NA&gt; ENSG00000185203.12_P..
#&gt;    ENSG00000270726.6_PAR_Y     gene       773      &lt;NA&gt; ENSG00000270726.6_PA..
#&gt;   ENSG00000182484.15_PAR_Y     gene      4618      &lt;NA&gt; ENSG00000182484.15_P..
#&gt;    ENSG00000227159.8_PAR_Y     gene      1306      &lt;NA&gt; ENSG00000227159.8_PA..
#&gt;                                         gene_type   gene_name       level
#&gt;                                       &lt;character&gt; &lt;character&gt; &lt;character&gt;
#&gt;          ENSG00000278704.1         protein_coding  BX004987.1           3
#&gt;          ENSG00000277400.1         protein_coding  AC145212.1           3
#&gt;          ENSG00000274847.1         protein_coding       MAFIP           3
#&gt;          ENSG00000277428.1               misc_RNA     RF00019           3
#&gt;          ENSG00000276256.1         protein_coding  AC011043.1           3
#&gt;                        ...                    ...         ...         ...
#&gt;   ENSG00000124334.17_PAR_Y         protein_coding        IL9R           2
#&gt;   ENSG00000185203.12_PAR_Y              antisense      WASIR1           2
#&gt;    ENSG00000270726.6_PAR_Y   processed_transcript  AJ271736.1           2
#&gt;   ENSG00000182484.15_PAR_Y transcribed_unproces..      WASH6P           2
#&gt;    ENSG00000227159.8_PAR_Y unprocessed_pseudogene    DDX11L16           2
#&gt;                                     havana_gene         tag
#&gt;                                     &lt;character&gt; &lt;character&gt;
#&gt;          ENSG00000278704.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000277400.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000274847.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000277428.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000276256.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;                        ...                  ...         ...
#&gt;   ENSG00000124334.17_PAR_Y OTTHUMG00000022720.1         PAR
#&gt;   ENSG00000185203.12_PAR_Y OTTHUMG00000022676.3         PAR
#&gt;    ENSG00000270726.6_PAR_Y OTTHUMG00000184987.2         PAR
#&gt;   ENSG00000182484.15_PAR_Y OTTHUMG00000022677.5         PAR
#&gt;    ENSG00000227159.8_PAR_Y OTTHUMG00000022678.1         PAR
#&gt;   -------
#&gt;   seqinfo: 406 sequences from an unspecified genome; no seqlengths</div><div class='input'>
<span class='co'>## Create an RSE object using FANTOM6_CAT instead of gencode_v26</span>
<span class='kw'>rse_gene_SRP009615_fantom6_cat</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='kw'>proj_info</span>,
    annotation = <span class='st'>"fantom6_cat"</span>
)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:18 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:18 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:18 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:19 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:19 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:19 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:19 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:19 caching file human.gene_sums.F006.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/gene_sums/human.gene_sums.F006.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:20 downloading and reading the counts: 12 samples across 124047 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:20 caching file sra.gene_sums.SRP009615.F006.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/15/SRP009615/sra.gene_sums.SRP009615.F006.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:21 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'>rowRanges</span>(<span class='kw'>rse_gene_SRP009615_fantom6_cat</span>)
</div><div class='output co'>#&gt; GRanges object with 124047 ranges and 5 metadata columns:
#&gt;                   seqnames            ranges strand |   source     type
#&gt;                      &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;factor&gt; &lt;factor&gt;
#&gt;   CATG00000042730     chr1     159537-162485      - |  FANTOM6     gene
#&gt;   CATG00000042731     chr1     273882-274416      - |  FANTOM6     gene
#&gt;   ENSG00000223659     chr1     627377-629095      - |  FANTOM6     gene
#&gt;   ENSG00000225630     chr1     630001-630683      + |  FANTOM6     gene
#&gt;   ENSG00000225972     chr1     629209-631743      + |  FANTOM6     gene
#&gt;               ...      ...               ...    ... .      ...      ...
#&gt;   CATG00000114975     chrY 56836712-56851323      + |  FANTOM6     gene
#&gt;   CATG00000115126     chrY 56855793-56856102      - |  FANTOM6     gene
#&gt;   CATG00000114976     chrY 56855491-56858320      + |  FANTOM6     gene
#&gt;   CATG00000114977     chrY 56867675-56882339      + |  FANTOM6     gene
#&gt;   CATG00000115127     chrY 56884759-56885317      - |  FANTOM6     gene
#&gt;                       score     phase         gene_id
#&gt;                   &lt;numeric&gt; &lt;integer&gt;     &lt;character&gt;
#&gt;   CATG00000042730      2949      &lt;NA&gt; CATG00000042730
#&gt;   CATG00000042731       535      &lt;NA&gt; CATG00000042731
#&gt;   ENSG00000223659       887      &lt;NA&gt; ENSG00000223659
#&gt;   ENSG00000225630       683      &lt;NA&gt; ENSG00000225630
#&gt;   ENSG00000225972       438      &lt;NA&gt; ENSG00000225972
#&gt;               ...       ...       ...             ...
#&gt;   CATG00000114975      4376      &lt;NA&gt; CATG00000114975
#&gt;   CATG00000115126       310      &lt;NA&gt; CATG00000115126
#&gt;   CATG00000114976      2830      &lt;NA&gt; CATG00000114976
#&gt;   CATG00000114977     14665      &lt;NA&gt; CATG00000114977
#&gt;   CATG00000115127       559      &lt;NA&gt; CATG00000115127
#&gt;   -------
#&gt;   seqinfo: 25 sequences from an unspecified genome; no seqlengths</div><div class='input'>
<span class='co'>## Create an RSE object using RefSeq instead of gencode_v26</span>
<span class='kw'>rse_gene_SRP009615_refseq</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='kw'>proj_info</span>,
    annotation = <span class='st'>"refseq"</span>
)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:21 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:21 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:22 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:22 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:22 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:22 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:22 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:23 caching file human.gene_sums.R109.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/gene_sums/human.gene_sums.R109.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:24 downloading and reading the counts: 12 samples across 54042 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:24 caching file sra.gene_sums.SRP009615.R109.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/15/SRP009615/sra.gene_sums.SRP009615.R109.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:24 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'>rowRanges</span>(<span class='kw'>rse_gene_SRP009615_refseq</span>)
</div><div class='output co'>#&gt; GRanges object with 54042 ranges and 25 metadata columns:
#&gt;               seqnames            ranges strand |   source     type     score
#&gt;                  &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;factor&gt; &lt;factor&gt; &lt;numeric&gt;
#&gt;   gene14440 GL000008.2     124376-125329      - |   RefSeq     gene       954
#&gt;   gene14441 GL000008.2     153090-153485      - |   RefSeq     gene       396
#&gt;   gene14442 GL000008.2     161987-178171      - |   RefSeq     gene      1744
#&gt;   gene38360 GL000009.2       32290-36345      + |   RefSeq     gene      1512
#&gt;   gene38362 GL000009.2       43342-56258      + |   RefSeq     gene       972
#&gt;         ...        ...               ...    ... .      ...      ...       ...
#&gt;   gene54245       chrY 57171874-57172771      - |   RefSeq     gene       898
#&gt;   gene54246       chrY 57184101-57199537      + |   RefSeq     gene      6193
#&gt;   gene54247       chrY 57201084-57203350      - |   RefSeq     gene      1231
#&gt;   gene54248       chrY 57208178-57212192      + |   RefSeq     gene      1570
#&gt;   gene54249       chrY 57212178-57214703      - |   RefSeq     gene      1643
#&gt;                 phase     gene_id                 Dbxref        Name
#&gt;             &lt;integer&gt; &lt;character&gt;            &lt;character&gt; &lt;character&gt;
#&gt;   gene14440      &lt;NA&gt;   gene14440 GeneID:100419019,HGN..    SNX18P15
#&gt;   gene14441      &lt;NA&gt;   gene14441 GeneID:100419020,HGN..    SNX18P16
#&gt;   gene14442      &lt;NA&gt;   gene14442 GeneID:100874392,Gen.. NR_046228.1
#&gt;   gene38360      &lt;NA&gt;   gene38360 GeneID:100533720,HGN.. ANKRD20A15P
#&gt;   gene38362      &lt;NA&gt;   gene38362 GeneID:105379272,Gen.. XR_949087.2
#&gt;         ...       ...         ...                    ...         ...
#&gt;   gene54245      &lt;NA&gt;   gene54245 GeneID:644218,HGNC:H..      TRPC6P
#&gt;   gene54246      &lt;NA&gt;   gene54246                   &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54247      &lt;NA&gt;   gene54247 GeneID:100128260,Gen.. NR_138048.1
#&gt;   gene54248      &lt;NA&gt;   gene54248 GeneID:653440,HGNC:H..      WASH6P
#&gt;   gene54249      &lt;NA&gt;   gene54249 GeneID:727856,Genban.. NR_110561.1
#&gt;                        description       gbkey gene_biotype    gene_name
#&gt;                        &lt;character&gt; &lt;character&gt;  &lt;character&gt;  &lt;character&gt;
#&gt;   gene14440 sorting nexin 18 pse..        Gene   pseudogene     SNX18P15
#&gt;   gene14441 sorting nexin 18 pse..        Gene   pseudogene     SNX18P16
#&gt;   gene14442                   &lt;NA&gt;    misc_RNA         &lt;NA&gt;  ANKRD20A12P
#&gt;   gene38360 ankyrin repeat domai..        Gene   pseudogene  ANKRD20A15P
#&gt;   gene38362                   &lt;NA&gt;       ncRNA         &lt;NA&gt; LOC105379272
#&gt;         ...                    ...         ...          ...          ...
#&gt;   gene54245 transient receptor p..        Gene   pseudogene       TRPC6P
#&gt;   gene54246                   &lt;NA&gt;        mRNA         &lt;NA&gt;         IL9R
#&gt;   gene54247                   &lt;NA&gt;       ncRNA         &lt;NA&gt;       WASIR1
#&gt;   gene54248 WAS protein family h..        Gene   pseudogene       WASH6P
#&gt;   gene54249                   &lt;NA&gt;    misc_RNA         &lt;NA&gt;     DDX11L16
#&gt;                  pseudo                product         model_evidence
#&gt;             &lt;character&gt;            &lt;character&gt;            &lt;character&gt;
#&gt;   gene14440        true                   &lt;NA&gt;                   &lt;NA&gt;
#&gt;   gene14441        true                   &lt;NA&gt;                   &lt;NA&gt;
#&gt;   gene14442        &lt;NA&gt; ankyrin repeat domai..                   &lt;NA&gt;
#&gt;   gene38360        true                   &lt;NA&gt;                   &lt;NA&gt;
#&gt;   gene38362        &lt;NA&gt; uncharacterized LOC1.. Supporting evidence ..
#&gt;         ...         ...                    ...                    ...
#&gt;   gene54245        true                   &lt;NA&gt;                   &lt;NA&gt;
#&gt;   gene54246        &lt;NA&gt;                   &lt;NA&gt;                   &lt;NA&gt;
#&gt;   gene54247        &lt;NA&gt; WASH and IL9R antise..                   &lt;NA&gt;
#&gt;   gene54248        true                   &lt;NA&gt;                   &lt;NA&gt;
#&gt;   gene54249        &lt;NA&gt; DEAD/H-box helicase ..                   &lt;NA&gt;
#&gt;               exception     partial        Note   inference      geneID
#&gt;             &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;character&gt;
#&gt;   gene14440        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene14441        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene14442        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene38360        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene38362        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;         ...         ...         ...         ...         ...         ...
#&gt;   gene54245        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54246        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54247        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54248        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54249        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;               anticodon        gene_synonym   end_range start_range
#&gt;             &lt;character&gt;         &lt;character&gt; &lt;character&gt; &lt;character&gt;
#&gt;   gene14440        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene14441        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene14442        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene38360        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene38362        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;         ...         ...                 ...         ...         ...
#&gt;   gene54245        &lt;NA&gt;              TRPC6L        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54246        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54247        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54248        &lt;NA&gt; CXYorf1,FAM39A,WASH        &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54249        &lt;NA&gt;                &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;             standard_name      codons
#&gt;               &lt;character&gt; &lt;character&gt;
#&gt;   gene14440          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene14441          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene14442          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene38360          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene38362          &lt;NA&gt;        &lt;NA&gt;
#&gt;         ...           ...         ...
#&gt;   gene54245          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54246          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54247          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54248          &lt;NA&gt;        &lt;NA&gt;
#&gt;   gene54249          &lt;NA&gt;        &lt;NA&gt;
#&gt;   -------
#&gt;   seqinfo: 436 sequences from an unspecified genome; no seqlengths</div><div class='input'>
<span class='co'>## Create an RSE object using ERCC instead of gencode_v26</span>
<span class='kw'>rse_gene_SRP009615_ercc</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='kw'>proj_info</span>,
    annotation = <span class='st'>"ercc"</span>
)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:24 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:25 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:25 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:25 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:25 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:25 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:26 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:26 caching file human.gene_sums.ERCC.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/gene_sums/human.gene_sums.ERCC.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:26 downloading and reading the counts: 12 samples across 92 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:26 caching file sra.gene_sums.SRP009615.ERCC.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/15/SRP009615/sra.gene_sums.SRP009615.ERCC.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:26 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'>rowRanges</span>(<span class='kw'>rse_gene_SRP009615_ercc</span>)
</div><div class='output co'>#&gt; GRanges object with 92 ranges and 6 metadata columns:
#&gt;                seqnames    ranges strand |   source     type     score
#&gt;                   &lt;Rle&gt; &lt;IRanges&gt;  &lt;Rle&gt; | &lt;factor&gt; &lt;factor&gt; &lt;numeric&gt;
#&gt;   ERCC-00002 ERCC-00002    1-1061      + |     ERCC     gene      1061
#&gt;   ERCC-00003 ERCC-00003    1-1023      + |     ERCC     gene      1023
#&gt;   ERCC-00004 ERCC-00004     1-523      + |     ERCC     gene       523
#&gt;   ERCC-00009 ERCC-00009     1-984      + |     ERCC     gene       984
#&gt;   ERCC-00012 ERCC-00012     1-994      + |     ERCC     gene       994
#&gt;          ...        ...       ...    ... .      ...      ...       ...
#&gt;   ERCC-00164 ERCC-00164    1-1022      + |     ERCC     gene      1022
#&gt;   ERCC-00165 ERCC-00165     1-872      + |     ERCC     gene       872
#&gt;   ERCC-00168 ERCC-00168    1-1024      + |     ERCC     gene      1024
#&gt;   ERCC-00170 ERCC-00170    1-1023      + |     ERCC     gene      1023
#&gt;   ERCC-00171 ERCC-00171     1-505      + |     ERCC     gene       505
#&gt;                  phase     gene_id transcript_id
#&gt;              &lt;integer&gt; &lt;character&gt;   &lt;character&gt;
#&gt;   ERCC-00002      &lt;NA&gt;  ERCC-00002      DQ459430
#&gt;   ERCC-00003      &lt;NA&gt;  ERCC-00003      DQ516784
#&gt;   ERCC-00004      &lt;NA&gt;  ERCC-00004      DQ516752
#&gt;   ERCC-00009      &lt;NA&gt;  ERCC-00009      DQ668364
#&gt;   ERCC-00012      &lt;NA&gt;  ERCC-00012      DQ883670
#&gt;          ...       ...         ...           ...
#&gt;   ERCC-00164      &lt;NA&gt;  ERCC-00164      DQ516779
#&gt;   ERCC-00165      &lt;NA&gt;  ERCC-00165      DQ668363
#&gt;   ERCC-00168      &lt;NA&gt;  ERCC-00168      DQ516776
#&gt;   ERCC-00170      &lt;NA&gt;  ERCC-00170      DQ516773
#&gt;   ERCC-00171      &lt;NA&gt;  ERCC-00171      DQ854994
#&gt;   -------
#&gt;   seqinfo: 92 sequences from an unspecified genome; no seqlengths</div><div class='input'>
<span class='co'>## Create an RSE object using SIRV instead of gencode_v26</span>
<span class='kw'>rse_gene_SRP009615_sirv</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='kw'>proj_info</span>,
    annotation = <span class='st'>"sirv"</span>
)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:26 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:26 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:27 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:27 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:27 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:27 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:28 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:28 caching file human.gene_sums.SIRV.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/gene_sums/human.gene_sums.SIRV.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:28 downloading and reading the counts: 12 samples across 7 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:28 caching file sra.gene_sums.SRP009615.SIRV.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/gene_sums/15/SRP009615/sra.gene_sums.SRP009615.SIRV.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:28 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'>rowRanges</span>(<span class='kw'>rse_gene_SRP009615_sirv</span>)
</div><div class='output co'>#&gt; GRanges object with 7 ranges and 5 metadata columns:
#&gt;         seqnames      ranges strand |          source     type     score
#&gt;            &lt;Rle&gt;   &lt;IRanges&gt;  &lt;Rle&gt; |        &lt;factor&gt; &lt;factor&gt; &lt;numeric&gt;
#&gt;   SIRV1    SIRV1  1001-11643      * | LexogenSIRVData     gene        NA
#&gt;   SIRV2    SIRV2   1001-5911      * | LexogenSIRVData     gene        NA
#&gt;   SIRV3    SIRV3   1001-9943      * | LexogenSIRVData     gene        NA
#&gt;   SIRV4    SIRV4  1001-15122      * | LexogenSIRVData     gene        NA
#&gt;   SIRV5    SIRV5  1001-13606      * | LexogenSIRVData     gene        NA
#&gt;   SIRV6    SIRV6  1001-11837      * | LexogenSIRVData     gene        NA
#&gt;   SIRV7    SIRV7 1001-147957      * | LexogenSIRVData     gene        NA
#&gt;             phase     gene_id
#&gt;         &lt;integer&gt; &lt;character&gt;
#&gt;   SIRV1         0       SIRV1
#&gt;   SIRV2         0       SIRV2
#&gt;   SIRV3         0       SIRV3
#&gt;   SIRV4         0       SIRV4
#&gt;   SIRV5         0       SIRV5
#&gt;   SIRV6         0       SIRV6
#&gt;   SIRV7         0       SIRV7
#&gt;   -------
#&gt;   seqinfo: 7 sequences from an unspecified genome; no seqlengths</div><div class='input'>
<span class='co'>## Obtain a list of RSE objects for all gene annotations</span>
<span class='kw'>rses_gene</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/lapply.html'>lapply</a></span>(<span class='fu'><a href='annotation_options.html'>annotation_options</a></span>(), <span class='fu'>function</span>(<span class='kw'>x</span>) {
    <span class='fu'>create_rse</span>(<span class='kw'>proj_info</span>, type = <span class='st'>"gene"</span>, annotation = <span class='kw'>x</span>)
})
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:28 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:28 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:29 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:29 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:29 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:29 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:30 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:30 caching file human.gene_sums.G026.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:30 downloading and reading the counts: 12 samples across 63856 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:30 caching file sra.gene_sums.SRP009615.G026.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:31 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:31 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:31 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:31 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:31 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:31 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:32 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:32 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:32 caching file human.gene_sums.G029.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:33 downloading and reading the counts: 12 samples across 64837 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:33 caching file sra.gene_sums.SRP009615.G029.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:33 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:33 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:33 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:33 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:33 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:34 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:34 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:34 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:34 caching file human.gene_sums.F006.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:35 downloading and reading the counts: 12 samples across 124047 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:35 caching file sra.gene_sums.SRP009615.F006.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:35 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:35 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:35 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:35 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:36 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:36 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:36 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:36 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:36 caching file human.gene_sums.R109.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:37 downloading and reading the counts: 12 samples across 54042 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:37 caching file sra.gene_sums.SRP009615.R109.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:38 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:38 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:38 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:39 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:39 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:39 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:39 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 caching file human.gene_sums.ERCC.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 downloading and reading the counts: 12 samples across 92 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 caching file sra.gene_sums.SRP009615.ERCC.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:40 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:41 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:41 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:41 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:41 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:41 caching file human.gene_sums.SIRV.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:42 downloading and reading the counts: 12 samples across 7 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:42 caching file sra.gene_sums.SRP009615.SIRV.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:42 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'><span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span>(<span class='kw'>rses_gene</span>) <span class='op'>&lt;-</span> <span class='fu'><a href='annotation_options.html'>annotation_options</a></span>()
<span class='kw'>rses_gene</span>
</div><div class='output co'>#&gt; $gencode_v26
#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 63856 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ...
#&gt;   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
#&gt; rowData names(10): source type ... havana_gene tag
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL
#&gt; 
#&gt; $gencode_v29
#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 64837 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(64837): ENSG00000278704.1 ENSG00000277400.1 ...
#&gt;   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
#&gt; rowData names(10): source type ... havana_gene tag
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL
#&gt; 
#&gt; $fantom6_cat
#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 124047 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(124047): CATG00000042730 CATG00000042731 ... CATG00000114977
#&gt;   CATG00000115127
#&gt; rowData names(5): source type score phase gene_id
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL
#&gt; 
#&gt; $refseq
#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 54042 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(54042): gene14440 gene14441 ... gene54248 gene54249
#&gt; rowData names(25): source type ... standard_name codons
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL
#&gt; 
#&gt; $ercc
#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 92 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(92): ERCC-00002 ERCC-00003 ... ERCC-00170 ERCC-00171
#&gt; rowData names(6): source type ... gene_id transcript_id
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL
#&gt; 
#&gt; $sirv
#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 7 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(7): SIRV1 SIRV2 ... SIRV6 SIRV7
#&gt; rowData names(5): source type score phase gene_id
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL
#&gt; </div><div class='input'>
<span class='co'>## Create a RSE object at the exon level</span>
<span class='kw'>rse_exon_SRP009615</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='kw'>proj_info</span>,
    type = <span class='st'>"exon"</span>
)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:42 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:42 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:42 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:42 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:43 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:43 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:43 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:41:43 caching file human.exon_sums.G026.gtf.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/annotations/exon_sums/human.exon_sums.G026.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:12 downloading and reading the counts: 12 samples across 1299686 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:12 caching file sra.exon_sums.SRP009615.G026.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/exon_sums/15/SRP009615/sra.exon_sums.SRP009615.G026.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:14 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'>
<span class='co'>## Explore the resulting RSE exon object</span>
<span class='kw'>rse_exon_SRP009615</span>
</div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 1299686 12 
#&gt; metadata(8): time_created recount3_version ... annotation recount3_url
#&gt; assays(1): raw_counts
#&gt; rownames(1299686): GL000009.2|56140|58376|- GL000194.1|53594|54832|-
#&gt;   ... chrY|57213880|57213964|- chrY|57214350|57214397|-
#&gt; rowData names(21): source type ... ont ccdsid
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL</div><div class='input'>
<span class='fu'><a href='https://rdrr.io/r/base/dim.html'>dim</a></span>(<span class='kw'>rse_exon_SRP009615</span>)
</div><div class='output co'>#&gt; [1] 1299686      12</div><div class='input'><span class='fu'>rowRanges</span>(<span class='kw'>rse_exon_SRP009615</span>)
</div><div class='output co'>#&gt; GRanges object with 1299686 ranges and 21 metadata columns:
#&gt;                                seqnames            ranges strand |   source
#&gt;                                   &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;factor&gt;
#&gt;     GL000009.2|56140|58376|- GL000009.2       56140-58376      - |  ENSEMBL
#&gt;     GL000194.1|53594|54832|- GL000194.1       53594-54832      - |  ENSEMBL
#&gt;     GL000194.1|55446|55676|- GL000194.1       55446-55676      - |  ENSEMBL
#&gt;     GL000194.1|53590|55676|- GL000194.1       53590-55676      - |  ENSEMBL
#&gt;   GL000194.1|112792|112850|- GL000194.1     112792-112850      - |  ENSEMBL
#&gt;                          ...        ...               ...    ... .      ...
#&gt;     chrY|57212184|57213125|-       chrY 57212184-57213125      - |   HAVANA
#&gt;     chrY|57213204|57213357|-       chrY 57213204-57213357      - |   HAVANA
#&gt;     chrY|57213526|57213602|-       chrY 57213526-57213602      - |   HAVANA
#&gt;     chrY|57213880|57213964|-       chrY 57213880-57213964      - |   HAVANA
#&gt;     chrY|57214350|57214397|-       chrY 57214350-57214397      - |   HAVANA
#&gt;                                  type     score     phase
#&gt;                              &lt;factor&gt; &lt;numeric&gt; &lt;integer&gt;
#&gt;     GL000009.2|56140|58376|-     exon      2237      &lt;NA&gt;
#&gt;     GL000194.1|53594|54832|-     exon      1239      &lt;NA&gt;
#&gt;     GL000194.1|55446|55676|-     exon       231      &lt;NA&gt;
#&gt;     GL000194.1|53590|55676|-     exon      2087      &lt;NA&gt;
#&gt;   GL000194.1|112792|112850|-     exon        59      &lt;NA&gt;
#&gt;                          ...      ...       ...       ...
#&gt;     chrY|57212184|57213125|-     exon       942      &lt;NA&gt;
#&gt;     chrY|57213204|57213357|-     exon       154      &lt;NA&gt;
#&gt;     chrY|57213526|57213602|-     exon        77      &lt;NA&gt;
#&gt;     chrY|57213880|57213964|-     exon        85      &lt;NA&gt;
#&gt;     chrY|57214350|57214397|-     exon        48      &lt;NA&gt;
#&gt;                                             gene_id          transcript_id
#&gt;                                         &lt;character&gt;            &lt;character&gt;
#&gt;     GL000009.2|56140|58376|-      ENSG00000278704.1      ENST00000618686.1
#&gt;     GL000194.1|53594|54832|-      ENSG00000274847.1      ENST00000400754.4
#&gt;     GL000194.1|55446|55676|-      ENSG00000274847.1      ENST00000400754.4
#&gt;     GL000194.1|53590|55676|-      ENSG00000277400.1      ENST00000613230.1
#&gt;   GL000194.1|112792|112850|-      ENSG00000274847.1      ENST00000400754.4
#&gt;                          ...                    ...                    ...
#&gt;     chrY|57212184|57213125|- ENSG00000227159.8_PA.. ENST00000507418.6_PA..
#&gt;     chrY|57213204|57213357|- ENSG00000227159.8_PA.. ENST00000507418.6_PA..
#&gt;     chrY|57213526|57213602|- ENSG00000227159.8_PA.. ENST00000507418.6_PA..
#&gt;     chrY|57213880|57213964|- ENSG00000227159.8_PA.. ENST00000507418.6_PA..
#&gt;     chrY|57214350|57214397|- ENSG00000227159.8_PA.. ENST00000507418.6_PA..
#&gt;                                           gene_type   gene_name
#&gt;                                         &lt;character&gt; &lt;character&gt;
#&gt;     GL000009.2|56140|58376|-         protein_coding  BX004987.1
#&gt;     GL000194.1|53594|54832|-         protein_coding  AC145212.1
#&gt;     GL000194.1|55446|55676|-         protein_coding  AC145212.1
#&gt;     GL000194.1|53590|55676|-         protein_coding  AC145212.2
#&gt;   GL000194.1|112792|112850|-         protein_coding  AC145212.1
#&gt;                          ...                    ...         ...
#&gt;     chrY|57212184|57213125|- unprocessed_pseudogene    DDX11L16
#&gt;     chrY|57213204|57213357|- unprocessed_pseudogene    DDX11L16
#&gt;     chrY|57213526|57213602|- unprocessed_pseudogene    DDX11L16
#&gt;     chrY|57213880|57213964|- unprocessed_pseudogene    DDX11L16
#&gt;     chrY|57214350|57214397|- unprocessed_pseudogene    DDX11L16
#&gt;                                     transcript_type transcript_name exon_number
#&gt;                                         &lt;character&gt;     &lt;character&gt; &lt;character&gt;
#&gt;     GL000009.2|56140|58376|-         protein_coding  BX004987.1-201           1
#&gt;     GL000194.1|53594|54832|-         protein_coding  AC145212.1-201           4
#&gt;     GL000194.1|55446|55676|-         protein_coding  AC145212.1-201           3
#&gt;     GL000194.1|53590|55676|-         protein_coding  AC145212.2-201           3
#&gt;   GL000194.1|112792|112850|-         protein_coding  AC145212.1-201           2
#&gt;                          ...                    ...             ...         ...
#&gt;     chrY|57212184|57213125|- unprocessed_pseudogene    DDX11L16-001           5
#&gt;     chrY|57213204|57213357|- unprocessed_pseudogene    DDX11L16-001           4
#&gt;     chrY|57213526|57213602|- unprocessed_pseudogene    DDX11L16-001           3
#&gt;     chrY|57213880|57213964|- unprocessed_pseudogene    DDX11L16-001           2
#&gt;     chrY|57214350|57214397|- unprocessed_pseudogene    DDX11L16-001           1
#&gt;                                        exon_id       level        protein_id
#&gt;                                    &lt;character&gt; &lt;character&gt;       &lt;character&gt;
#&gt;     GL000009.2|56140|58376|- ENSE00003753029.1           3 ENSP00000484918.1
#&gt;     GL000194.1|53594|54832|- ENSE00002218789.2           3 ENSP00000478910.1
#&gt;     GL000194.1|55446|55676|- ENSE00003714436.1           3 ENSP00000478910.1
#&gt;     GL000194.1|53590|55676|- ENSE00003723764.1           3 ENSP00000483280.1
#&gt;   GL000194.1|112792|112850|- ENSE00003713687.1           3 ENSP00000478910.1
#&gt;                          ...               ...         ...               ...
#&gt;     chrY|57212184|57213125|- ENSE00002023900.1           2              &lt;NA&gt;
#&gt;     chrY|57213204|57213357|- ENSE00002036959.1           2              &lt;NA&gt;
#&gt;     chrY|57213526|57213602|- ENSE00002021169.1           2              &lt;NA&gt;
#&gt;     chrY|57213880|57213964|- ENSE00002046926.1           2              &lt;NA&gt;
#&gt;     chrY|57214350|57214397|- ENSE00002072208.1           2              &lt;NA&gt;
#&gt;                              transcript_support_level         tag
#&gt;                                           &lt;character&gt; &lt;character&gt;
#&gt;     GL000009.2|56140|58376|-                       NA       basic
#&gt;     GL000194.1|53594|54832|-                        1       basic
#&gt;     GL000194.1|55446|55676|-                        1       basic
#&gt;     GL000194.1|53590|55676|-                        1       basic
#&gt;   GL000194.1|112792|112850|-                        1       basic
#&gt;                          ...                      ...         ...
#&gt;     chrY|57212184|57213125|-                       NA         PAR
#&gt;     chrY|57213204|57213357|-                       NA         PAR
#&gt;     chrY|57213526|57213602|-                       NA         PAR
#&gt;     chrY|57213880|57213964|-                       NA         PAR
#&gt;     chrY|57214350|57214397|-                       NA         PAR
#&gt;                                     recount_exon_id          havana_gene
#&gt;                                         &lt;character&gt;          &lt;character&gt;
#&gt;     GL000009.2|56140|58376|- GL000009.2|56140|583..                 &lt;NA&gt;
#&gt;     GL000194.1|53594|54832|- GL000194.1|53594|548..                 &lt;NA&gt;
#&gt;     GL000194.1|55446|55676|- GL000194.1|55446|556..                 &lt;NA&gt;
#&gt;     GL000194.1|53590|55676|- GL000194.1|53590|556..                 &lt;NA&gt;
#&gt;   GL000194.1|112792|112850|- GL000194.1|112792|11..                 &lt;NA&gt;
#&gt;                          ...                    ...                  ...
#&gt;     chrY|57212184|57213125|- chrY|57212184|572131.. OTTHUMG00000022678.1
#&gt;     chrY|57213204|57213357|- chrY|57213204|572133.. OTTHUMG00000022678.1
#&gt;     chrY|57213526|57213602|- chrY|57213526|572136.. OTTHUMG00000022678.1
#&gt;     chrY|57213880|57213964|- chrY|57213880|572139.. OTTHUMG00000022678.1
#&gt;     chrY|57214350|57214397|- chrY|57214350|572143.. OTTHUMG00000022678.1
#&gt;                                 havana_transcript         ont      ccdsid
#&gt;                                       &lt;character&gt; &lt;character&gt; &lt;character&gt;
#&gt;     GL000009.2|56140|58376|-                 &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;     GL000194.1|53594|54832|-                 &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;     GL000194.1|55446|55676|-                 &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;     GL000194.1|53590|55676|-                 &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;   GL000194.1|112792|112850|-                 &lt;NA&gt;        &lt;NA&gt;        &lt;NA&gt;
#&gt;                          ...                  ...         ...         ...
#&gt;     chrY|57212184|57213125|- OTTHUMT00000058841.1 PGO:0000005        &lt;NA&gt;
#&gt;     chrY|57213204|57213357|- OTTHUMT00000058841.1 PGO:0000005        &lt;NA&gt;
#&gt;     chrY|57213526|57213602|- OTTHUMT00000058841.1 PGO:0000005        &lt;NA&gt;
#&gt;     chrY|57213880|57213964|- OTTHUMT00000058841.1 PGO:0000005        &lt;NA&gt;
#&gt;     chrY|57214350|57214397|- OTTHUMT00000058841.1 PGO:0000005        &lt;NA&gt;
#&gt;   -------
#&gt;   seqinfo: 374 sequences from an unspecified genome; no seqlengths</div><div class='input'><span class='kw'>pryr</span>::<span class='fu'><a href='https://rdrr.io/pkg/pryr/man/object_size.html'>object_size</a></span>(<span class='kw'>rse_exon_SRP009615</span>)
</div><div class='output co'>#&gt; 466 MB</div><div class='input'>
<span class='co'>## Create a RSE object at the exon-exon junction level</span>
<span class='kw'>rse_jxn_SRP009615</span> <span class='op'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='kw'>proj_info</span>,
    type = <span class='st'>"jxn"</span>
)
</div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:17 downloading and reading the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:17 caching file sra.sra.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:18 caching file sra.recount_project.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:18 caching file sra.recount_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:18 caching file sra.recount_seq_qc.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:18 caching file sra.recount_pred.SRP009615.MD.gz.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:19 downloading and reading the feature information.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:19 caching file sra.junctions.SRP009615.ALL.RR.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/junctions/15/SRP009615/sra.junctions.SRP009615.ALL.RR.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:21 downloading and reading the counts: 12 samples across 281448 features.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:21 caching file sra.junctions.SRP009615.ALL.MM.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/junctions/15/SRP009615/sra.junctions.SRP009615.ALL.MM.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:23 matching exon-exon junction counts with the metadata.</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:23 caching file sra.junctions.SRP009615.ALL.ID.gz.</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://idies.jhu.edu/recount3/data/human/data_sources/sra/junctions/15/SRP009615/sra.junctions.SRP009615.ALL.ID.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-09-30 01:42:24 construcing the RangedSummarizedExperiment (rse) object.</span></div><div class='input'>
<span class='co'>## Explore the resulting RSE exon-exon junctions object</span>
<span class='kw'>rse_jxn_SRP009615</span>
</div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 281448 12 
#&gt; metadata(9): time_created recount3_version ... jxn_format recount3_url
#&gt; assays(1): counts
#&gt; rownames(281448): chr1:11845-12009:+ chr1:12698-13220:+ ...
#&gt;   chrY:56848810-56851543:- chrY:56850515-56850921:+
#&gt; rowData names(6): length annotated ... left_annotated right_annotated
#&gt; colnames(12): SRR387777 SRR387778 ... SRR389077 SRR389078
#&gt; colData names(173): rail_id external_id ... recount_pred.pred.type
#&gt;   BigWigURL</div><div class='input'>
<span class='fu'><a href='https://rdrr.io/r/base/dim.html'>dim</a></span>(<span class='kw'>rse_jxn_SRP009615</span>)
</div><div class='output co'>#&gt; [1] 281448     12</div><div class='input'><span class='fu'>rowRanges</span>(<span class='kw'>rse_jxn_SRP009615</span>)
</div><div class='output co'>#&gt; GRanges object with 281448 ranges and 6 metadata columns:
#&gt;                            seqnames            ranges strand |    length
#&gt;                               &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt;
#&gt;         chr1:11845-12009:+     chr1       11845-12009      + |       165
#&gt;         chr1:12698-13220:+     chr1       12698-13220      + |       523
#&gt;        chr1:14696-185174:-     chr1      14696-185174      - |    170479
#&gt;         chr1:14830-14969:-     chr1       14830-14969      - |       140
#&gt;         chr1:14830-15020:-     chr1       14830-15020      - |       191
#&gt;                        ...      ...               ...    ... .       ...
#&gt;   chrY:56846131-56846553:+     chrY 56846131-56846553      + |       423
#&gt;   chrY:56846268-56846553:+     chrY 56846268-56846553      + |       286
#&gt;   chrY:56846486-56846553:+     chrY 56846486-56846553      + |        68
#&gt;   chrY:56848810-56851543:-     chrY 56848810-56851543      - |      2734
#&gt;   chrY:56850515-56850921:+     chrY 56850515-56850921      + |       407
#&gt;                            annotated  left_motif right_motif
#&gt;                            &lt;integer&gt; &lt;character&gt; &lt;character&gt;
#&gt;         chr1:11845-12009:+         0          GT          AG
#&gt;         chr1:12698-13220:+         1          GT          AG
#&gt;        chr1:14696-185174:-         0          CT          AC
#&gt;         chr1:14830-14969:-         1          CT          AC
#&gt;         chr1:14830-15020:-         0          CT          AC
#&gt;                        ...       ...         ...         ...
#&gt;   chrY:56846131-56846553:+         0          GT          AG
#&gt;   chrY:56846268-56846553:+         0          GT          AG
#&gt;   chrY:56846486-56846553:+         0          GT          AG
#&gt;   chrY:56848810-56851543:-         0          CT          AC
#&gt;   chrY:56850515-56850921:+         0          GT          AG
#&gt;                                    left_annotated        right_annotated
#&gt;                                       &lt;character&gt;            &lt;character&gt;
#&gt;         chr1:11845-12009:+                      0              aC19,sG19
#&gt;         chr1:12698-13220:+ aC19,gC19,gC24,gC25,.. aC19,cH38,gC19,gC24,..
#&gt;        chr1:14696-185174:-                      0                      0
#&gt;         chr1:14830-14969:- aC19,cH38,gC19,kG19,.. aC19,cH38,gC19,kG19,..
#&gt;         chr1:14830-15020:- aC19,cH38,gC19,kG19,..                      0
#&gt;                        ...                    ...                    ...
#&gt;   chrY:56846131-56846553:+                      0                      0
#&gt;   chrY:56846268-56846553:+                      0                      0
#&gt;   chrY:56846486-56846553:+                      0                      0
#&gt;   chrY:56848810-56851543:-                      0                      0
#&gt;   chrY:56850515-56850921:+                      0                      0
#&gt;   -------
#&gt;   seqinfo: 97 sequences from an unspecified genome; no seqlengths</div><div class='input'><span class='kw'>pryr</span>::<span class='fu'><a href='https://rdrr.io/pkg/pryr/man/object_size.html'>object_size</a></span>(<span class='kw'>rse_jxn_SRP009615</span>)
</div><div class='output co'>#&gt; 60.3 MB</div><div class='input'>
<span class='co'>## Obtain a list of RSE objects for all exon annotations</span>
<span class='co'>if</span> (<span class='fl'>FALSE</span>) {
<span class='kw'>rses_exon</span> <span class='op'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/lapply.html'>lapply</a></span>(<span class='fu'><a href='annotation_options.html'>annotation_options</a></span>(), <span class='fu'>function</span>(<span class='kw'>x</span>) {
    <span class='fu'>create_rse</span>(<span class='kw'>proj_info</span>, type = <span class='st'>"exon"</span>, annotation = <span class='kw'>x</span>)
})
<span class='fu'><a href='https://rdrr.io/r/base/names.html'>names</a></span>(<span class='kw'>rses_exon</span>) <span class='op'>&lt;-</span> <span class='fu'><a href='annotation_options.html'>annotation_options</a></span>()
}
</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.9000.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


