<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Create a recount3 RangedSummarizedExperiment gene or exon object — create_rse • recount3</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Create a recount3 RangedSummarizedExperiment gene or exon object — create_rse" />
<meta property="og:description" content="Once you have identified a project you want to work with, you can use this
function to construct a recount3
RangedSummarizedExperiment-class
(RSE) object at the gene or exon expression feature level. This function will
retrieve the data, cache it, then assemble the RSE object." />
<meta property="og:image" content="/logo.png" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LieberInstitute/recount3/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Create a recount3 RangedSummarizedExperiment gene or exon object</h1>
    <small class="dont-index">Source: <a href='https://github.com/LieberInstitute/recount3/blob/master/R/create_rse.R'><code>R/create_rse.R</code></a></small>
    <div class="hidden name"><code>create_rse.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Once you have identified a project you want to work with, you can use this
function to construct a <code>recount3</code>
<a href='https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html'>RangedSummarizedExperiment-class</a>
(RSE) object at the gene or exon expression feature level. This function will
retrieve the data, cache it, then assemble the RSE object.</p>
    </div>

    <pre class="usage"><span class='fu'>create_rse</span>(
  <span class='no'>project_info</span>,
  <span class='kw'>type</span> <span class='kw'>=</span> <span class='fu'><a href='https://rdrr.io/r/base/c.html'>c</a></span>(<span class='st'>"gene"</span>, <span class='st'>"exon"</span>),
  <span class='kw'>annotation</span> <span class='kw'>=</span> <span class='fu'><a href='annotation_options.html'>annotation_options</a></span>(<span class='no'>project_info</span>$<span class='no'>organism</span>),
  <span class='kw'>bfc</span> <span class='kw'>=</span> <span class='kw pkg'>BiocFileCache</span><span class='kw ns'>::</span><span class='fu'><a href='https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html'>BiocFileCache</a></span>(),
  <span class='kw'>recount3_url</span> <span class='kw'>=</span> <span class='st'>"http://snaptron.cs.jhu.edu/data/temp/recount3"</span>
)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>project_info</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/data.frame.html'>data.frame()</a></code> with one row that contains the
information for the project you are interested in. You can find which
project to work on using <code>todo()</code>.</p></td>
    </tr>
    <tr>
      <th>type</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying whether you want to access gene
counts or exon counts.</p></td>
    </tr>
    <tr>
      <th>annotation</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying which annotation you want to
download. Only used when <code>type</code> is either <code>gene</code> or <code>exon</code>.</p></td>
    </tr>
    <tr>
      <th>bfc</th>
      <td><p>A <a href='https://rdrr.io/pkg/BiocFileCache/man/BiocFileCache-class.html'>BiocFileCache-class</a>
object where the files will be cached to.</p></td>
    </tr>
    <tr>
      <th>recount3_url</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> specifying the home URL for <code>recount3</code>
or a local directory where you have mirrored <code>recount3</code>.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A
<a href='https://rdrr.io/pkg/SummarizedExperiment/man/RangedSummarizedExperiment-class.html'>RangedSummarizedExperiment-class</a>
object.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='co'>## Find the project you are interested in</span>
<span class='co'>## (temporarily build this data.frame)</span>
<span class='no'>proj_info</span> <span class='kw'>&lt;-</span> <span class='fu'><a href='https://rdrr.io/r/base/data.frame.html'>data.frame</a></span>(
    <span class='kw'>project</span> <span class='kw'>=</span> <span class='st'>"ERP110066"</span>,
    <span class='kw'>project_home</span> <span class='kw'>=</span> <span class='st'>"data_sources/sra"</span>,
    <span class='kw'>organism</span> <span class='kw'>=</span> <span class='st'>"human"</span>
)

<span class='co'>## Create a RSE object at the gene level</span>
<span class='no'>rse_gene_ERP110066</span> <span class='kw'>&lt;-</span> <span class='fu'>create_rse</span>(<span class='no'>proj_info</span>)</div><div class='output co'>#&gt; <span class='message'>using temporary cache /tmp/RtmpvzkdzB/BiocFileCache</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources'</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/collections'</span></div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:36:44 downloading and reading the metadata</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.ncbi.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/metadata/66/ERP110066/sra.ncbi.ERP110066.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_project.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/metadata/66/ERP110066/sra.recount_project.ERP110066.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_qc.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/metadata/66/ERP110066/sra.recount_qc.ERP110066.MD.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:36:47 downloading and reading the feature information</span></div><div class='output co'>#&gt; <span class='message'>Caching file human.gene_sums.G026.gtf.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/annotations/gene_sums/human.gene_sums.G026.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:36:48 downloading and reading the counts: 1024 samples across 63856 features.</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.gene_sums.ERP110066.G026.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/gene_sums/66/ERP110066/sra.gene_sums.ERP110066.G026.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:36:56 construcing the RangedSummarizedExperiment (rse) object</span></div><div class='input'>
<span class='co'>## Explore the resulting RSE gene object</span>
<span class='no'>rse_gene_ERP110066</span></div><div class='output co'>#&gt; class: RangedSummarizedExperiment 
#&gt; dim: 63856 1024 
#&gt; metadata(0):
#&gt; assays(1): counts
#&gt; rownames(63856): ENSG00000278704.1 ENSG00000277400.1 ...
#&gt;   ENSG00000182484.15_PAR_Y ENSG00000227159.8_PAR_Y
#&gt; rowData names(10): source type ... havana_gene tag
#&gt; colnames(1024): ERR2713604 ERR2713605 ... ERR2713998 ERR2713999
#&gt; colData names(139): rail_id run_acc ...
#&gt;   recount_qc.junction_avg_coverage BigWigURL</div><div class='input'>
<span class='fu'><a href='https://rdrr.io/r/base/dim.html'>dim</a></span>(<span class='no'>rse_gene_ERP110066</span>)</div><div class='output co'>#&gt; [1] 63856  1024</div><div class='input'><span class='fu'>rowRanges</span>(<span class='no'>rse_gene_ERP110066</span>)</div><div class='output co'>#&gt; GRanges object with 63856 ranges and 10 metadata columns:
#&gt;                              seqnames            ranges strand |   source
#&gt;                                 &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;factor&gt;
#&gt;          ENSG00000278704.1 GL000009.2       56140-58376      - |  ENSEMBL
#&gt;          ENSG00000277400.1 GL000194.1      53590-115018      - |  ENSEMBL
#&gt;          ENSG00000274847.1 GL000194.1      53594-115055      - |  ENSEMBL
#&gt;          ENSG00000277428.1 GL000195.1       37434-37534      - |  ENSEMBL
#&gt;          ENSG00000276256.1 GL000195.1       42939-49164      - |  ENSEMBL
#&gt;                        ...        ...               ...    ... .      ...
#&gt;   ENSG00000124334.17_PAR_Y       chrY 57184101-57197337      + |   HAVANA
#&gt;   ENSG00000185203.12_PAR_Y       chrY 57201143-57203357      - |   HAVANA
#&gt;    ENSG00000270726.6_PAR_Y       chrY 57190738-57208756      + |   HAVANA
#&gt;   ENSG00000182484.15_PAR_Y       chrY 57207346-57212230      + |   HAVANA
#&gt;    ENSG00000227159.8_PAR_Y       chrY 57212184-57214397      - |   HAVANA
#&gt;                                type     score     phase                gene_id
#&gt;                            &lt;factor&gt; &lt;numeric&gt; &lt;integer&gt;            &lt;character&gt;
#&gt;          ENSG00000278704.1     gene      2237      &lt;NA&gt;      ENSG00000278704.1
#&gt;          ENSG00000277400.1     gene      2179      &lt;NA&gt;      ENSG00000277400.1
#&gt;          ENSG00000274847.1     gene      1599      &lt;NA&gt;      ENSG00000274847.1
#&gt;          ENSG00000277428.1     gene       101      &lt;NA&gt;      ENSG00000277428.1
#&gt;          ENSG00000276256.1     gene      2195      &lt;NA&gt;      ENSG00000276256.1
#&gt;                        ...      ...       ...       ...                    ...
#&gt;   ENSG00000124334.17_PAR_Y     gene      2504      &lt;NA&gt; ENSG00000124334.17_P..
#&gt;   ENSG00000185203.12_PAR_Y     gene      1054      &lt;NA&gt; ENSG00000185203.12_P..
#&gt;    ENSG00000270726.6_PAR_Y     gene       773      &lt;NA&gt; ENSG00000270726.6_PA..
#&gt;   ENSG00000182484.15_PAR_Y     gene      4618      &lt;NA&gt; ENSG00000182484.15_P..
#&gt;    ENSG00000227159.8_PAR_Y     gene      1306      &lt;NA&gt; ENSG00000227159.8_PA..
#&gt;                                         gene_type   gene_name       level
#&gt;                                       &lt;character&gt; &lt;character&gt; &lt;character&gt;
#&gt;          ENSG00000278704.1         protein_coding  BX004987.1           3
#&gt;          ENSG00000277400.1         protein_coding  AC145212.2           3
#&gt;          ENSG00000274847.1         protein_coding  AC145212.1           3
#&gt;          ENSG00000277428.1               misc_RNA       Y_RNA           3
#&gt;          ENSG00000276256.1         protein_coding  AC011043.1           3
#&gt;                        ...                    ...         ...         ...
#&gt;   ENSG00000124334.17_PAR_Y         protein_coding        IL9R           2
#&gt;   ENSG00000185203.12_PAR_Y              antisense      WASIR1           2
#&gt;    ENSG00000270726.6_PAR_Y   processed_transcript AJ271736.10           2
#&gt;   ENSG00000182484.15_PAR_Y transcribed_unproces..      WASH6P           2
#&gt;    ENSG00000227159.8_PAR_Y unprocessed_pseudogene    DDX11L16           2
#&gt;                                     havana_gene         tag
#&gt;                                     &lt;character&gt; &lt;character&gt;
#&gt;          ENSG00000278704.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000277400.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000274847.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000277428.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;          ENSG00000276256.1                 &lt;NA&gt;        &lt;NA&gt;
#&gt;                        ...                  ...         ...
#&gt;   ENSG00000124334.17_PAR_Y OTTHUMG00000022720.1         PAR
#&gt;   ENSG00000185203.12_PAR_Y OTTHUMG00000022676.3         PAR
#&gt;    ENSG00000270726.6_PAR_Y OTTHUMG00000184987.2         PAR
#&gt;   ENSG00000182484.15_PAR_Y OTTHUMG00000022677.5         PAR
#&gt;    ENSG00000227159.8_PAR_Y OTTHUMG00000022678.1         PAR
#&gt;   -------
#&gt;   seqinfo: 374 sequences from an unspecified genome; no seqlengths</div><div class='input'><span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>colnames</a></span>(<span class='fu'>colData</span>(<span class='no'>rse_gene_ERP110066</span>))</div><div class='output co'>#&gt;   [1] "rail_id"                                                       
#&gt;   [2] "run_acc"                                                       
#&gt;   [3] "study_acc"                                                     
#&gt;   [4] "ncbi.sample_acc"                                               
#&gt;   [5] "ncbi.experiment_acc"                                           
#&gt;   [6] "ncbi.submission_acc"                                           
#&gt;   [7] "ncbi.submission_center"                                        
#&gt;   [8] "ncbi.submission_lab"                                           
#&gt;   [9] "ncbi.study_title"                                              
#&gt;  [10] "ncbi.study_abstract"                                           
#&gt;  [11] "ncbi.study_description"                                        
#&gt;  [12] "ncbi.experiment_title"                                         
#&gt;  [13] "ncbi.design_description"                                       
#&gt;  [14] "ncbi.sample_description"                                       
#&gt;  [15] "ncbi.library_name"                                             
#&gt;  [16] "ncbi.library_strategy"                                         
#&gt;  [17] "ncbi.library_source"                                           
#&gt;  [18] "ncbi.library_selection"                                        
#&gt;  [19] "ncbi.library_layout"                                           
#&gt;  [20] "ncbi.paired_nominal_length"                                    
#&gt;  [21] "ncbi.paired_nominal_stdev"                                     
#&gt;  [22] "ncbi.library_construction_protocol"                            
#&gt;  [23] "ncbi.platform_model"                                           
#&gt;  [24] "ncbi.sample_attributes"                                        
#&gt;  [25] "ncbi.experiment_attributes"                                    
#&gt;  [26] "ncbi.spot_length"                                              
#&gt;  [27] "ncbi.sample_name"                                              
#&gt;  [28] "ncbi.sample_title"                                             
#&gt;  [29] "ncbi.sample_bases"                                             
#&gt;  [30] "ncbi.sample_spots"                                             
#&gt;  [31] "ncbi.run_published"                                            
#&gt;  [32] "ncbi.size"                                                     
#&gt;  [33] "ncbi.run_total_bases"                                          
#&gt;  [34] "ncbi.run_total_spots"                                          
#&gt;  [35] "ncbi.num_reads"                                                
#&gt;  [36] "ncbi.num_spots"                                                
#&gt;  [37] "ncbi.read_info"                                                
#&gt;  [38] "ncbi.run_alias"                                                
#&gt;  [39] "ncbi.run_center_name"                                          
#&gt;  [40] "ncbi.run_broker_name"                                          
#&gt;  [41] "ncbi.run_center"                                               
#&gt;  [42] "recount_project.project"                                       
#&gt;  [43] "recount_project.organism"                                      
#&gt;  [44] "recount_project.file_source"                                   
#&gt;  [45] "recount_project.metadata_source"                               
#&gt;  [46] "recount_project.date_processed"                                
#&gt;  [47] "recount_qc.aligned_reads%.chrm"                                
#&gt;  [48] "recount_qc.aligned_reads%.chrx"                                
#&gt;  [49] "recount_qc.aligned_reads%.chry"                                
#&gt;  [50] "recount_qc.bc_auc.all_reads_all_bases"                         
#&gt;  [51] "recount_qc.bc_auc.all_reads_annotated_bases"                   
#&gt;  [52] "recount_qc.bc_auc.unique_reads_all_bases"                      
#&gt;  [53] "recount_qc.bc_auc.unique_reads_annotated_bases"                
#&gt;  [54] "recount_qc.bc_auc.all_%"                                       
#&gt;  [55] "recount_qc.bc_auc.unique_%"                                    
#&gt;  [56] "recount_qc.bc_frag.count"                                      
#&gt;  [57] "recount_qc.bc_frag.kallisto_count"                             
#&gt;  [58] "recount_qc.bc_frag.kallisto_mean_length"                       
#&gt;  [59] "recount_qc.bc_frag.mean_length"                                
#&gt;  [60] "recount_qc.bc_frag.mode_length"                                
#&gt;  [61] "recount_qc.bc_frag.mode_length_count"                          
#&gt;  [62] "recount_qc.exon_fc.all_%"                                      
#&gt;  [63] "recount_qc.exon_fc.unique_%"                                   
#&gt;  [64] "recount_qc.exon_fc_count_all.total"                            
#&gt;  [65] "recount_qc.exon_fc_count_all.assigned"                         
#&gt;  [66] "recount_qc.exon_fc_count_unique.total"                         
#&gt;  [67] "recount_qc.exon_fc_count_unique.assigned"                      
#&gt;  [68] "recount_qc.gene_fc.all_%"                                      
#&gt;  [69] "recount_qc.gene_fc.unique_%"                                   
#&gt;  [70] "recount_qc.gene_fc_count_all.total"                            
#&gt;  [71] "recount_qc.gene_fc_count_all.assigned"                         
#&gt;  [72] "recount_qc.gene_fc_count_unique.total"                         
#&gt;  [73] "recount_qc.gene_fc_count_unique.assigned"                      
#&gt;  [74] "recount_qc.intron_sum"                                         
#&gt;  [75] "recount_qc.intron_sum_%"                                       
#&gt;  [76] "recount_qc.star.%_of_chimeric_reads"                           
#&gt;  [77] "recount_qc.star.%_of_chimeric_reads2"                          
#&gt;  [78] "recount_qc.star.%_of_reads_mapped_to_multiple_loci"            
#&gt;  [79] "recount_qc.star.%_of_reads_mapped_to_multiple_loci2"           
#&gt;  [80] "recount_qc.star.%_of_reads_mapped_to_too_many_loci"            
#&gt;  [81] "recount_qc.star.%_of_reads_mapped_to_too_many_loci2"           
#&gt;  [82] "recount_qc.star.%_of_reads_unmapped:_other"                    
#&gt;  [83] "recount_qc.star.%_of_reads_unmapped:_other2"                   
#&gt;  [84] "recount_qc.star.%_of_reads_unmapped:_too_many_mismatches"      
#&gt;  [85] "recount_qc.star.%_of_reads_unmapped:_too_many_mismatches2"     
#&gt;  [86] "recount_qc.star.%_of_reads_unmapped:_too_short"                
#&gt;  [87] "recount_qc.star.%_of_reads_unmapped:_too_short2"               
#&gt;  [88] "recount_qc.star.all_mapped_reads"                              
#&gt;  [89] "recount_qc.star.all_mapped_reads2"                             
#&gt;  [90] "recount_qc.star.average_input_read_length"                     
#&gt;  [91] "recount_qc.star.average_input_read_length2"                    
#&gt;  [92] "recount_qc.star.average_mapped_length"                         
#&gt;  [93] "recount_qc.star.average_mapped_length2"                        
#&gt;  [94] "recount_qc.star.deletion_average_length"                       
#&gt;  [95] "recount_qc.star.deletion_average_length2"                      
#&gt;  [96] "recount_qc.star.deletion_rate_per_base"                        
#&gt;  [97] "recount_qc.star.deletion_rate_per_base2"                       
#&gt;  [98] "recount_qc.star.insertion_average_length"                      
#&gt;  [99] "recount_qc.star.insertion_average_length2"                     
#&gt; [100] "recount_qc.star.insertion_rate_per_base"                       
#&gt; [101] "recount_qc.star.insertion_rate_per_base2"                      
#&gt; [102] "recount_qc.star.mapping_speed,_million_of_reads_per_hour"      
#&gt; [103] "recount_qc.star.mapping_speed,_million_of_reads_per_hour2"     
#&gt; [104] "recount_qc.star.mismatch_rate_per_base,_%"                     
#&gt; [105] "recount_qc.star.mismatch_rate_per_base,_%2"                    
#&gt; [106] "recount_qc.star.number_of_chimeric_reads"                      
#&gt; [107] "recount_qc.star.number_of_chimeric_reads2"                     
#&gt; [108] "recount_qc.star.number_of_input_reads"                         
#&gt; [109] "recount_qc.star.number_of_input_reads2"                        
#&gt; [110] "recount_qc.star.number_of_reads_mapped_to_multiple_loci"       
#&gt; [111] "recount_qc.star.number_of_reads_mapped_to_multiple_loci2"      
#&gt; [112] "recount_qc.star.number_of_reads_mapped_to_too_many_loci"       
#&gt; [113] "recount_qc.star.number_of_reads_mapped_to_too_many_loci2"      
#&gt; [114] "recount_qc.star.number_of_reads_unmapped:_other"               
#&gt; [115] "recount_qc.star.number_of_reads_unmapped:_other2"              
#&gt; [116] "recount_qc.star.number_of_reads_unmapped:_too_many_mismatches" 
#&gt; [117] "recount_qc.star.number_of_reads_unmapped:_too_many_mismatches2"
#&gt; [118] "recount_qc.star.number_of_reads_unmapped:_too_short"           
#&gt; [119] "recount_qc.star.number_of_reads_unmapped:_too_short2"          
#&gt; [120] "recount_qc.star.number_of_splices:_at/ac"                      
#&gt; [121] "recount_qc.star.number_of_splices:_at/ac2"                     
#&gt; [122] "recount_qc.star.number_of_splices:_annotated_(sjdb)"           
#&gt; [123] "recount_qc.star.number_of_splices:_annotated_(sjdb)2"          
#&gt; [124] "recount_qc.star.number_of_splices:_gc/ag"                      
#&gt; [125] "recount_qc.star.number_of_splices:_gc/ag2"                     
#&gt; [126] "recount_qc.star.number_of_splices:_gt/ag"                      
#&gt; [127] "recount_qc.star.number_of_splices:_gt/ag2"                     
#&gt; [128] "recount_qc.star.number_of_splices:_non-canonical"              
#&gt; [129] "recount_qc.star.number_of_splices:_non-canonical2"             
#&gt; [130] "recount_qc.star.number_of_splices:_total"                      
#&gt; [131] "recount_qc.star.number_of_splices:_total2"                     
#&gt; [132] "recount_qc.star.uniquely_mapped_reads_%"                       
#&gt; [133] "recount_qc.star.uniquely_mapped_reads_%2"                      
#&gt; [134] "recount_qc.star.uniquely_mapped_reads_number"                  
#&gt; [135] "recount_qc.star.uniquely_mapped_reads_number2"                 
#&gt; [136] "recount_qc.junction_count"                                     
#&gt; [137] "recount_qc.junction_coverage"                                  
#&gt; [138] "recount_qc.junction_avg_coverage"                              
#&gt; [139] "BigWigURL"                                                     </div><div class='input'>
<span class='co'>## Create a second RSE object using another annotation</span>
<span class='no'>rse_gene_ERP110066_gencode_v29</span> <span class='kw'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='no'>proj_info</span>,
    <span class='kw'>annotation</span> <span class='kw'>=</span> <span class='st'>"gencode_v29"</span>
)</div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:36:56 downloading and reading the metadata</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.ncbi.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_project.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.recount_qc.ERP110066.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:36:58 downloading and reading the feature information</span></div><div class='output co'>#&gt; <span class='message'>Caching file human.gene_sums.G029.gtf.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/annotations/gene_sums/human.gene_sums.G029.gtf.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:36:59 downloading and reading the counts: 1024 samples across 64837 features.</span></div><div class='output co'>#&gt; <span class='message'>Caching file sra.gene_sums.ERP110066.G029.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/sra/gene_sums/66/ERP110066/sra.gene_sums.ERP110066.G029.gz'</span></div><div class='output co'>#&gt; <span class='message'>2020-05-27 01:37:04 construcing the RangedSummarizedExperiment (rse) object</span></div><div class='input'>
<span class='co'>## Doesn't work right now</span>
<span class='co'># rse_gene_ERP110066_refseq &lt;- create_rse(</span>
<span class='co'>#     proj_info,</span>
<span class='co'>#     annotation = "refseq"</span>
<span class='co'># )</span>

<span class='co'>## Create a RSE object at the exon level</span>
<span class='kw'>if</span> (<span class='fl'>FALSE</span>) {
<span class='no'>rse_exon_ERP110066</span> <span class='kw'>&lt;-</span> <span class='fu'>create_rse</span>(
    <span class='no'>proj_info</span>,
    <span class='kw'>type</span> <span class='kw'>=</span> <span class='st'>"exon"</span>
)

<span class='co'>## Explore the resulting RSE exon object</span>
<span class='no'>rse_exon_ERP110066</span>

<span class='fu'><a href='https://rdrr.io/r/base/dim.html'>dim</a></span>(<span class='no'>rse_exon_ERP110066</span>)
<span class='fu'>rowRanges</span>(<span class='no'>rse_exon_ERP110066</span>)
<span class='fu'>colData</span>(<span class='no'>rse_exon_ERP110066</span>)
<span class='fu'><a href='https://rdrr.io/r/base/colnames.html'>colnames</a></span>(<span class='fu'>colData</span>(<span class='no'>rse_exon_ERP110066</span>))
}</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.9000.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


