<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Read a metadata file — read_metadata • recount3</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Read a metadata file — read_metadata" />
<meta property="og:description" content="This function reads in a recount3 metadata file into R. You can first
locate the file using file_locate_url() then download it to your computer
using file_retrieve()." />
<meta property="og:image" content="/logo.png" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LieberInstitute/recount3/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Read a metadata file</h1>
    <small class="dont-index">Source: <a href='https://github.com/LieberInstitute/recount3/blob/master/R/read_metadata.R'><code>R/read_metadata.R</code></a></small>
    <div class="hidden name"><code>read_metadata.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function reads in a <code>recount3</code> metadata file into R. You can first
locate the file using <code><a href='file_locate_url.html'>file_locate_url()</a></code> then download it to your computer
using <code><a href='file_retrieve.html'>file_retrieve()</a></code>.</p>
    </div>

    <pre class="usage"><span class='fu'>read_metadata</span>(<span class='no'>metadata_file</span>)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>metadata_file</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> with the local path to a <code>recount3</code>
metadata file.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A <code><a href='https://rdrr.io/r/base/data.frame.html'>data.frame()</a></code> with all lower case column names for the sample
metadata.</p>
    <h2 class="hasAnchor" id="see-also"><a class="anchor" href="#see-also"></a>See also</h2>

    <div class='dont-index'><p>Other internal functions for accessing the recount3 data: 
<code><a href='annotation_ext.html'>annotation_ext</a>()</code>,
<code><a href='create_rse_manual.html'>create_rse_manual</a>()</code>,
<code><a href='file_locate_url_annotation.html'>file_locate_url_annotation</a>()</code>,
<code><a href='file_locate_url.html'>file_locate_url</a>()</code>,
<code><a href='file_retrieve.html'>file_retrieve</a>()</code>,
<code><a href='project_home_available.html'>project_home_available</a>()</code>,
<code><a href='read_counts.html'>read_counts</a>()</code></p></div>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='co'>## Download the metadata file for project ERP110066</span>
<span class='no'>url_ERP110066_meta</span> <span class='kw'>&lt;-</span> <span class='fu'><a href='file_locate_url.html'>file_locate_url</a></span>(<span class='st'>"ERP110066"</span>, <span class='st'>"data_sources/sra"</span>)
<span class='no'>local_ERP110066_meta</span> <span class='kw'>&lt;-</span> <span class='fu'><a href='file_retrieve.html'>file_retrieve</a></span>(<span class='kw'>url</span> <span class='kw'>=</span> <span class='no'>url_ERP110066_meta</span>)</div><div class='output co'>#&gt; <span class='message'>Caching file sra.metadata.ERP110066.MD.gz</span></div><div class='input'>
<span class='co'>## Read the metadata</span>
<span class='no'>ERP110066_meta</span> <span class='kw'>&lt;-</span> <span class='fu'>read_metadata</span>(<span class='no'>local_ERP110066_meta</span>)
<span class='fu'><a href='https://rdrr.io/r/utils/head.html'>head</a></span>(<span class='no'>ERP110066_meta</span>)</div><div class='output co'>#&gt;   rail_id    run_acc study_acc sample_acc experiment_acc submission_acc
#&gt; 1  526388 ERR2713106 ERP110066 ERS2624036     ERX2727275     ERA1552443
#&gt; 2  526396 ERR2713107 ERP110066 ERS2624037     ERX2727276     ERA1552443
#&gt; 3  526404 ERR2713108 ERP110066 ERS2624038     ERX2727277     ERA1552443
#&gt; 4  526412 ERR2713109 ERP110066 ERS2624039     ERX2727278     ERA1552443
#&gt; 5  526468 ERR2713110 ERP110066 ERS2624040     ERX2727279     ERA1552443
#&gt; 6  526476 ERR2713111 ERP110066 ERS2624041     ERX2727280     ERA1552443
#&gt;   submission_center              submission_lab
#&gt; 1                NA European Nucleotide Archive
#&gt; 2                NA European Nucleotide Archive
#&gt; 3                NA European Nucleotide Archive
#&gt; 4                NA European Nucleotide Archive
#&gt; 5                NA European Nucleotide Archive
#&gt; 6                NA European Nucleotide Archive
#&gt;                                             study_title
#&gt; 1 Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 2 Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 3 Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 4 Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 5 Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 6 Endoderm Differentiation - Single Cell RNA-sequencing
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                    study_abstract
#&gt; 1 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 2 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 3 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 4 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 5 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 6 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                 study_description
#&gt; 1 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 2 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 3 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 4 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 5 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt; 6 Single-cell RNA-seq (scRNA-seq) on nocodazole and DMSO treated cells before and after differentiation into endoderm. hPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours and induced to differentiate into definitive endoderm for three days. Single cells were subsequently collected in either undifferentiated conditions (Und) or after 3 days of endoderm differentiation (Endoderm) and then sorted onto 384 well plates for Smart-Seq2 processing.
#&gt;                                                                                   experiment_title
#&gt; 1 Illumina HiSeq 4000 paired end sequencing; Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 2 Illumina HiSeq 4000 paired end sequencing; Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 3 Illumina HiSeq 4000 paired end sequencing; Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 4 Illumina HiSeq 4000 paired end sequencing; Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 5 Illumina HiSeq 4000 paired end sequencing; Endoderm Differentiation - Single Cell RNA-sequencing
#&gt; 6 Illumina HiSeq 4000 paired end sequencing; Endoderm Differentiation - Single Cell RNA-sequencing
#&gt;                                      design_description sample_description
#&gt; 1 Endoderm Differentiation - Single Cell RNA-sequencing                 NA
#&gt; 2 Endoderm Differentiation - Single Cell RNA-sequencing                 NA
#&gt; 3 Endoderm Differentiation - Single Cell RNA-sequencing                 NA
#&gt; 4 Endoderm Differentiation - Single Cell RNA-sequencing                 NA
#&gt; 5 Endoderm Differentiation - Single Cell RNA-sequencing                 NA
#&gt; 6 Endoderm Differentiation - Single Cell RNA-sequencing                 NA
#&gt;        library_name library_strategy library_source library_selection
#&gt; 1  0hS_22728_4_10_p          RNA-Seq TRANSCRIPTOMIC             PolyA
#&gt; 2 0hS_22728_4_105_p          RNA-Seq TRANSCRIPTOMIC             PolyA
#&gt; 3 0hS_22728_4_106_p          RNA-Seq TRANSCRIPTOMIC             PolyA
#&gt; 4 0hS_22728_4_107_p          RNA-Seq TRANSCRIPTOMIC             PolyA
#&gt; 5 0hS_22728_4_108_p          RNA-Seq TRANSCRIPTOMIC             PolyA
#&gt; 6 0hS_22728_4_109_p          RNA-Seq TRANSCRIPTOMIC             PolyA
#&gt;   library_layout paired_nominal_length paired_nominal_stdev
#&gt; 1         paired                   400                    0
#&gt; 2         paired                   400                    0
#&gt; 3         paired                   400                    0
#&gt; 4         paired                   400                    0
#&gt; 5         paired                   400                    0
#&gt; 6         paired                   400                    0
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     library_construction_protocol
#&gt; 1 Cells  were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo – TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit – Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.
#&gt; 2 Cells  were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo – TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit – Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.
#&gt; 3 Cells  were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo – TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit – Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.
#&gt; 4 Cells  were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo – TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit – Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.
#&gt; 5 Cells  were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo – TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit – Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.
#&gt; 6 Cells  were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo – TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit – Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.
#&gt;        platform_model
#&gt; 1 Illumina HiSeq 4000
#&gt; 2 Illumina HiSeq 4000
#&gt; 3 Illumina HiSeq 4000
#&gt; 4 Illumina HiSeq 4000
#&gt; 5 Illumina HiSeq 4000
#&gt; 6 Illumina HiSeq 4000
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                sample_attributes
#&gt; 1   Alias;;E-MTAB-7008:0hS_22728_4_10|Broker name;;ArrayExpress|cell_line;;HPSI0813i-fpdm_2|cell_type;;induced pluripotent stem cell|Description;;Protocols: Cells were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo - TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit - Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.|developmental stage;;adult|disease;;normal|ENA checklist;;ERC000011|genotype;;wild type genotype|growth condition;;normal culture to maintain pluripotency|INSDC center alias;;Wellcome Sanger Institute|INSDC center name;;Wellcome Sanger Institute|INSDC first public;;2019-06-16T17:03:41Z|INSDC last update;;2018-07-25T16:48:03Z|INSDC status;;public|organism part;;blood|organism;;Homo sapiens|progenitor cell type;;blood cell|Sample Name;;ERS2624036|sex;;male|SRA accession;;ERS2624036|stimulus;;synchronization|Title;;0hS_22728_4_10
#&gt; 2 Alias;;E-MTAB-7008:0hS_22728_4_105|Broker name;;ArrayExpress|cell_line;;HPSI0813i-fpdm_2|cell_type;;induced pluripotent stem cell|Description;;Protocols: Cells were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo - TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit - Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.|developmental stage;;adult|disease;;normal|ENA checklist;;ERC000011|genotype;;wild type genotype|growth condition;;normal culture to maintain pluripotency|INSDC center alias;;Wellcome Sanger Institute|INSDC center name;;Wellcome Sanger Institute|INSDC first public;;2019-06-16T17:03:41Z|INSDC last update;;2018-07-25T16:48:03Z|INSDC status;;public|organism part;;blood|organism;;Homo sapiens|progenitor cell type;;blood cell|Sample Name;;ERS2624037|sex;;male|SRA accession;;ERS2624037|stimulus;;synchronization|Title;;0hS_22728_4_105
#&gt; 3 Alias;;E-MTAB-7008:0hS_22728_4_106|Broker name;;ArrayExpress|cell_line;;HPSI0813i-fpdm_2|cell_type;;induced pluripotent stem cell|Description;;Protocols: Cells were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo - TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit - Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.|developmental stage;;adult|disease;;normal|ENA checklist;;ERC000011|genotype;;wild type genotype|growth condition;;normal culture to maintain pluripotency|INSDC center alias;;Wellcome Sanger Institute|INSDC center name;;Wellcome Sanger Institute|INSDC first public;;2019-06-16T17:03:41Z|INSDC last update;;2018-07-25T16:48:03Z|INSDC status;;public|organism part;;blood|organism;;Homo sapiens|progenitor cell type;;blood cell|Sample Name;;ERS2624038|sex;;male|SRA accession;;ERS2624038|stimulus;;synchronization|Title;;0hS_22728_4_106
#&gt; 4 Alias;;E-MTAB-7008:0hS_22728_4_107|Broker name;;ArrayExpress|cell_line;;HPSI0813i-fpdm_2|cell_type;;induced pluripotent stem cell|Description;;Protocols: Cells were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo - TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit - Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.|developmental stage;;adult|disease;;normal|ENA checklist;;ERC000011|genotype;;wild type genotype|growth condition;;normal culture to maintain pluripotency|INSDC center alias;;Wellcome Sanger Institute|INSDC center name;;Wellcome Sanger Institute|INSDC first public;;2019-06-16T17:03:41Z|INSDC last update;;2018-07-25T16:48:03Z|INSDC status;;public|organism part;;blood|organism;;Homo sapiens|progenitor cell type;;blood cell|Sample Name;;ERS2624039|sex;;male|SRA accession;;ERS2624039|stimulus;;synchronization|Title;;0hS_22728_4_107
#&gt; 5 Alias;;E-MTAB-7008:0hS_22728_4_108|Broker name;;ArrayExpress|cell_line;;HPSI0813i-fpdm_2|cell_type;;induced pluripotent stem cell|Description;;Protocols: Cells were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo - TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit - Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.|developmental stage;;adult|disease;;normal|ENA checklist;;ERC000011|genotype;;wild type genotype|growth condition;;normal culture to maintain pluripotency|INSDC center alias;;Wellcome Sanger Institute|INSDC center name;;Wellcome Sanger Institute|INSDC first public;;2019-06-16T17:03:41Z|INSDC last update;;2018-07-25T16:48:03Z|INSDC status;;public|organism part;;blood|organism;;Homo sapiens|progenitor cell type;;blood cell|Sample Name;;ERS2624040|sex;;male|SRA accession;;ERS2624040|stimulus;;synchronization|Title;;0hS_22728_4_108
#&gt; 6 Alias;;E-MTAB-7008:0hS_22728_4_109|Broker name;;ArrayExpress|cell_line;;HPSI0813i-fpdm_2|cell_type;;induced pluripotent stem cell|Description;;Protocols: Cells were treated with Accutase and, once obtained single cells, were filtered (40um strainers), counted and resuspended in PBS/BSA (at 1%) at 10^6/mL to sort them into 384-well plates (containing the lysis buffer and ERCCs). The hiPSC line FSPS13B was plated on vitronectin coated plates (10µg/ml, Stem Cell Technologies) and cultured in E6 media supplemented with 2ng/ml TGF-β (R&amp;D) and 25ng/ml FGF2 making complete E8 media. Cells were maintained by weekly passaging using 0.5mM EDTA (Thermo Fisher Scientific). hiPSC colonies were treated with DMSO or 100ng/ml nocodazole for 16 hours. Following synchronisation, cells were maintained in pluripotency conditions or induced to form definitive endoderm using a 3-day protocol. On day 1 cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4, 100ng/ml Activin A and 3µM CHIR99021. On day 2 the cells were cultured in CDM-PVA supplemented with 80ng/ml FGF2, 10µM LY294002, 10ng/ml BMP4 and 100ng/ml Activin A. On day 3 the cells were cultured in RPMI-B27 media supplemented with 80ng/ml FGF2 and 100ng/ml Activin. Conversion was performed following the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) using a poly-Dt30 oligo and a TSO oligo. RNA extraction is based on the Picelli et al SS2 protocol, using a polydT primer plus the template switching oligo - TSO, as published (https://www.nature.com/articles/nprot.2014.006). Cell sorting and library preparation was carried out by the sequencing core facility at the Wellcome Sanger Institute. Single hPSCs were isolated into 384 wells plates and libraries were prepared for 120 cells/per condition using the SMART-Seq2 protocol (https://www.nature.com/articles/nprot.2014.006) and using Nextera XT DNA Library Preparation Kit - Illumina. A constant amount of spike-in RNA from the External RNA Controls Consortium (ERCC) was also added to the lysis buffer prior to sorting.|developmental stage;;adult|disease;;normal|ENA checklist;;ERC000011|genotype;;wild type genotype|growth condition;;normal culture to maintain pluripotency|INSDC center alias;;Wellcome Sanger Institute|INSDC center name;;Wellcome Sanger Institute|INSDC first public;;2019-06-16T17:03:42Z|INSDC last update;;2018-07-25T16:48:04Z|INSDC status;;public|organism part;;blood|organism;;Homo sapiens|progenitor cell type;;blood cell|Sample Name;;ERS2624041|sex;;male|SRA accession;;ERS2624041|stimulus;;synchronization|Title;;0hS_22728_4_109
#&gt;                                                                                                                                                               experiment_attributes
#&gt; 1  Experimental Factor: compound;;nocodazole|Experimental Factor: dose;;100|Experimental Factor: sampling time point;;0|Experimental Factor: single cell identifier;;0hS_22728_4_10
#&gt; 2 Experimental Factor: compound;;nocodazole|Experimental Factor: dose;;100|Experimental Factor: sampling time point;;0|Experimental Factor: single cell identifier;;0hS_22728_4_105
#&gt; 3 Experimental Factor: compound;;nocodazole|Experimental Factor: dose;;100|Experimental Factor: sampling time point;;0|Experimental Factor: single cell identifier;;0hS_22728_4_106
#&gt; 4 Experimental Factor: compound;;nocodazole|Experimental Factor: dose;;100|Experimental Factor: sampling time point;;0|Experimental Factor: single cell identifier;;0hS_22728_4_107
#&gt; 5 Experimental Factor: compound;;nocodazole|Experimental Factor: dose;;100|Experimental Factor: sampling time point;;0|Experimental Factor: single cell identifier;;0hS_22728_4_108
#&gt; 6 Experimental Factor: compound;;nocodazole|Experimental Factor: dose;;100|Experimental Factor: sampling time point;;0|Experimental Factor: single cell identifier;;0hS_22728_4_109
#&gt;   spot_length  sample_name                sample_title sample_bases
#&gt; 1         150 SAMEA4804123  E-MTAB-7008:0hS_22728_4_10     49258200
#&gt; 2         150 SAMEA4804124 E-MTAB-7008:0hS_22728_4_105     14499000
#&gt; 3         150 SAMEA4804125 E-MTAB-7008:0hS_22728_4_106     95237100
#&gt; 4         150 SAMEA4804126 E-MTAB-7008:0hS_22728_4_107     12761700
#&gt; 5         150 SAMEA4804127 E-MTAB-7008:0hS_22728_4_108    117535950
#&gt; 6         150 SAMEA4804128 E-MTAB-7008:0hS_22728_4_109     97473900
#&gt;   sample_spots       run_published     size run_total_bases run_total_spots
#&gt; 1       328388 2019-06-17 08:03:32 19387330        49258200          328388
#&gt; 2        96660 2019-06-17 08:03:32  5895708        14499000           96660
#&gt; 3       634914 2019-06-17 08:03:32 39337877        95237100          634914
#&gt; 4        85078 2019-06-17 08:03:32  5194717        12761700           85078
#&gt; 5       783573 2019-06-17 08:03:32 48515159       117535950          783573
#&gt; 6       649826 2019-06-17 08:03:32 39893587        97473900          649826
#&gt;   num_reads num_spots
#&gt; 1         2    328388
#&gt; 2         2     96660
#&gt; 3         2    634914
#&gt; 4         2     85078
#&gt; 5         2    783573
#&gt; 6         2    649826
#&gt;                                                                         read_info
#&gt; 1 index:0,count:328388,average:75,stdev:0|index:1,count:328388,average:75,stdev:0
#&gt; 2   index:0,count:96660,average:75,stdev:0|index:1,count:96660,average:75,stdev:0
#&gt; 3 index:0,count:634914,average:75,stdev:0|index:1,count:634914,average:75,stdev:0
#&gt; 4   index:0,count:85078,average:75,stdev:0|index:1,count:85078,average:75,stdev:0
#&gt; 5 index:0,count:783573,average:75,stdev:0|index:1,count:783573,average:75,stdev:0
#&gt; 6 index:0,count:649826,average:75,stdev:0|index:1,count:649826,average:75,stdev:0
#&gt;                      run_alias           run_center_name run_broker_name
#&gt; 1  E-MTAB-7008:0hS_22728_4_10_ Wellcome Sanger Institute    ArrayExpress
#&gt; 2 E-MTAB-7008:0hS_22728_4_105_ Wellcome Sanger Institute    ArrayExpress
#&gt; 3 E-MTAB-7008:0hS_22728_4_106_ Wellcome Sanger Institute    ArrayExpress
#&gt; 4 E-MTAB-7008:0hS_22728_4_107_ Wellcome Sanger Institute    ArrayExpress
#&gt; 5 E-MTAB-7008:0hS_22728_4_108_ Wellcome Sanger Institute    ArrayExpress
#&gt; 6 E-MTAB-7008:0hS_22728_4_109_ Wellcome Sanger Institute    ArrayExpress
#&gt;   run_center aligned_reads%.chrm aligned_reads%.chrx aligned_reads%.chry
#&gt; 1         SC                0.01                0.13                0.01
#&gt; 2         SC                0.58                0.24                0.01
#&gt; 3         SC                4.69                1.54                0.08
#&gt; 4         SC                0.58                0.26                0.01
#&gt; 5         SC                4.26                1.73                0.08
#&gt; 6         SC                4.09                2.28                0.19
#&gt;   bc_auc.all_reads_all_bases bc_auc.all_reads_annotated_bases
#&gt; 1                   44344618                         45347466
#&gt; 2                   12969528                         13302507
#&gt; 3                   89662462                        112112367
#&gt; 4                   11403396                         11688165
#&gt; 5                  111182337                        140112241
#&gt; 6                   89229648                        109930694
#&gt;   bc_auc.unique_reads_all_bases bc_auc.unique_reads_annotated_bases
#&gt; 1                      43467166                            44747846
#&gt; 2                      12852528                            13202960
#&gt; 3                      74980409                           102695715
#&gt; 4                      11305148                            11606039
#&gt; 5                      93642370                           128573987
#&gt; 6                      80904057                           103065773
#&gt;   bc_auc.all_% bc_auc.unique_% bc_frag.count bc_frag.kallisto_count
#&gt; 1       102.26          102.95        303856                 294125
#&gt; 2       102.57          102.73         88769                  86650
#&gt; 3       125.04          136.96        611231                 475649
#&gt; 4       102.50          102.66         78035                  76259
#&gt; 5       126.02          137.30        758261                 575377
#&gt; 6       123.20          127.39        612754                 464493
#&gt;   bc_frag.kallisto_mean_length bc_frag.mean_length bc_frag.mode_length
#&gt; 1                      321.277             552.146                 230
#&gt; 2                      335.648             595.274                 230
#&gt; 3                      371.482            2085.693                 296
#&gt; 4                      335.415             548.169                 230
#&gt; 5                      364.896            2203.148                 257
#&gt; 6                      341.423            2314.059                 246
#&gt;   bc_frag.mode_length_count exon_fc.all_% exon_fc.unique_%
#&gt; 1                      2282         97.78            99.74
#&gt; 2                       647         98.23            99.13
#&gt; 3                      1524         69.84            83.54
#&gt; 4                       586         98.29            99.15
#&gt; 5                      1990         70.61            83.84
#&gt; 6                      1690         67.74            74.72
#&gt;   exon_fc_count_all.total exon_fc_count_all.assigned exon_fc_count_unique.total
#&gt; 1                  333112                     297102                     333112
#&gt; 2                   91743                      87196                      91743
#&gt; 3                 1126389                     426876                    1126389
#&gt; 4                   80679                      76704                      80679
#&gt; 5                 1358281                     535378                    1358281
#&gt; 6                  829238                     415091                     829238
#&gt;   exon_fc_count_unique.assigned gene_fc.all_% gene_fc.unique_%
#&gt; 1                        297102         98.27            99.35
#&gt; 2                         87196         98.40            98.80
#&gt; 3                        426876         73.78            79.86
#&gt; 4                         76704         98.47            98.85
#&gt; 5                        535378         74.22            79.96
#&gt; 6                        415091         69.00            70.69
#&gt;   gene_fc_count_all.total gene_fc_count_all.assigned gene_fc_count_unique.total
#&gt; 1                  333112                     298607                     333112
#&gt; 2                   91743                      87352                      91743
#&gt; 3                 1126389                     450992                    1126389
#&gt; 4                   80679                      76843                      80679
#&gt; 5                 1358281                     562756                    1358281
#&gt; 6                  829238                     422811                     829238
#&gt;   gene_fc_count_unique.assigned intron_sum intron_sum_%
#&gt; 1                        295944      36893         0.08
#&gt; 2                         86907      55865         0.43
#&gt; 3                        408069    7345392         8.19
#&gt; 4                         76470      43850         0.38
#&gt; 5                        510601    9054481         8.14
#&gt; 6                        392677   10351490        11.60
#&gt;   star.%_of_chimeric_reads star.%_of_chimeric_reads2
#&gt; 1                     0.53                         0
#&gt; 2                     0.42                         0
#&gt; 3                     1.26                         0
#&gt; 4                     0.40                         0
#&gt; 5                     1.05                         0
#&gt; 6                     1.47                         0
#&gt;   star.%_of_reads_mapped_to_multiple_loci
#&gt; 1                                    1.82
#&gt; 2                                    0.83
#&gt; 3                                   15.79
#&gt; 4                                    0.79
#&gt; 5                                   15.28
#&gt; 6                                    8.81
#&gt;   star.%_of_reads_mapped_to_multiple_loci2
#&gt; 1                                        0
#&gt; 2                                        0
#&gt; 3                                        0
#&gt; 4                                        0
#&gt; 5                                        0
#&gt; 6                                        0
#&gt;   star.%_of_reads_mapped_to_too_many_loci
#&gt; 1                                    0.00
#&gt; 2                                    0.01
#&gt; 3                                    0.09
#&gt; 4                                    0.00
#&gt; 5                                    0.07
#&gt; 6                                    0.10
#&gt;   star.%_of_reads_mapped_to_too_many_loci2 star.%_of_reads_unmapped:_other
#&gt; 1                                        0                            0.00
#&gt; 2                                        0                            0.00
#&gt; 3                                        0                            0.05
#&gt; 4                                        0                            0.00
#&gt; 5                                        0                            0.04
#&gt; 6                                        0                            0.12
#&gt;   star.%_of_reads_unmapped:_other2
#&gt; 1                                0
#&gt; 2                                0
#&gt; 3                                0
#&gt; 4                                0
#&gt; 5                                0
#&gt; 6                                0
#&gt;   star.%_of_reads_unmapped:_too_many_mismatches
#&gt; 1                                             0
#&gt; 2                                             0
#&gt; 3                                             0
#&gt; 4                                             0
#&gt; 5                                             0
#&gt; 6                                             0
#&gt;   star.%_of_reads_unmapped:_too_many_mismatches2
#&gt; 1                                              0
#&gt; 2                                              0
#&gt; 3                                              0
#&gt; 4                                              0
#&gt; 5                                              0
#&gt; 6                                              0
#&gt;   star.%_of_reads_unmapped:_too_short star.%_of_reads_unmapped:_too_short2
#&gt; 1                                7.47                                    0
#&gt; 2                                8.16                                    0
#&gt; 3                                3.59                                    0
#&gt; 4                                8.27                                    0
#&gt; 5                                3.12                                    0
#&gt; 6                                5.49                                    0
#&gt;   star.all_mapped_reads star.all_mapped_reads2 star.average_input_read_length
#&gt; 1                303856                      0                            150
#&gt; 2                 88769                      0                            150
#&gt; 3                611231                      0                            150
#&gt; 4                 78035                      0                            150
#&gt; 5                758261                      0                            150
#&gt; 6                612754                      0                            150
#&gt;   star.average_input_read_length2 star.average_mapped_length
#&gt; 1                               0                     147.58
#&gt; 2                               0                     147.51
#&gt; 3                               0                     148.13
#&gt; 4                               0                     147.53
#&gt; 5                               0                     148.20
#&gt; 6                               0                     147.68
#&gt;   star.average_mapped_length2 star.deletion_average_length
#&gt; 1                           0                         1.47
#&gt; 2                           0                         1.40
#&gt; 3                           0                         1.46
#&gt; 4                           0                         1.37
#&gt; 5                           0                         1.47
#&gt; 6                           0                         1.49
#&gt;   star.deletion_average_length2 star.deletion_rate_per_base
#&gt; 1                             0                        0.01
#&gt; 2                             0                        0.01
#&gt; 3                             0                        0.01
#&gt; 4                             0                        0.01
#&gt; 5                             0                        0.01
#&gt; 6                             0                        0.02
#&gt;   star.deletion_rate_per_base2 star.insertion_average_length
#&gt; 1                            0                          1.10
#&gt; 2                            0                          1.15
#&gt; 3                            0                          1.26
#&gt; 4                            0                          1.17
#&gt; 5                            0                          1.26
#&gt; 6                            0                          1.27
#&gt;   star.insertion_average_length2 star.insertion_rate_per_base
#&gt; 1                              0                         0.01
#&gt; 2                              0                         0.01
#&gt; 3                              0                         0.01
#&gt; 4                              0                         0.01
#&gt; 5                              0                         0.01
#&gt; 6                              0                         0.01
#&gt;   star.insertion_rate_per_base2 star.mapping_speed,_million_of_reads_per_hour
#&gt; 1                             0                                        107.47
#&gt; 2                             0                                         58.00
#&gt; 3                             0                                        120.30
#&gt; 4                             0                                         34.03
#&gt; 5                             0                                        156.71
#&gt; 6                             0                                        111.40
#&gt;   star.mapping_speed,_million_of_reads_per_hour2 star.mismatch_rate_per_base,_%
#&gt; 1                                              0                           0.15
#&gt; 2                                              0                           0.17
#&gt; 3                                              0                           0.27
#&gt; 4                                              0                           0.18
#&gt; 5                                              0                           0.27
#&gt; 6                                              0                           0.30
#&gt;   star.mismatch_rate_per_base,_%2 star.number_of_chimeric_reads
#&gt; 1                               0                          1755
#&gt; 2                               0                           408
#&gt; 3                               0                          8018
#&gt; 4                               0                           343
#&gt; 5                               0                          8198
#&gt; 6                               0                          9526
#&gt;   star.number_of_chimeric_reads2 star.number_of_input_reads
#&gt; 1                              0                     328388
#&gt; 2                              0                      96660
#&gt; 3                              0                     634914
#&gt; 4                              0                      85078
#&gt; 5                              0                     783573
#&gt; 6                              0                     649826
#&gt;   star.number_of_input_reads2 star.number_of_reads_mapped_to_multiple_loci
#&gt; 1                           0                                         5972
#&gt; 2                           0                                          807
#&gt; 3                           0                                       100238
#&gt; 4                           0                                          674
#&gt; 5                           0                                       119705
#&gt; 6                           0                                        57228
#&gt;   star.number_of_reads_mapped_to_multiple_loci2
#&gt; 1                                             0
#&gt; 2                                             0
#&gt; 3                                             0
#&gt; 4                                             0
#&gt; 5                                             0
#&gt; 6                                             0
#&gt;   star.number_of_reads_mapped_to_too_many_loci
#&gt; 1                                           11
#&gt; 2                                            7
#&gt; 3                                          562
#&gt; 4                                            3
#&gt; 5                                          578
#&gt; 6                                          655
#&gt;   star.number_of_reads_mapped_to_too_many_loci2
#&gt; 1                                             0
#&gt; 2                                             0
#&gt; 3                                             0
#&gt; 4                                             0
#&gt; 5                                             0
#&gt; 6                                             0
#&gt;   star.number_of_reads_unmapped:_other star.number_of_reads_unmapped:_other2
#&gt; 1                                    2                                     0
#&gt; 2                                    1                                     0
#&gt; 3                                  324                                     0
#&gt; 4                                    2                                     0
#&gt; 5                                  307                                     0
#&gt; 6                                  760                                     0
#&gt;   star.number_of_reads_unmapped:_too_many_mismatches
#&gt; 1                                                  0
#&gt; 2                                                  0
#&gt; 3                                                  0
#&gt; 4                                                  0
#&gt; 5                                                  0
#&gt; 6                                                  0
#&gt;   star.number_of_reads_unmapped:_too_many_mismatches2
#&gt; 1                                                   0
#&gt; 2                                                   0
#&gt; 3                                                   0
#&gt; 4                                                   0
#&gt; 5                                                   0
#&gt; 6                                                   0
#&gt;   star.number_of_reads_unmapped:_too_short
#&gt; 1                                    24519
#&gt; 2                                     7883
#&gt; 3                                    22797
#&gt; 4                                     7038
#&gt; 5                                    24427
#&gt; 6                                    35657
#&gt;   star.number_of_reads_unmapped:_too_short2 star.number_of_splices:_at/ac
#&gt; 1                                         0                            29
#&gt; 2                                         0                             0
#&gt; 3                                         0                            75
#&gt; 4                                         0                             1
#&gt; 5                                         0                           115
#&gt; 6                                         0                            82
#&gt;   star.number_of_splices:_at/ac2 star.number_of_splices:_annotated_(sjdb)
#&gt; 1                              0                                        0
#&gt; 2                              0                                        0
#&gt; 3                              0                                        0
#&gt; 4                              0                                        0
#&gt; 5                              0                                        0
#&gt; 6                              0                                        0
#&gt;   star.number_of_splices:_annotated_(sjdb)2 star.number_of_splices:_gc/ag
#&gt; 1                                         0                            86
#&gt; 2                                         0                            16
#&gt; 3                                         0                           761
#&gt; 4                                         0                            15
#&gt; 5                                         0                          1148
#&gt; 6                                         0                           888
#&gt;   star.number_of_splices:_gc/ag2 star.number_of_splices:_gt/ag
#&gt; 1                              0                          9504
#&gt; 2                              0                          2151
#&gt; 3                              0                        132445
#&gt; 4                              0                          1871
#&gt; 5                              0                        185981
#&gt; 6                              0                        152591
#&gt;   star.number_of_splices:_gt/ag2 star.number_of_splices:_non-canonical
#&gt; 1                              0                                   135
#&gt; 2                              0                                    19
#&gt; 3                              0                                   907
#&gt; 4                              0                                    15
#&gt; 5                              0                                  1108
#&gt; 6                              0                                  1003
#&gt;   star.number_of_splices:_non-canonical2 star.number_of_splices:_total
#&gt; 1                                      0                          9754
#&gt; 2                                      0                          2186
#&gt; 3                                      0                        134188
#&gt; 4                                      0                          1902
#&gt; 5                                      0                        188352
#&gt; 6                                      0                        154564
#&gt;   star.number_of_splices:_total2 star.uniquely_mapped_reads_%
#&gt; 1                              0                        90.71
#&gt; 2                              0                        91.00
#&gt; 3                              0                        80.48
#&gt; 4                              0                        90.93
#&gt; 5                              0                        81.49
#&gt; 6                              0                        85.49
#&gt;   star.uniquely_mapped_reads_%2 star.uniquely_mapped_reads_number
#&gt; 1                             0                            297884
#&gt; 2                             0                             87962
#&gt; 3                             0                            510993
#&gt; 4                             0                             77361
#&gt; 5                             0                            638556
#&gt; 6                             0                            555526
#&gt;   star.uniquely_mapped_reads_number2 junction_count junction_coverage
#&gt; 1                                  0           1054             10064
#&gt; 2                                  0           1879              2197
#&gt; 3                                  0          34809            137235
#&gt; 4                                  0           1604              1895
#&gt; 5                                  0          42691            192409
#&gt; 6                                  0          30762            157827
#&gt;   junction_avg_coverage
#&gt; 1              9.548387
#&gt; 2              1.169239
#&gt; 3              3.942515
#&gt; 4              1.181421
#&gt; 5              4.507016
#&gt; 6              5.130583</div><div class='input'>
<span class='co'>## Locate and read the GTEx bladder metadata</span>
<span class='no'>gtex_bladder_meta</span> <span class='kw'>&lt;-</span> <span class='fu'>read_metadata</span>(
    <span class='fu'><a href='file_retrieve.html'>file_retrieve</a></span>(
        <span class='fu'><a href='file_locate_url.html'>file_locate_url</a></span>(<span class='st'>"BLADDER"</span>, <span class='st'>"data_sources/gtex"</span>)
    )
)</div><div class='output co'>#&gt; <span class='message'>Caching file gtex.metadata.BLADDER.MD.gz</span></div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/gtex/metadata/ER/BLADDER/gtex.metadata.BLADDER.MD.gz'</span></div><div class='input'>
<span class='fu'><a href='https://rdrr.io/r/utils/head.html'>head</a></span>(<span class='no'>gtex_bladder_meta</span>)</div><div class='output co'>#&gt;   rail_id    run_acc              rail_barcode    subjid sex   age dthhrdy
#&gt; 1  153881 SRR1071717 GTEX-U3ZM-0826-SM-4DXU6.1 GTEX-U3ZM   1 40-49       0
#&gt; 2  309816 SRR1079830 GTEX-U4B1-1226-SM-4DXT7.1 GTEX-U4B1   1 40-49       0
#&gt; 3  270898 SRR1081765 GTEX-S4Q7-0926-SM-4AD5D.1 GTEX-S4Q7   1 20-29       0
#&gt; 4  615530 SRR1085402 GTEX-SNMC-0826-SM-4DM66.1 GTEX-SNMC   1 20-29       0
#&gt; 5  607337 SRR1086236 GTEX-TMMY-1526-SM-4DXST.1 GTEX-TMMY   2 40-49       0
#&gt; 6  277053 SRR1092208 GTEX-SE5C-1026-SM-4BRUG.1 GTEX-SE5C   2 40-49       0
#&gt;                    sampid smatsscr smcenter
#&gt; 1 GTEX-U3ZM-0826-SM-4DXU6        2       B1
#&gt; 2 GTEX-U4B1-1226-SM-4DXT7        1       B1
#&gt; 3 GTEX-S4Q7-0926-SM-4AD5D        1       B1
#&gt; 4 GTEX-SNMC-0826-SM-4DM66        2       B1
#&gt; 5 GTEX-TMMY-1526-SM-4DXST        2       C1
#&gt; 6 GTEX-SE5C-1026-SM-4BRUG        1       B1
#&gt;                                                                                                   smpthnts
#&gt; 1 6 pieces  ~10x4mm.  Urothelium variably sloughed, better preserved than usual, less than 1% of thickness
#&gt; 2                                                         8 pieces, up to 10x3 mm; preserved mucosa on all
#&gt; 3                          7 pieces,  ~7x3.5mm.  Urothelium remarkably well-preserved,  ~2-3% of thickness
#&gt; 4                   6 pieces up to 8x4 mm; mucosa is variably preserved: some areas well and some sloughed
#&gt; 5                            6 pieces  ~7x5mm.  Urothelium partly sloughed but better preserved than usual
#&gt; 6                                                         6 pieces up to 8x5 mm; only one piece has mucosa
#&gt;   smrin    smts   smtsd smubrid smtsisch smtspax smnabtch
#&gt; 1   7.5 Bladder Bladder    1255      230     747 BP-25560
#&gt; 2   7.8 Bladder Bladder    1255      191    1225 BP-25601
#&gt; 3   7.7 Bladder Bladder    1255      161     412 BP-23051
#&gt; 4   8.6 Bladder Bladder    1255      230    1045 BP-23794
#&gt; 5   8.0 Bladder Bladder    1255      439    1181 BP-25190
#&gt; 6   7.7 Bladder Bladder    1255      185     972 BP-23389
#&gt;                            smnabtcht  smnabtchd   smgebtch  smgebtchd smgebtcht
#&gt; 1 RNA isolation_PAXgene Tissue miRNA 05/22/2012 LCSET-3206 06/04/2013 TruSeq.v1
#&gt; 2 RNA isolation_PAXgene Tissue miRNA 05/23/2012 LCSET-3206 06/04/2013 TruSeq.v1
#&gt; 3 RNA isolation_PAXgene Tissue miRNA 03/13/2012 LCSET-3098 05/14/2013 TruSeq.v1
#&gt; 4 RNA isolation_PAXgene Tissue miRNA 04/04/2012 LCSET-3205 05/31/2013 TruSeq.v1
#&gt; 5 RNA isolation_PAXgene Tissue miRNA 05/09/2012 LCSET-3206 06/04/2013 TruSeq.v1
#&gt; 6 RNA isolation_PAXgene Tissue miRNA 03/26/2012 LCSET-3152 05/27/2013 TruSeq.v1
#&gt;   smafrze smgtc sme2mprt smchmprs smntrart smnumgps  smmaprt smexncrt sm550nrm
#&gt; 1  RNASEQ    NA 0.988668   339167 0.975561       NA 0.991069 0.843516       NA
#&gt; 2  RNASEQ    NA 0.988598   536713 0.973821       NA 0.991280 0.821863       NA
#&gt; 3  RNASEQ    NA 0.990838   290714 0.973011       NA 0.993009 0.801211       NA
#&gt; 4  RNASEQ    NA 0.988295   293534 0.967582       NA 0.990607 0.763716       NA
#&gt; 5  RNASEQ    NA 0.988508   369265 0.975620       NA 0.991073 0.823269       NA
#&gt; 6  RNASEQ    NA 0.991046   394834 0.976320       NA 0.993359 0.848322       NA
#&gt;   smgnsdtc smunmprt sm350nrm smrdlgth smmncpb   sme1mmrt smsflgth smestlbs
#&gt; 1    21262        1       NA       76      NA 0.00209364      139        0
#&gt; 2    22592        1       NA       76      NA 0.00224570      157        0
#&gt; 3    23205        1       NA       76      NA 0.00202882      131        0
#&gt; 4    23092        1       NA       76      NA 0.00208335      110        0
#&gt; 5    22184        1       NA       76      NA 0.00222215      144        0
#&gt; 6    21412        1       NA       76      NA 0.00221069      158        0
#&gt;     smmppd  smnterrt smrrnanm  smrdttl smvqcfl smmncv smtrscpt smmppdpr
#&gt; 1 63926700 0.0241488   738605 64502700 5839470     NA    21277 31792900
#&gt; 2 73718500 0.0258600   374650 74367000 6567550     NA    22622 36646200
#&gt; 3 61337000 0.0266825   212377 61768800 3683040     NA    23225 30529500
#&gt; 4 51773500 0.0321327   235558 52264400 3637220     NA    23118 25760400
#&gt; 5 68070100 0.0240411   283345 68683200 6200030     NA    22201 33853700
#&gt; 6 72052900 0.0234005   491168 72534500 7105490     NA    21427 35856700
#&gt;   smcglgth smgappct smunpdrd smntrnrt smmpunrt smexpeff smmppdun   sme2mmrt
#&gt; 1       NA       NA        0 0.132045 0.991069 0.835983 63926700 0.00297382
#&gt; 2       NA       NA        0 0.151958 0.991280 0.814696 73718500 0.00313601
#&gt; 3       NA       NA        0 0.171800 0.993009 0.795610 61337000 0.00277657
#&gt; 4       NA       NA        0 0.203866 0.990607 0.756543 51773500 0.00345301
#&gt; 5       NA       NA        0 0.152351 0.991073 0.815920 68070100 0.00305874
#&gt; 6       NA       NA        0 0.127998 0.993359 0.842688 72052900 0.00291208
#&gt;   sme2anti smaltalg sme2snse smmflgth sme1anti smspltrd   smbsmmrt sme1snse
#&gt; 1 13807000  5680860 13836800      159 13810400 11069400 0.00253266 13979700
#&gt; 2 15965200  6506070 15978200      169 16007900 13524600 0.00268965 16124700
#&gt; 3 13458000  5397620 13489700      141 13510700 10989200 0.00240188 13565700
#&gt; 4 11429000  6115360 11465000      112 11507200  9054400 0.00276658 11502900
#&gt; 5 14772000  6248800 14774700      154 14798900 12928200 0.00263936 14915200
#&gt; 6 15389200  5930350 15519600      172 15501800 13665500 0.00256057 15558500
#&gt;   sme1pcts   smrrnart sme1mprt smnum5cd smdpmprt sme2pcts junction_count
#&gt; 1  50.3047 0.01145080 0.993470       NA        0  50.0538         247600
#&gt; 2  50.1818 0.00503785 0.993962       NA        0  50.0203         188783
#&gt; 3  50.1015 0.00343826 0.995181       NA        0  50.0589         302563
#&gt; 4  49.9907 0.00450704 0.992919       NA        0  50.0785         253719
#&gt; 5  50.1956 0.00412539 0.993638       NA        0  50.0046         227259
#&gt; 6  50.0913 0.00677150 0.995672       NA        0  50.2108         458369
#&gt;   junction_coverage junction_avg_coverage
#&gt; 1           7282179              29.41106
#&gt; 2          15845256              83.93370
#&gt; 3          16243715              53.68705
#&gt; 4          13421701              52.89987
#&gt; 5          16123862              70.94928
#&gt; 6          10194323              22.24043</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.9000.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


