<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Read a metadata file — read_metadata • recount3</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Read a metadata file — read_metadata" />
<meta property="og:description" content="This function reads in a recount3 metadata file into R. You can first
locate the file using file_locate_url() then download it to your computer
using file_retrieve()." />
<meta property="og:image" content="/logo.png" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">recount3</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/recount3-quickstart.html">recount3 quick start guide</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/LieberInstitute/recount3/">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Read a metadata file</h1>
    <small class="dont-index">Source: <a href='https://github.com/LieberInstitute/recount3/blob/master/R/read_metadata.R'><code>R/read_metadata.R</code></a></small>
    <div class="hidden name"><code>read_metadata.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This function reads in a <code>recount3</code> metadata file into R. You can first
locate the file using <code><a href='file_locate_url.html'>file_locate_url()</a></code> then download it to your computer
using <code><a href='file_retrieve.html'>file_retrieve()</a></code>.</p>
    </div>

    <pre class="usage"><span class='fu'>read_metadata</span>(<span class='no'>metadata_file</span>)</pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>metadata_file</th>
      <td><p>A <code><a href='https://rdrr.io/r/base/character.html'>character(1)</a></code> with the local path to a <code>recount3</code>
metadata file.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A <code><a href='https://rdrr.io/r/base/data.frame.html'>data.frame()</a></code> with all lower case column names for the sample
metadata.</p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>
<span class='co'>## Download the metadata file for project ERP001942</span>
<span class='no'>url_ERP001942_meta</span> <span class='kw'>&lt;-</span> <span class='fu'><a href='file_locate_url.html'>file_locate_url</a></span>(<span class='st'>"ERP001942"</span>, <span class='st'>"data_sources/sra"</span>)
<span class='no'>local_ERP001942_meta</span> <span class='kw'>&lt;-</span> <span class='fu'><a href='file_retrieve.html'>file_retrieve</a></span>(<span class='kw'>url</span> <span class='kw'>=</span> <span class='no'>url_ERP001942_meta</span>)

<span class='co'>## Read the metadata</span>
<span class='no'>ERP001942_meta</span> <span class='kw'>&lt;-</span> <span class='fu'>read_metadata</span>(<span class='no'>local_ERP001942_meta</span>)
<span class='fu'><a href='https://rdrr.io/r/utils/head.html'>head</a></span>(<span class='no'>ERP001942_meta</span>)</div><div class='output co'>#&gt;   rail_id   run_acc study_acc sample_acc experiment_acc submission_acc
#&gt; 1   98371 ERR188021 ERP001942  ERS185362      ERX163159      ERA169774
#&gt; 2   98373 ERR188022 ERP001942  ERS185103      ERX162912      ERA169774
#&gt; 3   98375 ERR188023 ERP001942  ERS185236      ERX162777      ERA169774
#&gt; 4   98377 ERR188024 ERP001942  ERS185432      ERX162876      ERA169774
#&gt; 5   98379 ERR188025 ERP001942  ERS185451      ERX163121      ERA169774
#&gt; 6   98381 ERR188026 ERP001942  ERS185219      ERX162823      ERA169774
#&gt;   submission_center submission_lab
#&gt; 1          Geuvadis   ArrayExpress
#&gt; 2          Geuvadis   ArrayExpress
#&gt; 3          Geuvadis   ArrayExpress
#&gt; 4          Geuvadis   ArrayExpress
#&gt; 5          Geuvadis   ArrayExpress
#&gt; 6          Geuvadis   ArrayExpress
#&gt;                                                             study_title
#&gt; 1 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 2 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 3 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 4 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 5 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 6 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt;                                                                                                                                                                                                                                                                                         study_abstract
#&gt; 1 The data is under embargo until the first publication by the investigators in early 2013. This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium.
#&gt; 2 The data is under embargo until the first publication by the investigators in early 2013. This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium.
#&gt; 3 The data is under embargo until the first publication by the investigators in early 2013. This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium.
#&gt; 4 The data is under embargo until the first publication by the investigators in early 2013. This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium.
#&gt; 5 The data is under embargo until the first publication by the investigators in early 2013. This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium.
#&gt; 6 The data is under embargo until the first publication by the investigators in early 2013. This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium.
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 study_description
#&gt; 1 This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium (www.geuvadis.org, http://www.geuvadis.org/web/geuvadis/our-rnaseq-project). The data is under embargo until the first publication by the investigators in early 2013. This accession contains mRNA sequencing data, and small RNA data from the same samples are available under accession E-GEUV-2.
#&gt; 2 This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium (www.geuvadis.org, http://www.geuvadis.org/web/geuvadis/our-rnaseq-project). The data is under embargo until the first publication by the investigators in early 2013. This accession contains mRNA sequencing data, and small RNA data from the same samples are available under accession E-GEUV-2.
#&gt; 3 This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium (www.geuvadis.org, http://www.geuvadis.org/web/geuvadis/our-rnaseq-project). The data is under embargo until the first publication by the investigators in early 2013. This accession contains mRNA sequencing data, and small RNA data from the same samples are available under accession E-GEUV-2.
#&gt; 4 This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium (www.geuvadis.org, http://www.geuvadis.org/web/geuvadis/our-rnaseq-project). The data is under embargo until the first publication by the investigators in early 2013. This accession contains mRNA sequencing data, and small RNA data from the same samples are available under accession E-GEUV-2.
#&gt; 5 This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium (www.geuvadis.org, http://www.geuvadis.org/web/geuvadis/our-rnaseq-project). The data is under embargo until the first publication by the investigators in early 2013. This accession contains mRNA sequencing data, and small RNA data from the same samples are available under accession E-GEUV-2.
#&gt; 6 This RNA sequencing data set of 465 human lymphoblastoid cell line samples from the CEU, FIN, GBR, TSI and YRI populations from the 1000 Genomes sample collection was created by the Geuvadis consortium (www.geuvadis.org, http://www.geuvadis.org/web/geuvadis/our-rnaseq-project). The data is under embargo until the first publication by the investigators in early 2013. This accession contains mRNA sequencing data, and small RNA data from the same samples are available under accession E-GEUV-2.
#&gt;                                                        experiment_title
#&gt; 1 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 2 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 3 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 4 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 5 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt; 6 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000 Genomes
#&gt;                                                                      design_description
#&gt; 1 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000                 Genomes
#&gt; 2 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000                 Genomes
#&gt; 3 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000                 Genomes
#&gt; 4 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000                 Genomes
#&gt; 5 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000                 Genomes
#&gt; 6 RNA-sequencing of 465 lymphoblastoid cell lines from the 1000                 Genomes
#&gt;   sample_description                 library_name library_strategy
#&gt; 1                 NA NA20508.1.M_111124_2 extract          RNA-Seq
#&gt; 2                 NA NA12812.1.M_111124_4 extract          RNA-Seq
#&gt; 3                 NA HG00315.2.M_111215_5 extract          RNA-Seq
#&gt; 4                 NA NA12749.1.M_111124_2 extract          RNA-Seq
#&gt; 5                 NA HG00377.2.M_120131_2 extract          RNA-Seq
#&gt; 6                 NA NA20510.3.M_120202_3 extract          RNA-Seq
#&gt;   library_source library_selection library_layout paired_nominal_length
#&gt; 1 TRANSCRIPTOMIC              cDNA         paired                   280
#&gt; 2 TRANSCRIPTOMIC              cDNA         paired                   280
#&gt; 3 TRANSCRIPTOMIC              cDNA         paired                   280
#&gt; 4 TRANSCRIPTOMIC              cDNA         paired                   280
#&gt; 5 TRANSCRIPTOMIC              cDNA         paired                   280
#&gt; 6 TRANSCRIPTOMIC              cDNA         paired                   280
#&gt;   paired_nominal_stdev
#&gt; 1                    0
#&gt; 2                    0
#&gt; 3                    0
#&gt; 4                    0
#&gt; 5                    0
#&gt; 6                    0
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   library_construction_protocol
#&gt; 1 Total RNA was extracted from cell pellets using the                     TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any                     additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol                     was added in each sample, and the samples were transferred to eppendorf tubes.                     The rest of the protocol followed the manufacturer's guidelines. No DNase                     treatment was done to the RNA samples. Sample processing for sequencing was                     done in a random order in each of the seven laboratories. Library preparation                     was done with TruSeq RNA Sample Prep Kit v2 using the high-throughput                     protocol.
#&gt; 2 Total RNA was extracted from cell pellets using the                     TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any                     additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol                     was added in each sample, and the samples were transferred to eppendorf tubes.                     The rest of the protocol followed the manufacturer's guidelines. No DNase                     treatment was done to the RNA samples. Sample processing for sequencing was                     done in a random order in each of the seven laboratories. Library preparation                     was done with TruSeq RNA Sample Prep Kit v2 using the high-throughput                     protocol.
#&gt; 3 Total RNA was extracted from cell pellets using the                     TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any                     additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol                     was added in each sample, and the samples were transferred to eppendorf tubes.                     The rest of the protocol followed the manufacturer's guidelines. No DNase                     treatment was done to the RNA samples. Sample processing for sequencing was                     done in a random order in each of the seven laboratories. Library preparation                     was done with TruSeq RNA Sample Prep Kit v2 using the high-throughput                     protocol.
#&gt; 4 Total RNA was extracted from cell pellets using the                     TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any                     additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol                     was added in each sample, and the samples were transferred to eppendorf tubes.                     The rest of the protocol followed the manufacturer's guidelines. No DNase                     treatment was done to the RNA samples. Sample processing for sequencing was                     done in a random order in each of the seven laboratories. Library preparation                     was done with TruSeq RNA Sample Prep Kit v2 using the high-throughput                     protocol.
#&gt; 5 Total RNA was extracted from cell pellets using the                     TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any                     additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol                     was added in each sample, and the samples were transferred to eppendorf tubes.                     The rest of the protocol followed the manufacturer's guidelines. No DNase                     treatment was done to the RNA samples. Sample processing for sequencing was                     done in a random order in each of the seven laboratories. Library preparation                     was done with TruSeq RNA Sample Prep Kit v2 using the high-throughput                     protocol.
#&gt; 6 Total RNA was extracted from cell pellets using the                     TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any                     additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol                     was added in each sample, and the samples were transferred to eppendorf tubes.                     The rest of the protocol followed the manufacturer's guidelines. No DNase                     treatment was done to the RNA samples. Sample processing for sequencing was                     done in a random order in each of the seven laboratories. Library preparation                     was done with TruSeq RNA Sample Prep Kit v2 using the high-throughput                     protocol.
#&gt;        platform_model
#&gt; 1 Illumina HiSeq 2000
#&gt; 2 Illumina HiSeq 2000
#&gt; 3 Illumina HiSeq 2000
#&gt; 4 Illumina HiSeq 2000
#&gt; 5 Illumina HiSeq 2000
#&gt; 6 Illumina HiSeq 2000
#&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             sample_attributes
#&gt; 1 Alias;;GEUV:NA20508|Broker name;;ArrayExpress|Description;;Protocols: For more information about the populations, ELSI, etc., see http://www.1000genomes.org/about#ProjectSamples and http://ccr.coriell.org/Default.aspx. EVB transformed lymphoblastoid cell lines (LCLs) from Coriell Cell Repositories (GBR, FIN, TSI) and University of Geneva (CEU, YRI) were shipped to ECACC (European COllection of Cell Cultures) as live cultures (06/2011 - 09/2011), in batches of ~ 30 samples from Coriell (GBR/FIN/TSI more or less randomized) and 2 x ~90 samples (by population) from Geneva. In ECACC, these cell lines were cultures to approximately 1.2 x 10e8 cells (06/2011-10/2011). These cultures were split to produce 8 x cell banks of the samples, and a snap frozen pellet of 2 x 10e7 cells from a proliferating culture. The cell pellets were shipped from ECACC to University of Geneva in three batches, the first batch consisting of CEU/GBR/FIN/TSI samples, and the second and third batch with YRI and some CEU samples. Total RNA was extracted from cell pellets using the TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol was added in each sample, and the samples were transferred to eppendorf tubes. The rest of the protocol followed the manufacturer's guidelines. No DNAse treatment was done to the RNA samples. The RNA quality was assessed by Agilent Bioanalyzer RNA 6000 Nano Kit according to the manufacturer's instructions. The sample information includes a visual inspection of RNA quality (degradation), in the scale 0, 0.5, 1. RNA quantity was measured by Qubit 2.0 (Invitrogen) using the RNA Broad range kit according to the manufacturer's instructions.|INSDC center alias;;Geuvadis|INSDC center name;;Geuvadis|INSDC first public;;2012-11-06T17:02:05Z|INSDC last update;;2018-03-08T16:06:35Z|INSDC status;;public|population;;TSI|Sample Name;;ERS185362|SRA accession;;ERS185362|Strain;;lymphoblastoid cell line|Title;;NA20508
#&gt; 2 Alias;;GEUV:NA12812|Broker name;;ArrayExpress|Description;;Protocols: For more information about the populations, ELSI, etc., see http://www.1000genomes.org/about#ProjectSamples and http://ccr.coriell.org/Default.aspx. EVB transformed lymphoblastoid cell lines (LCLs) from Coriell Cell Repositories (GBR, FIN, TSI) and University of Geneva (CEU, YRI) were shipped to ECACC (European COllection of Cell Cultures) as live cultures (06/2011 - 09/2011), in batches of ~ 30 samples from Coriell (GBR/FIN/TSI more or less randomized) and 2 x ~90 samples (by population) from Geneva. In ECACC, these cell lines were cultures to approximately 1.2 x 10e8 cells (06/2011-10/2011). These cultures were split to produce 8 x cell banks of the samples, and a snap frozen pellet of 2 x 10e7 cells from a proliferating culture. The cell pellets were shipped from ECACC to University of Geneva in three batches, the first batch consisting of CEU/GBR/FIN/TSI samples, and the second and third batch with YRI and some CEU samples. Total RNA was extracted from cell pellets using the TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol was added in each sample, and the samples were transferred to eppendorf tubes. The rest of the protocol followed the manufacturer's guidelines. No DNAse treatment was done to the RNA samples. The RNA quality was assessed by Agilent Bioanalyzer RNA 6000 Nano Kit according to the manufacturer's instructions. The sample information includes a visual inspection of RNA quality (degradation), in the scale 0, 0.5, 1. RNA quantity was measured by Qubit 2.0 (Invitrogen) using the RNA Broad range kit according to the manufacturer's instructions.|INSDC center alias;;Geuvadis|INSDC center name;;Geuvadis|INSDC first public;;2012-11-06T17:02:05Z|INSDC last update;;2018-03-08T16:06:49Z|INSDC status;;public|population;;CEU|Sample Name;;ERS185103|SRA accession;;ERS185103|Strain;;lymphoblastoid cell line|Title;;NA12812
#&gt; 3 Alias;;GEUV:HG00315|Broker name;;ArrayExpress|Description;;Protocols: For more information about the populations, ELSI, etc., see http://www.1000genomes.org/about#ProjectSamples and http://ccr.coriell.org/Default.aspx. EVB transformed lymphoblastoid cell lines (LCLs) from Coriell Cell Repositories (GBR, FIN, TSI) and University of Geneva (CEU, YRI) were shipped to ECACC (European COllection of Cell Cultures) as live cultures (06/2011 - 09/2011), in batches of ~ 30 samples from Coriell (GBR/FIN/TSI more or less randomized) and 2 x ~90 samples (by population) from Geneva. In ECACC, these cell lines were cultures to approximately 1.2 x 10e8 cells (06/2011-10/2011). These cultures were split to produce 8 x cell banks of the samples, and a snap frozen pellet of 2 x 10e7 cells from a proliferating culture. The cell pellets were shipped from ECACC to University of Geneva in three batches, the first batch consisting of CEU/GBR/FIN/TSI samples, and the second and third batch with YRI and some CEU samples. Total RNA was extracted from cell pellets using the TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol was added in each sample, and the samples were transferred to eppendorf tubes. The rest of the protocol followed the manufacturer's guidelines. No DNAse treatment was done to the RNA samples. The RNA quality was assessed by Agilent Bioanalyzer RNA 6000 Nano Kit according to the manufacturer's instructions. The sample information includes a visual inspection of RNA quality (degradation), in the scale 0, 0.5, 1. RNA quantity was measured by Qubit 2.0 (Invitrogen) using the RNA Broad range kit according to the manufacturer's instructions.|INSDC center alias;;Geuvadis|INSDC center name;;Geuvadis|INSDC first public;;2012-11-06T17:02:05Z|INSDC last update;;2018-03-08T16:07:06Z|INSDC status;;public|population;;FIN|Sample Name;;ERS185236|SRA accession;;ERS185236|Strain;;lymphoblastoid cell line|Title;;HG00315
#&gt; 4 Alias;;GEUV:NA12749|Broker name;;ArrayExpress|Description;;Protocols: For more information about the populations, ELSI, etc., see http://www.1000genomes.org/about#ProjectSamples and http://ccr.coriell.org/Default.aspx. EVB transformed lymphoblastoid cell lines (LCLs) from Coriell Cell Repositories (GBR, FIN, TSI) and University of Geneva (CEU, YRI) were shipped to ECACC (European COllection of Cell Cultures) as live cultures (06/2011 - 09/2011), in batches of ~ 30 samples from Coriell (GBR/FIN/TSI more or less randomized) and 2 x ~90 samples (by population) from Geneva. In ECACC, these cell lines were cultures to approximately 1.2 x 10e8 cells (06/2011-10/2011). These cultures were split to produce 8 x cell banks of the samples, and a snap frozen pellet of 2 x 10e7 cells from a proliferating culture. The cell pellets were shipped from ECACC to University of Geneva in three batches, the first batch consisting of CEU/GBR/FIN/TSI samples, and the second and third batch with YRI and some CEU samples. Total RNA was extracted from cell pellets using the TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol was added in each sample, and the samples were transferred to eppendorf tubes. The rest of the protocol followed the manufacturer's guidelines. No DNAse treatment was done to the RNA samples. The RNA quality was assessed by Agilent Bioanalyzer RNA 6000 Nano Kit according to the manufacturer's instructions. The sample information includes a visual inspection of RNA quality (degradation), in the scale 0, 0.5, 1. RNA quantity was measured by Qubit 2.0 (Invitrogen) using the RNA Broad range kit according to the manufacturer's instructions.|INSDC center alias;;Geuvadis|INSDC center name;;Geuvadis|INSDC first public;;2012-11-06T17:02:05Z|INSDC last update;;2018-03-08T16:07:11Z|INSDC status;;public|population;;CEU|Sample Name;;ERS185432|SRA accession;;ERS185432|Strain;;lymphoblastoid cell line|Title;;NA12749
#&gt; 5 Alias;;GEUV:HG00377|Broker name;;ArrayExpress|Description;;Protocols: For more information about the populations, ELSI, etc., see http://www.1000genomes.org/about#ProjectSamples and http://ccr.coriell.org/Default.aspx. EVB transformed lymphoblastoid cell lines (LCLs) from Coriell Cell Repositories (GBR, FIN, TSI) and University of Geneva (CEU, YRI) were shipped to ECACC (European COllection of Cell Cultures) as live cultures (06/2011 - 09/2011), in batches of ~ 30 samples from Coriell (GBR/FIN/TSI more or less randomized) and 2 x ~90 samples (by population) from Geneva. In ECACC, these cell lines were cultures to approximately 1.2 x 10e8 cells (06/2011-10/2011). These cultures were split to produce 8 x cell banks of the samples, and a snap frozen pellet of 2 x 10e7 cells from a proliferating culture. The cell pellets were shipped from ECACC to University of Geneva in three batches, the first batch consisting of CEU/GBR/FIN/TSI samples, and the second and third batch with YRI and some CEU samples. Total RNA was extracted from cell pellets using the TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol was added in each sample, and the samples were transferred to eppendorf tubes. The rest of the protocol followed the manufacturer's guidelines. No DNAse treatment was done to the RNA samples. The RNA quality was assessed by Agilent Bioanalyzer RNA 6000 Nano Kit according to the manufacturer's instructions. The sample information includes a visual inspection of RNA quality (degradation), in the scale 0, 0.5, 1. RNA quantity was measured by Qubit 2.0 (Invitrogen) using the RNA Broad range kit according to the manufacturer's instructions.|INSDC center alias;;Geuvadis|INSDC center name;;Geuvadis|INSDC first public;;2012-11-06T17:02:05Z|INSDC last update;;2018-03-08T16:06:54Z|INSDC status;;public|population;;FIN|Sample Name;;ERS185451|SRA accession;;ERS185451|Strain;;lymphoblastoid cell line|Title;;HG00377
#&gt; 6 Alias;;GEUV:NA20510|Broker name;;ArrayExpress|Description;;Protocols: For more information about the populations, ELSI, etc., see http://www.1000genomes.org/about#ProjectSamples and http://ccr.coriell.org/Default.aspx. EVB transformed lymphoblastoid cell lines (LCLs) from Coriell Cell Repositories (GBR, FIN, TSI) and University of Geneva (CEU, YRI) were shipped to ECACC (European COllection of Cell Cultures) as live cultures (06/2011 - 09/2011), in batches of ~ 30 samples from Coriell (GBR/FIN/TSI more or less randomized) and 2 x ~90 samples (by population) from Geneva. In ECACC, these cell lines were cultures to approximately 1.2 x 10e8 cells (06/2011-10/2011). These cultures were split to produce 8 x cell banks of the samples, and a snap frozen pellet of 2 x 10e7 cells from a proliferating culture. The cell pellets were shipped from ECACC to University of Geneva in three batches, the first batch consisting of CEU/GBR/FIN/TSI samples, and the second and third batch with YRI and some CEU samples. Total RNA was extracted from cell pellets using the TRIzol Reagent (Ambion). The pellets had been frozen at ECACC without any additives like RNAlater or Trizol. In Geneva they were thawed, 1mL of trizol was added in each sample, and the samples were transferred to eppendorf tubes. The rest of the protocol followed the manufacturer's guidelines. No DNAse treatment was done to the RNA samples. The RNA quality was assessed by Agilent Bioanalyzer RNA 6000 Nano Kit according to the manufacturer's instructions. The sample information includes a visual inspection of RNA quality (degradation), in the scale 0, 0.5, 1. RNA quantity was measured by Qubit 2.0 (Invitrogen) using the RNA Broad range kit according to the manufacturer's instructions.|INSDC center alias;;Geuvadis|INSDC center name;;Geuvadis|INSDC first public;;2012-11-06T17:02:05Z|INSDC last update;;2018-03-08T16:06:54Z|INSDC status;;public|population;;TSI|Sample Name;;ERS185219|SRA accession;;ERS185219|Strain;;lymphoblastoid cell line|Title;;NA20510
#&gt;                                                     experiment_attributes
#&gt; 1 Experimental Factor: laboratory;;1|Experimental Factor: population;;TSI
#&gt; 2 Experimental Factor: laboratory;;1|Experimental Factor: population;;CEU
#&gt; 3 Experimental Factor: laboratory;;2|Experimental Factor: population;;FIN
#&gt; 4 Experimental Factor: laboratory;;1|Experimental Factor: population;;CEU
#&gt; 5 Experimental Factor: laboratory;;2|Experimental Factor: population;;FIN
#&gt; 6 Experimental Factor: laboratory;;3|Experimental Factor: population;;TSI
#&gt;   spot_length  sample_name sample_title sample_bases sample_spots
#&gt; 1          75 SAMEA1573446 GEUV:NA20508   4876174200     32507828
#&gt; 2          75 SAMEA1573207 GEUV:NA12812   3348579900     22323866
#&gt; 3          75 SAMEA1573500 GEUV:HG00315   5548298680     36501965
#&gt; 4          75 SAMEA1573203 GEUV:NA12749   3572181450     23814543
#&gt; 5          75 SAMEA1573346 GEUV:HG00377   5348693800     35188775
#&gt; 6          75 SAMEA1573212 GEUV:NA20510   2370019050     15800127
#&gt;         run_published       size run_total_bases run_total_spots num_reads
#&gt; 1 2012-11-07 04:16:08 3623033347      4876174200        32507828         2
#&gt; 2 2012-11-07 04:14:09 2432987749      3348579900        22323866         2
#&gt; 3 2012-11-07 04:18:08 3993335999      5548298680        36501965         2
#&gt; 4 2012-11-07 04:17:08 2543501984      3572181450        23814543         2
#&gt; 5 2012-11-07 04:19:11 3678693525      5348693800        35188775         2
#&gt; 6 2012-11-07 04:15:12 1634077963      2370019050        15800127         2
#&gt;   num_spots
#&gt; 1  32507828
#&gt; 2  22323866
#&gt; 3  36501965
#&gt; 4  23814543
#&gt; 5  35188775
#&gt; 6  15800127
#&gt;                                                                             read_info
#&gt; 1 index:0,count:32507828,average:75,stdev:0|index:1,count:32507828,average:75,stdev:0
#&gt; 2 index:0,count:22323866,average:75,stdev:0|index:1,count:22323866,average:75,stdev:0
#&gt; 3 index:0,count:36501965,average:76,stdev:0|index:1,count:36501965,average:76,stdev:0
#&gt; 4 index:0,count:23814543,average:75,stdev:0|index:1,count:23814543,average:75,stdev:0
#&gt; 5 index:0,count:35188775,average:76,stdev:0|index:1,count:35188775,average:76,stdev:0
#&gt; 6 index:0,count:15800127,average:75,stdev:0|index:1,count:15800127,average:75,stdev:0
#&gt;                       run_alias run_center_name run_broker_name run_center
#&gt; 1 E-GEUV-1:NA20508.1.M_111124_2        Geuvadis    ArrayExpress      UNIGE
#&gt; 2 E-GEUV-1:NA12812.1.M_111124_4        Geuvadis    ArrayExpress      UNIGE
#&gt; 3 E-GEUV-1:HG00315.2.M_111215_5        Geuvadis    ArrayExpress        CRG
#&gt; 4 E-GEUV-1:NA12749.1.M_111124_2        Geuvadis    ArrayExpress      UNIGE
#&gt; 5 E-GEUV-1:HG00377.2.M_120131_2        Geuvadis    ArrayExpress        CRG
#&gt; 6 E-GEUV-1:NA20510.3.M_120202_3        Geuvadis    ArrayExpress      MPIMG
#&gt;   aligned_reads%.chrm aligned_reads%.chrx aligned_reads%.chry
#&gt; 1                3.84                2.45                0.01
#&gt; 2                4.71                2.38                0.08
#&gt; 3                5.42                3.26                0.01
#&gt; 4                2.96                2.59                0.01
#&gt; 5                5.50                3.02                0.01
#&gt; 6                5.32                3.21                0.08
#&gt;   bc_auc.all_reads_all_bases bc_auc.all_reads_annotated_bases
#&gt; 1                 4487006736                       5937023036
#&gt; 2                 3160774094                       4254962867
#&gt; 3                 4986944225                       6978656596
#&gt; 4                 3391969427                       4449918757
#&gt; 5                 4790947777                       6808795523
#&gt; 6                 2146937605                       3001028496
#&gt;   bc_auc.unique_reads_all_bases bc_auc.unique_reads_annotated_bases
#&gt; 1                    4031104159                          5553402006
#&gt; 2                    2877609107                          3997629999
#&gt; 3                    4651727114                          6551951006
#&gt; 4                    3046407903                          4163370586
#&gt; 5                    4448615007                          6371631019
#&gt; 6                    2005862336                          2826588229
#&gt;   bc_auc.all_% bc_auc.unique_% bc_frag.count bc_frag.kallisto_count
#&gt; 1       132.32          137.76      31597417               28687997
#&gt; 2       134.62          138.92      21812480               19707049
#&gt; 3       139.94          140.85      35631267               32627309
#&gt; 4       131.19          136.66      23281510               20759336
#&gt; 5       142.12          143.23      34203246               31389193
#&gt; 6       139.78          140.92      15532452               14286866
#&gt;   bc_frag.kallisto_mean_length bc_frag.mean_length bc_frag.mode_length
#&gt; 1                      195.032             923.168                 146
#&gt; 2                      211.248            1179.931                 157
#&gt; 3                      189.127            1343.918                 135
#&gt; 4                      217.225             984.437                 156
#&gt; 5                      187.503            2149.511                 135
#&gt; 6                      181.631            1337.894                 135
#&gt;   bc_frag.mode_length_count exon_fc.all_% exon_fc.unique_%
#&gt; 1                    339861         83.68            93.12
#&gt; 2                    195738         81.10            89.07
#&gt; 3                    379357         88.52            94.97
#&gt; 4                    207838         79.03            87.97
#&gt; 5                    379030         89.91            96.92
#&gt; 6                    176539         88.93            95.25
#&gt;   exon_fc_count_all.total exon_fc_count_all.assigned exon_fc_count_unique.total
#&gt; 1                44086639                   26441235                   44086639
#&gt; 2                28613656                   17689077                   28613656
#&gt; 3                39383364                   31539949                   39383364
#&gt; 4                32328629                   18398884                   32328629
#&gt; 5                38269668                   30753643                   38269668
#&gt; 6                17366656                   13813455                   17366656
#&gt;   exon_fc_count_unique.assigned gene_fc.all_% gene_fc.unique_%
#&gt; 1                      26441235         84.22            88.09
#&gt; 2                      17689077         81.48            84.20
#&gt; 3                      31539949         88.87            89.32
#&gt; 4                      18398884         79.88            83.35
#&gt; 5                      30753643         90.05            90.75
#&gt; 6                      13813455         89.04            89.83
#&gt;   gene_fc_count_all.total gene_fc_count_all.assigned gene_fc_count_unique.total
#&gt; 1                44086639                   26612458                   44086639
#&gt; 2                28613656                   17773582                   28613656
#&gt; 3                39383364                   31666596                   39383364
#&gt; 4                32328629                   18597797                   32328629
#&gt; 5                38269668                   30800684                   38269668
#&gt; 6                17366656                   13830598                   17366656
#&gt;   gene_fc_count_unique.assigned intron_sum intron_sum_%
#&gt; 1                      25012479  114005967         2.54
#&gt; 2                      16721445  132732157         4.20
#&gt; 3                      29663835   67507957         1.35
#&gt; 4                      17433271  152366351         4.49
#&gt; 5                      28797127   39915997         0.83
#&gt; 6                      13027645   29062312         1.35
#&gt;   star.%_of_chimeric_reads star.%_of_chimeric_reads2
#&gt; 1                     0.94                         0
#&gt; 2                     0.89                         0
#&gt; 3                     1.16                         0
#&gt; 4                     0.74                         0
#&gt; 5                     1.12                         0
#&gt; 6                     0.59                         0
#&gt;   star.%_of_reads_mapped_to_multiple_loci
#&gt; 1                                    9.86
#&gt; 2                                    8.75
#&gt; 3                                    6.63
#&gt; 4                                    9.93
#&gt; 5                                    7.02
#&gt; 6                                    6.52
#&gt;   star.%_of_reads_mapped_to_multiple_loci2
#&gt; 1                                        0
#&gt; 2                                        0
#&gt; 3                                        0
#&gt; 4                                        0
#&gt; 5                                        0
#&gt; 6                                        0
#&gt;   star.%_of_reads_mapped_to_too_many_loci
#&gt; 1                                    0.06
#&gt; 2                                    0.06
#&gt; 3                                    0.03
#&gt; 4                                    0.07
#&gt; 5                                    0.03
#&gt; 6                                    0.03
#&gt;   star.%_of_reads_mapped_to_too_many_loci2 star.%_of_reads_unmapped:_other
#&gt; 1                                        0                            0.01
#&gt; 2                                        0                            0.02
#&gt; 3                                        0                            0.02
#&gt; 4                                        0                            0.02
#&gt; 5                                        0                            0.02
#&gt; 6                                        0                            0.02
#&gt;   star.%_of_reads_unmapped:_other2
#&gt; 1                                0
#&gt; 2                                0
#&gt; 3                                0
#&gt; 4                                0
#&gt; 5                                0
#&gt; 6                                0
#&gt;   star.%_of_reads_unmapped:_too_many_mismatches
#&gt; 1                                             0
#&gt; 2                                             0
#&gt; 3                                             0
#&gt; 4                                             0
#&gt; 5                                             0
#&gt; 6                                             0
#&gt;   star.%_of_reads_unmapped:_too_many_mismatches2
#&gt; 1                                              0
#&gt; 2                                              0
#&gt; 3                                              0
#&gt; 4                                              0
#&gt; 5                                              0
#&gt; 6                                              0
#&gt;   star.%_of_reads_unmapped:_too_short star.%_of_reads_unmapped:_too_short2
#&gt; 1                                2.72                                    0
#&gt; 2                                2.21                                    0
#&gt; 3                                2.33                                    0
#&gt; 4                                2.15                                    0
#&gt; 5                                2.75                                    0
#&gt; 6                                1.64                                    0
#&gt;   star.all_mapped_reads star.all_mapped_reads2 star.average_input_read_length
#&gt; 1              31597417                      0                            150
#&gt; 2              21812480                      0                            150
#&gt; 3              35631267                      0                            152
#&gt; 4              23281510                      0                            150
#&gt; 5              34203246                      0                            152
#&gt; 6              15532452                      0                            150
#&gt;   star.average_input_read_length2 star.average_mapped_length
#&gt; 1                               0                     148.21
#&gt; 2                               0                     148.43
#&gt; 3                               0                     150.14
#&gt; 4                               0                     148.40
#&gt; 5                               0                     150.03
#&gt; 6                               0                     148.17
#&gt;   star.average_mapped_length2 star.deletion_average_length
#&gt; 1                           0                         1.60
#&gt; 2                           0                         1.62
#&gt; 3                           0                         1.50
#&gt; 4                           0                         1.64
#&gt; 5                           0                         1.55
#&gt; 6                           0                         1.46
#&gt;   star.deletion_average_length2 star.deletion_rate_per_base
#&gt; 1                             0                        0.01
#&gt; 2                             0                        0.01
#&gt; 3                             0                        0.01
#&gt; 4                             0                        0.01
#&gt; 5                             0                        0.01
#&gt; 6                             0                        0.01
#&gt;   star.deletion_rate_per_base2 star.insertion_average_length
#&gt; 1                            0                          1.35
#&gt; 2                            0                          1.37
#&gt; 3                            0                          1.30
#&gt; 4                            0                          1.30
#&gt; 5                            0                          1.31
#&gt; 6                            0                          1.26
#&gt;   star.insertion_average_length2 star.insertion_rate_per_base
#&gt; 1                              0                         0.01
#&gt; 2                              0                         0.01
#&gt; 3                              0                         0.01
#&gt; 4                              0                         0.01
#&gt; 5                              0                         0.01
#&gt; 6                              0                         0.01
#&gt;   star.insertion_rate_per_base2 star.mapping_speed,_million_of_reads_per_hour
#&gt; 1                             0                                        281.32
#&gt; 2                             0                                        278.08
#&gt; 3                             0                                        353.24
#&gt; 4                             0                                        267.91
#&gt; 5                             0                                        329.04
#&gt; 6                             0                                        342.65
#&gt;   star.mapping_speed,_million_of_reads_per_hour2 star.mismatch_rate_per_base,_%
#&gt; 1                                              0                           0.58
#&gt; 2                                              0                           0.47
#&gt; 3                                              0                           0.43
#&gt; 4                                              0                           0.40
#&gt; 5                                              0                           0.45
#&gt; 6                                              0                           0.47
#&gt;   star.mismatch_rate_per_base,_%2 star.number_of_chimeric_reads
#&gt; 1                               0                        304080
#&gt; 2                               0                        198970
#&gt; 3                               0                        425006
#&gt; 4                               0                        177343
#&gt; 5                               0                        394268
#&gt; 6                               0                         93040
#&gt;   star.number_of_chimeric_reads2 star.number_of_input_reads
#&gt; 1                              0                   32507828
#&gt; 2                              0                   22323866
#&gt; 3                              0                   36501965
#&gt; 4                              0                   23814543
#&gt; 5                              0                   35188775
#&gt; 6                              0                   15800127
#&gt;   star.number_of_input_reads2 star.number_of_reads_mapped_to_multiple_loci
#&gt; 1                           0                                      3203701
#&gt; 2                           0                                      1953159
#&gt; 3                           0                                      2421483
#&gt; 4                           0                                      2365933
#&gt; 5                           0                                      2471803
#&gt; 6                           0                                      1029787
#&gt;   star.number_of_reads_mapped_to_multiple_loci2
#&gt; 1                                             0
#&gt; 2                                             0
#&gt; 3                                             0
#&gt; 4                                             0
#&gt; 5                                             0
#&gt; 6                                             0
#&gt;   star.number_of_reads_mapped_to_too_many_loci
#&gt; 1                                        20436
#&gt; 2                                        14297
#&gt; 3                                        11077
#&gt; 4                                        17372
#&gt; 5                                        11281
#&gt; 6                                         5478
#&gt;   star.number_of_reads_mapped_to_too_many_loci2
#&gt; 1                                             0
#&gt; 2                                             0
#&gt; 3                                             0
#&gt; 4                                             0
#&gt; 5                                             0
#&gt; 6                                             0
#&gt;   star.number_of_reads_unmapped:_other star.number_of_reads_unmapped:_other2
#&gt; 1                                 4330                                     0
#&gt; 2                                 3797                                     0
#&gt; 3                                 8350                                     0
#&gt; 4                                 4292                                     0
#&gt; 5                                 6600                                     0
#&gt; 6                                 2606                                     0
#&gt;   star.number_of_reads_unmapped:_too_many_mismatches
#&gt; 1                                                  0
#&gt; 2                                                  0
#&gt; 3                                                  0
#&gt; 4                                                  0
#&gt; 5                                                  0
#&gt; 6                                                  0
#&gt;   star.number_of_reads_unmapped:_too_many_mismatches2
#&gt; 1                                                   0
#&gt; 2                                                   0
#&gt; 3                                                   0
#&gt; 4                                                   0
#&gt; 5                                                   0
#&gt; 6                                                   0
#&gt;   star.number_of_reads_unmapped:_too_short
#&gt; 1                                   885645
#&gt; 2                                   493292
#&gt; 3                                   851271
#&gt; 4                                   511369
#&gt; 5                                   967648
#&gt; 6                                   259591
#&gt;   star.number_of_reads_unmapped:_too_short2 star.number_of_splices:_at/ac
#&gt; 1                                         0                          5771
#&gt; 2                                         0                          3297
#&gt; 3                                         0                          6699
#&gt; 4                                         0                          3576
#&gt; 5                                         0                          6260
#&gt; 6                                         0                          2602
#&gt;   star.number_of_splices:_at/ac2 star.number_of_splices:_annotated_(sjdb)
#&gt; 1                              0                                        0
#&gt; 2                              0                                        0
#&gt; 3                              0                                        0
#&gt; 4                              0                                        0
#&gt; 5                              0                                        0
#&gt; 6                              0                                        0
#&gt;   star.number_of_splices:_annotated_(sjdb)2 star.number_of_splices:_gc/ag
#&gt; 1                                         0                         70736
#&gt; 2                                         0                         39374
#&gt; 3                                         0                         81695
#&gt; 4                                         0                         46323
#&gt; 5                                         0                         77631
#&gt; 6                                         0                         32725
#&gt;   star.number_of_splices:_gc/ag2 star.number_of_splices:_gt/ag
#&gt; 1                              0                      10360293
#&gt; 2                              0                       6042510
#&gt; 3                              0                      12198197
#&gt; 4                              0                       6632274
#&gt; 5                              0                      11914541
#&gt; 6                              0                       4817828
#&gt;   star.number_of_splices:_gt/ag2 star.number_of_splices:_non-canonical
#&gt; 1                              0                                 22871
#&gt; 2                              0                                 14958
#&gt; 3                              0                                 38809
#&gt; 4                              0                                 18031
#&gt; 5                              0                                 43545
#&gt; 6                              0                                 10566
#&gt;   star.number_of_splices:_non-canonical2 star.number_of_splices:_total
#&gt; 1                                      0                      10459671
#&gt; 2                                      0                       6100139
#&gt; 3                                      0                      12325400
#&gt; 4                                      0                       6700204
#&gt; 5                                      0                      12041977
#&gt; 6                                      0                       4863721
#&gt;   star.number_of_splices:_total2 star.uniquely_mapped_reads_%
#&gt; 1                              0                        87.34
#&gt; 2                              0                        88.96
#&gt; 3                              0                        90.98
#&gt; 4                              0                        87.83
#&gt; 5                              0                        90.17
#&gt; 6                              0                        91.79
#&gt;   star.uniquely_mapped_reads_%2 star.uniquely_mapped_reads_number
#&gt; 1                             0                          28393716
#&gt; 2                             0                          19859321
#&gt; 3                             0                          33209784
#&gt; 4                             0                          20915577
#&gt; 5                             0                          31731443
#&gt; 6                             0                          14502665
#&gt;   star.uniquely_mapped_reads_number2 junction_count junction_coverage
#&gt; 1                                  0         208563          10751073
#&gt; 2                                  0         179052           6310384
#&gt; 3                                  0         217431          12581781
#&gt; 4                                  0         192621           6940931
#&gt; 5                                  0         198828          12487692
#&gt; 6                                  0         164382           4986477
#&gt;   junction_avg_coverage
#&gt; 1              51.54832
#&gt; 2              35.24330
#&gt; 3              57.86563
#&gt; 4              36.03413
#&gt; 5              62.80651
#&gt; 6              30.33469</div><div class='input'>
<span class='co'>## Locate and read the GTEx bladder metadata</span>
<span class='no'>gtex_bladder_meta</span> <span class='kw'>&lt;-</span> <span class='fu'>read_metadata</span>(
    <span class='fu'><a href='file_retrieve.html'>file_retrieve</a></span>(
        <span class='fu'><a href='file_locate_url.html'>file_locate_url</a></span>(<span class='st'>"BLADDER"</span>, <span class='st'>"data_sources/gtex"</span>)
    )
)</div><div class='output co'>#&gt; <span class='message'>adding rname 'http://snaptron.cs.jhu.edu/data/temp/recount3/human/data_sources/gtex/metadata/ER/BLADDER/gtex.metadata.BLADDER.MD.gz'</span></div><div class='input'>
<span class='fu'><a href='https://rdrr.io/r/utils/head.html'>head</a></span>(<span class='no'>gtex_bladder_meta</span>)</div><div class='output co'>#&gt;   rail_id    run_acc              rail_barcode    subjid sex   age dthhrdy
#&gt; 1  153881 SRR1071717 GTEX-U3ZM-0826-SM-4DXU6.1 GTEX-U3ZM   1 40-49       0
#&gt; 2  309816 SRR1079830 GTEX-U4B1-1226-SM-4DXT7.1 GTEX-U4B1   1 40-49       0
#&gt; 3  270898 SRR1081765 GTEX-S4Q7-0926-SM-4AD5D.1 GTEX-S4Q7   1 20-29       0
#&gt; 4  615530 SRR1085402 GTEX-SNMC-0826-SM-4DM66.1 GTEX-SNMC   1 20-29       0
#&gt; 5  607337 SRR1086236 GTEX-TMMY-1526-SM-4DXST.1 GTEX-TMMY   2 40-49       0
#&gt; 6  277053 SRR1092208 GTEX-SE5C-1026-SM-4BRUG.1 GTEX-SE5C   2 40-49       0
#&gt;                    sampid smatsscr smcenter
#&gt; 1 GTEX-U3ZM-0826-SM-4DXU6        2       B1
#&gt; 2 GTEX-U4B1-1226-SM-4DXT7        1       B1
#&gt; 3 GTEX-S4Q7-0926-SM-4AD5D        1       B1
#&gt; 4 GTEX-SNMC-0826-SM-4DM66        2       B1
#&gt; 5 GTEX-TMMY-1526-SM-4DXST        2       C1
#&gt; 6 GTEX-SE5C-1026-SM-4BRUG        1       B1
#&gt;                                                                                                   smpthnts
#&gt; 1 6 pieces  ~10x4mm.  Urothelium variably sloughed, better preserved than usual, less than 1% of thickness
#&gt; 2                                                         8 pieces, up to 10x3 mm; preserved mucosa on all
#&gt; 3                          7 pieces,  ~7x3.5mm.  Urothelium remarkably well-preserved,  ~2-3% of thickness
#&gt; 4                   6 pieces up to 8x4 mm; mucosa is variably preserved: some areas well and some sloughed
#&gt; 5                            6 pieces  ~7x5mm.  Urothelium partly sloughed but better preserved than usual
#&gt; 6                                                         6 pieces up to 8x5 mm; only one piece has mucosa
#&gt;   smrin    smts   smtsd smubrid smtsisch smtspax smnabtch
#&gt; 1   7.5 Bladder Bladder    1255      230     747 BP-25560
#&gt; 2   7.8 Bladder Bladder    1255      191    1225 BP-25601
#&gt; 3   7.7 Bladder Bladder    1255      161     412 BP-23051
#&gt; 4   8.6 Bladder Bladder    1255      230    1045 BP-23794
#&gt; 5   8.0 Bladder Bladder    1255      439    1181 BP-25190
#&gt; 6   7.7 Bladder Bladder    1255      185     972 BP-23389
#&gt;                            smnabtcht  smnabtchd   smgebtch  smgebtchd smgebtcht
#&gt; 1 RNA isolation_PAXgene Tissue miRNA 05/22/2012 LCSET-3206 06/04/2013 TruSeq.v1
#&gt; 2 RNA isolation_PAXgene Tissue miRNA 05/23/2012 LCSET-3206 06/04/2013 TruSeq.v1
#&gt; 3 RNA isolation_PAXgene Tissue miRNA 03/13/2012 LCSET-3098 05/14/2013 TruSeq.v1
#&gt; 4 RNA isolation_PAXgene Tissue miRNA 04/04/2012 LCSET-3205 05/31/2013 TruSeq.v1
#&gt; 5 RNA isolation_PAXgene Tissue miRNA 05/09/2012 LCSET-3206 06/04/2013 TruSeq.v1
#&gt; 6 RNA isolation_PAXgene Tissue miRNA 03/26/2012 LCSET-3152 05/27/2013 TruSeq.v1
#&gt;   smafrze smgtc sme2mprt smchmprs smntrart smnumgps  smmaprt smexncrt sm550nrm
#&gt; 1  RNASEQ    NA 0.988668   339167 0.975561       NA 0.991069 0.843516       NA
#&gt; 2  RNASEQ    NA 0.988598   536713 0.973821       NA 0.991280 0.821863       NA
#&gt; 3  RNASEQ    NA 0.990838   290714 0.973011       NA 0.993009 0.801211       NA
#&gt; 4  RNASEQ    NA 0.988295   293534 0.967582       NA 0.990607 0.763716       NA
#&gt; 5  RNASEQ    NA 0.988508   369265 0.975620       NA 0.991073 0.823269       NA
#&gt; 6  RNASEQ    NA 0.991046   394834 0.976320       NA 0.993359 0.848322       NA
#&gt;   smgnsdtc smunmprt sm350nrm smrdlgth smmncpb   sme1mmrt smsflgth smestlbs
#&gt; 1    21262        1       NA       76      NA 0.00209364      139        0
#&gt; 2    22592        1       NA       76      NA 0.00224570      157        0
#&gt; 3    23205        1       NA       76      NA 0.00202882      131        0
#&gt; 4    23092        1       NA       76      NA 0.00208335      110        0
#&gt; 5    22184        1       NA       76      NA 0.00222215      144        0
#&gt; 6    21412        1       NA       76      NA 0.00221069      158        0
#&gt;     smmppd  smnterrt smrrnanm  smrdttl smvqcfl smmncv smtrscpt smmppdpr
#&gt; 1 63926700 0.0241488   738605 64502700 5839470     NA    21277 31792900
#&gt; 2 73718500 0.0258600   374650 74367000 6567550     NA    22622 36646200
#&gt; 3 61337000 0.0266825   212377 61768800 3683040     NA    23225 30529500
#&gt; 4 51773500 0.0321327   235558 52264400 3637220     NA    23118 25760400
#&gt; 5 68070100 0.0240411   283345 68683200 6200030     NA    22201 33853700
#&gt; 6 72052900 0.0234005   491168 72534500 7105490     NA    21427 35856700
#&gt;   smcglgth smgappct smunpdrd smntrnrt smmpunrt smexpeff smmppdun   sme2mmrt
#&gt; 1       NA       NA        0 0.132045 0.991069 0.835983 63926700 0.00297382
#&gt; 2       NA       NA        0 0.151958 0.991280 0.814696 73718500 0.00313601
#&gt; 3       NA       NA        0 0.171800 0.993009 0.795610 61337000 0.00277657
#&gt; 4       NA       NA        0 0.203866 0.990607 0.756543 51773500 0.00345301
#&gt; 5       NA       NA        0 0.152351 0.991073 0.815920 68070100 0.00305874
#&gt; 6       NA       NA        0 0.127998 0.993359 0.842688 72052900 0.00291208
#&gt;   sme2anti smaltalg sme2snse smmflgth sme1anti smspltrd   smbsmmrt sme1snse
#&gt; 1 13807000  5680860 13836800      159 13810400 11069400 0.00253266 13979700
#&gt; 2 15965200  6506070 15978200      169 16007900 13524600 0.00268965 16124700
#&gt; 3 13458000  5397620 13489700      141 13510700 10989200 0.00240188 13565700
#&gt; 4 11429000  6115360 11465000      112 11507200  9054400 0.00276658 11502900
#&gt; 5 14772000  6248800 14774700      154 14798900 12928200 0.00263936 14915200
#&gt; 6 15389200  5930350 15519600      172 15501800 13665500 0.00256057 15558500
#&gt;   sme1pcts   smrrnart sme1mprt smnum5cd smdpmprt sme2pcts junction_count
#&gt; 1  50.3047 0.01145080 0.993470       NA        0  50.0538         247600
#&gt; 2  50.1818 0.00503785 0.993962       NA        0  50.0203         188783
#&gt; 3  50.1015 0.00343826 0.995181       NA        0  50.0589         302563
#&gt; 4  49.9907 0.00450704 0.992919       NA        0  50.0785         253719
#&gt; 5  50.1956 0.00412539 0.993638       NA        0  50.0046         227259
#&gt; 6  50.0913 0.00677150 0.995672       NA        0  50.2108         458369
#&gt;   junction_coverage junction_avg_coverage
#&gt; 1           7282179              29.41106
#&gt; 2          15845256              83.93370
#&gt; 3          16243715              53.68705
#&gt; 4          13421701              52.89987
#&gt; 5          16123862              70.94928
#&gt; 6          10194323              22.24043</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Leonardo Collado-Torres.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.5.1.9000.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


